# mgsa.taxonomy

## Table of Contents
- [people](#people)
- [video](#video)
- [updates](#updates)
- [GTDB](#gtdb) Genome Taxonomy Database
- [PhyloCode](#phylocode) International Code of Phylogenetic Nomenclatureï¼ˆå›½éš›ç³»çµ±å‘½åè¦ç´„ï¼‰
- [tag](#tag)
- [](#)


----------

2008å¹´9æœˆå·
https://www.nikkei-science.com/page/magazine/0809/200809_060.html
ç”Ÿç‰©ã®ç¨®ã¨ã¯ä½•ã‹ | æ—¥çµŒã‚µã‚¤ã‚¨ãƒ³ã‚¹
C. ã‚¸ãƒ³ãƒãƒ¼ï¼ˆã‚µã‚¤ã‚¨ãƒ³ã‚¹ãƒ©ã‚¤ã‚¿ãƒ¼ï¼‰

ã—ã‹ã—ï¼Œç³»çµ±å­¦çš„ç¨®æ¦‚å¿µã®æ¡ç”¨ã«ã‚ˆã‚Šï¼Œç¨®ã®ç´°åˆ†åŒ–ãŒä»Šå¾Œã©ã‚“ã©ã‚“é€²ã‚“ã§ã—ã¾ã†ã®ã§ã¯ãªã„ã‹ã¨è­¦æˆ’ã™ã‚‹æ„è¦‹ã‚‚ã‚ã‚‹ã€‚ãƒ­ãƒ³ãƒ‰ãƒ³å¤§å­¦ã‚¤ãƒ³ãƒšãƒªã‚¢ãƒ«ã‚«ãƒ¬ãƒƒã‚¸ã®ãƒ¡ã‚¤ã‚¹ï¼ˆGeorgina Maceï¼‰ã¯ï¼Œã€Œç³»çµ±å­¦çš„ç¨®æ¦‚å¿µã®å•é¡Œç‚¹ã¯ï¼Œã©ã“ã¾ã§åˆ†ã‘ã‚Œã°ã„ã„ã®ã‹ãŒã‚ã‹ã‚‰ãªã„ã¨ã„ã†ç‚¹ã«ã‚ã‚‹ã€ã¨ã„ã†ã€‚å°‘ãªãã¨ã‚‚åŸç†çš„ã«ã¯ï¼Œçªç„¶å¤‰ç•°ãŒ1ã¤ã§ã‚‚ã‚ã‚Œã°ï¼Œãã‚Œã‚’å…±æœ‰ã™ã‚‹å°ã•ãªå‹•ç‰©ç¾¤ã«å¯¾ã—ã¦ç¨®åã‚’ä¸ãˆã‚‹æ ¹æ‹ ã¨ãªã‚‹ã ã‚ã†ã€‚

----------
## video

2023/12/05  HOBART
https://www.youtube.com/watch?v=jN2IB6V6SW4
SYM08 2 2 Genome Taxonomy Database and SeqCode: Microbial Taxonomy and Nomenclature in the Age of...

Genome Taxonomy Database and SeqCode: Microbial Taxonomy and Nomenclature in the Age of Big Sequence Data. Presented by Maria Chuvochina as part of SYM08 Exchange and reuse of environmental DNA and metabarcoding data at the TDWG 2023 Hybrid Annual Conference in Hobart, Tasmania 9â€“13 October. See content in TDWG Proceedings 2023: https://doi.org/10.3897/biss.7.111135 and check media tab for slides.

2023/11/02
https://www.youtube.com/watch?v=F6kxU2Fxz7g
27:37
Phil Hugenholtz - Does genome contamination compromise the GTDB taxonomy? | S02 MVIF.16
Microbiome Virtual International Forum
Title of the presentation: Does genome contamination compromise the GTDB taxonomy?
Speaker: Phil Hugenholtz

----------
## people

### leeswijzer
ä¸‰ä¸­ä¿¡å®

2013å¹´9æœˆ3æ—¥ç¬¬1åˆ·åˆŠè¡Œ
http://leeswijzer.org/files/NamingNature.html
ã‚­ãƒ£ãƒ­ãƒ«ãƒ»ã‚­ã‚µã‚¯ãƒ»ãƒ¨ãƒ¼ãƒ³ã€è‡ªç„¶ã‚’åã¥ã‘ã‚‹ï¼šãªãœç”Ÿç‰©åˆ†é¡ã§ã¯ç›´æ„Ÿã¨ç§‘å­¦ãŒè¡çªã™ã‚‹ã®ã‹ã€

### skato_jcm

https://twitter.com/skato_jcm
åŠ è—¤çœŸæ‚Ÿ (@skato_jcm) / X
åšå£«ï¼ˆç”Ÿå‘½ç§‘å­¦ï¼‰ã€‚å¾®ç”Ÿç‰©å­¦è€…ã€‚ç†åŒ–å­¦ç ”ç©¶æ‰€ãƒã‚¤ã‚ªãƒªã‚½ãƒ¼ã‚¹ç ”ç©¶ã‚»ãƒ³ã‚¿ãƒ¼ï¼ˆRIKEN-BRCï¼‰ãƒ»å¾®ç”Ÿç‰©ææ–™é–‹ç™ºå®¤ï¼ˆJCMï¼‰ãƒ»ä¸Šç´šç ”ç©¶å“¡ã€‚æµ·æ´‹ç ”ç©¶é–‹ç™ºæ©Ÿæ§‹ãƒ»æµ·åº•è³‡æºã‚»ãƒ³ã‚¿ãƒ¼ãƒ»æ‹›è˜ä¸»ä»»ç ”ç©¶å“¡ã€‚æ—¥æœ¬Archaeaç ”ç©¶ä¼šäº‹å‹™å±€ã€‚ä¸»ã«ç ”ç©¶é–¢é€£ã®æƒ…å ±ã‚’ç™ºä¿¡ã—ã¦ã„ã¾ã™ã€‚


9:54 AM Â· Jan 15, 2024
https://twitter.com/skato_jcm/status/1746697436234662105
åŠ è—¤çœŸæ‚Ÿ on X: "ICNPã¨SeqCodeã®æŠ˜è¡·æ¡ˆãŒIJSEMèªŒã§å…¬é–‹ã•ã‚ŒãŸã‚ˆã†ã§ã™ã€‚ The best of both worlds: a proposal for further integration of Candidatus names into the International Code of Nomenclature of Prokaryotes - https://t.co/oZUGttfb3Y" / X

----------

2023å¹´3æœˆ6æ—¥ï¼ˆæœˆï¼‰
https://www.jsmrs.jp/ja/#jsmrs_sympo2023
å…¬ç›Šè²¡å›£æ³•äººç™ºé…µç ”ç©¶æ‰€ å­¦ä¼šãƒ»ç ”ç©¶éƒ¨ä¼šåŠ©æˆ é§’å½¢å’Œç”·å…ˆç”Ÿè¿½æ‚¼å…¬é–‹ã‚·ãƒ³ãƒã‚¸ã‚¦ãƒ 
ã€Œæ–°ãŸãªæ™‚ä»£ã‚’æ‹“ãå¾®ç”Ÿç‰©åˆ†é¡å­¦ï¼šç¾çŠ¶ã¨å±•æœ›ã€

è¦æ—¨é›†ã¯ã“ã¡ã‚‰ 
https://www.jsmrs.jp/event/sympo2023/proceedings_hp.pdf

----------

2023 å¹´ 1 æœˆ 11 æ—¥
æ—¥æœ¬ã‚²ãƒãƒ å¾®ç”Ÿç‰©å­¦ä¼šãƒ‹ãƒ¥ãƒ¼ã‚¹ãƒ¬ã‚¿ãƒ¼ No.26
(PDF: 6.33 MB)
https://www.sgmj.org/pdf/newsletter/sgmj_no26.pdf
ãƒ¼åŸæ ¸ç”Ÿç‰©ã€€å‘½åæ³•ã€€Q&A
å¸‚å· å¤å­ã€ä¼Šè—¤ éš†ã€å²©å´ æ¸‰ã€ç‰æœ¨ ç§€å¹¸ã€å¸ƒæµ¦ æ‹“éƒã€ãƒ‹ãƒ¥ãƒ¼ã‚¹ãƒ¬ã‚¿ãƒ¼ç·¨é›†å§”å“¡

----------

2022 å¹´ 7 æœˆ 22 æ—¥
æ—¥æœ¬ã‚²ãƒãƒ å¾®ç”Ÿç‰©å­¦ä¼šãƒ‹ãƒ¥ãƒ¼ã‚¹ãƒ¬ã‚¿ãƒ¼ No.25
(PDF: 4.67 MB)
https://www.sgmj.org/pdf/newsletter/sgmj_no25.pdf
ãƒ¼åŸæ ¸ç”Ÿç‰© Prokaryote å‘½åæ³• Q&A
å¸‚å· å¤å­ã€ä¼Šè—¤ éš†ã€å²©å´ æ¸‰ã€ç‰æœ¨ ç§€å¹¸ã€å¸ƒæµ¦ æ‹“éƒã€ãƒ‹ãƒ¥ãƒ¼ã‚¹ãƒ¬ã‚¿ãƒ¼ç·¨é›†å§”å“¡


----------

----------
## ToL
tree of life

2:09 AM Â· Aug 6, 2024
https://x.com/miangoar/status/1820507357300002884
GAMA Miguel Angel on X: "Who was? ğŸ‘€ "The tree is featured on the Wikipedia [...], has been included in three museum exhibitions, in murals, on university swag and as ğ˜ğ—®ğ˜ğ˜ğ—¼ğ—¼ğ˜€ ğ—¼ğ—» ğ—®ğ˜ ğ—¹ğ—²ğ—®ğ˜€ğ˜ ğ˜ğ—µğ—¿ğ—²ğ—² ğ—½ğ—²ğ—¼ğ—½ğ—¹ğ—²" The ever-changing tree of life https://t.co/RWFXQ10GCv" / X

3:04 AM Â· Sep 9, 2023
https://x.com/miangoar/status/1700208600084271271
GAMA Miguel Angel on X: "@archaeal And you did it! ğŸ‰Congrats for that, it's a seminal work :) To my knowledge this is the last version of the tree of life with emphasis in uncultured species https://t.co/9LF5sBgMdY" / X

2:54 AM Â· Sep 9, 2023
https://x.com/miangoar/status/1700205995148898674
GAMA Miguel Angel on X: "Fun fact: there is no presentation where you talk about the new microbial species discovered by #metagenomics and not show the tree of Hug et al. If you don't put it into your slides, the ğŸ¦ law will come for you. (2016) A new view of the tree of life https://t.co/ZREJ3tG01B https://t.co/qwm72rA47e" / X

1:28 AM Â· Feb 23, 2023
https://x.com/SGribaldo/status/1628431492459241472
Simonetta Gribaldo on X: "Need an up-to-date #TreeofLife for teaching/conferences/wall paper? Talented PhD student @BasileBeaud drew a schematic consensus ToL from the most recently published phylogenies. Hope this is useful! Downloadable version in the threadğŸ‘‡ https://t.co/92GFL610zJ" / X

## 2024-08

https://pubmed.ncbi.nlm.nih.gov/39095498/
Nat Microbiol
. 2024 Aug;9(8):1906-1908. doi: 10.1038/s41564-024-01768-w.
The ever-changing tree of life
Laura A Hug 1
https://www.nature.com/articles/s41564-024-01768-w
The ever-changing tree of life | Nature Microbiology
https://www.nature.com/articles/s41564-024-01768-w/figures/1
Fig. 1: Key updates to the 2016 new view of the tree of life.

### 2018-03-08

https://pubmed.ncbi.nlm.nih.gov/29522741/
Review Cell
. 2018 Mar 8;172(6):1181-1197. doi: 10.1016/j.cell.2018.02.016.
Major New Microbial Groups Expand Diversity and Alter our Understanding of the Tree of Life
Cindy J Castelle 1, Jillian F Banfield 2
https://www.cell.com/cell/fulltext/S0092-8674(18)30160-0
https://www.sciencedirect.com/science/article/pii/S0092867418301600

### 2016-04-11

https://pubmed.ncbi.nlm.nih.gov/27572647/
Nat Microbiol
. 2016 Apr 11:1:16048. doi: 10.1038/nmicrobiol.2016.48.
A new view of the tree of life
Laura A Hug 1, Brett J Baker 2, Karthik Anantharaman 1, Christopher T Brown 3, Alexander J Probst 1, Cindy J Castelle 1, Cristina N Butterfield 1, Alex W Hernsdorf 3, Yuki Amano 4, Kotaro Ise 4, Yohey Suzuki 5, Natasha Dudek 6, David A Relman 7 8, Kari M Finstad 9, Ronald Amundson 9, Brian C Thomas 1, Jillian F Banfield 1 9
https://www.nature.com/articles/nmicrobiol201648

----------



----------
## updates

## 2024-07-28

12:48 PM Â· Jul 27, 2024
https://x.com/nishimura_luca/status/1817044304403267603
è¥¿æ‘ç‘ ä½³ (Luca Nishimura) on X: "ã€620æ—¥ç›®ã€‘ã‚¦ã‚¤ãƒ«ã‚¹ã‚²ãƒãƒ ã‚’æ¯”è¼ƒã—ã€ANIã®é•ã„ãŒç¨®ã‚’å®šç¾©ã™ã‚‹ã®ã«å¦¥å½“ã‹è©•ä¾¡ã—ãŸç ”ç©¶ã€‚99.5% ANIãŒå¤šãã®ãƒ•ã‚¡ãƒ¼ã‚¸ã«ãŠã‘ã‚‹ç¨®ã‚’è¦å®šã™ã‚‹é–¾å€¤ã«ãªã‚‹ã“ã¨ã‚’ç¤ºå”†ã€‚ANIã®é•ã„ã¯çµ„æ›ãˆã‚„é¸æŠã€ç”Ÿæ…‹å­¦çš„ãªè¦å› ã«ã‚ˆã£ã¦ç”Ÿã˜ã€ç¨®ã®å½¢æˆã‚„ç¶­æŒã«é–¢ä¿‚ã™ã‚‹å¯èƒ½æ€§ã‚’ç¤ºã—ãŸã€‚#æ¯æ—¥ã‚ã‚“ã¶ã‚“ https://t.co/zUHojusXGp" / X
https://pubmed.ncbi.nlm.nih.gov/39037288/
mBio
. 2024 Jul 22:e0153624. doi: 10.1128/mbio.01536-24. Online ahead of print.
A natural ANI gap that can define intra-species units of bacteriophages and other viruses
Borja Aldeguer-Riquelme 1, Roth E Conrad 1, Josefa AntÃ³n 2, Ramon Rossello-Mora 3, Konstantinos T Konstantinidis 1


## 2024-05

From: æ—¥æœ¬å¾®ç”Ÿç‰©ç”Ÿæ…‹å­¦ä¼šã€€äº‹å‹™å±€ <office@microbial-ecology.jp>
Subject: SeqCodeã«åŸºã¥ãå‘½åè¦ç´„ã«ã¤ã„ã¦ï¼ˆç¶šå ±ï¼‰
Date: May 24, 2024 14:54:43 JST

æ—¥æœ¬å¾®ç”Ÿç‰©ç”Ÿæ…‹å­¦ä¼šä¼šå“¡ã®çš†æ§˜

æœªåŸ¹é¤Šç³»çµ±ç¾¤ã«é–¢ã™ã‚‹å‘½åè¦ç´„ã«ã¤ã„ã¦è­°è«–ãŒãªã•ã‚Œã¦ã„ã‚‹ã¨ã“ã‚ã§ã™ãŒã€Bergey's International Society for Microbial Systematics (BISMiS)ã‹ã‚‰æ„è¦‹ãŒç™ºä¿¡ã•ã‚Œã¦ã„ã¾ã™ã€‚ä¼šå“¡çš†æ§˜ã¸ã”æ¡ˆå†…è‡´ã—ã¾ã™ã€‚

BISMiS YouTube channel: https://www.youtube.com/watch?v=1ot1DR65Tbg&t=937s&authuser=0

2:35 PM Â· May 20, 2024
https://x.com/BISMiS_/status/1792428966797848989
Bergey's Intl. Society for Microbial Systematics on X: "If you missed the talks and discussion from #BISMiSLive on the proposal to integrate Candidatus into ICNP, watch them now at https://t.co/CtJKsr9d0A @ICSP_news @seq_code @jangidk_ #TaxonomyIsFun @mcm_aripune @CCUG_Sweden @Leibniz_DSMZ_en @LPSN_DSMZ @MicrobioSoc @ISMEJournal" / X


### 2024-01-16

https://pubmed.ncbi.nlm.nih.gov/38085031/
mBio
. 2024 Jan 16;15(1):e0269623. doi: 10.1128/mbio.02696-23. Epub 2023 Dec 12.
An ANI gap within bacterial species that advances the definitions of intra-species units
Luis M Rodriguez-R # 1, Roth E Conrad # 2, Tomeu Viver 3, Dorian J Feistel 2, Blake G Lindner 2, Stephanus N Venter 4, Luis H Orellana 5, Rudolf Amann 5, Ramon Rossello-Mora 3, Konstantinos T Konstantinidis 2
https://journals.asm.org/doi/10.1128/mbio.02696-23
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10790751/

Intermediate identity genotypes, for example, sharing 85%â€“95% ANI, when present, are generally ecologically differentiated and scarcer in abundance, and thus should probably be considered distinct species (6, 13, 14) rather than representing cultivation or other sampling biases (15). 

## 2023-01-04

https://pubmed.ncbi.nlm.nih.gov/36585846/
Mol Biol Evol
. 2023 Jan 4;40(1):msac273. doi: 10.1093/molbev/msac273.
Escherichia Coli: What Is and Which Are?
Marta Cobo-SimÃ³n 1, Rowan Hart 1, Howard Ochman 1
https://academic.oup.com/mbe/article/40/1/msac273/6966034?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9830988/
We evaluated all complete genomes assigned to E. coli and its closest relatives according to the biological species concept (BSC), using evidence of reproductive isolation and gene flow (i.e., homologous recombination in the case of asexual bacteria) to ascertain species boundaries.

## 2022

2022 å¹´ 33 å·» 2 å· p. 77-85
https://www.jstage.jst.go.jp/article/jslab/33/2/33_77/_article/-char/ja/
ç·èª¬
ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã‹ã‚‰æ¢ã‚‹ä¹³é…¸èŒã®å¤šæ§˜æ€§
è°·æ¾¤ é–æ´‹

42) Parks DH, Chuvochina M, Reeves PR, Beatson SA, Hugenholtz P. (2021) Reclassification of Shigella species as later heterotypic synonyms of Escherichia coli in the Genome Taxonomy Database. bioRxiv. doi: https://doi.org/10.1101/2021.09.22.461432

## 2021-09-22

Posted September 22, 2021.
doi: https://doi.org/10.1101/2021.09.22.461432
Reclassification of Shigella species as later heterotypic synonyms of Escherichia coli in the Genome Taxonomy Database | bioRxiv
https://www.biorxiv.org/content/10.1101/2021.09.22.461432v1.full

### 2021-07-07

https://pubmed.ncbi.nlm.nih.gov/34234115/
Comment Nat Commun
. 2021 Jul 7;12(1):4060. doi: 10.1038/s41467-021-24129-1.
Reply to: "Re-evaluating the evidence for a universal genetic boundary among microbial species"
Luis M Rodriguez-R 1 2, Chirag Jain 3, Roth E Conrad 1, Srinivas Aluru 4, Konstantinos T Konstantinidis 5
https://www.nature.com/articles/s41467-021-24129-1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8263725/

https://pubmed.ncbi.nlm.nih.gov/34234129/
Comment Nat Commun
. 2021 Jul 7;12(1):4059. doi: 10.1038/s41467-021-24128-2.
Re-evaluating the evidence for a universal genetic boundary among microbial species
Connor S Murray 1, Yingnan Gao 1, Martin Wu 2
https://www.nature.com/articles/s41467-021-24128-2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8263626/

2:08 AM Â· Jul 31, 2020
https://twitter.com/MattagenOlmics/status/1288884073755074561
Matt Olm on Twitter: "Glad to see more recent interest on the question "Are bacterial species discrete units in nature?". A pre-print article came out a few days ago entirely in the "NO" camp (https://t.co/tQM4NRbKLM), but after reading I still think the answer is "YES". Here's why. (1/4)" / Twitter

In the final paragraphs the authors make theoretical arguments against using metagenomics (enviro DNA sequencing) to answer this question, but the arguments have no data to support them and have other problems (detailed rebuttal in bioRxiv comment http://disq.us/p/2av19w2) (4/4)

https://www.biorxiv.org/content/10.1101/2020.07.27.223511v1#comment-5010354002

### 2020-01-14

https://pubmed.ncbi.nlm.nih.gov/31937678/
mSystems
. 2020 Jan 14;5(1):e00731-19. doi: 10.1128/mSystems.00731-19.
Consistent Metagenome-Derived Metrics Verify and Delineate Bacterial Species Boundaries
Matthew R Olm 1 2, Alexander Crits-Christoph 2, Spencer Diamond 1, Adi Lavy 1, Paula B Matheus Carnevali 1, Jillian F Banfield 3 4 5 6
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6967389/

## 2019-11-28

https://www.ncbi.nlm.nih.gov/pubmed/31778148
Bioinformatics. 2019 Nov 28. pii: btz899. doi: 10.1093/bioinformatics/btz899. [Epub ahead of print]
Struo: a pipeline for building custom databases for common metagenome profilers.
de la Cuesta-Zuluaga J1, Ley RE1, Youngblut ND1.
AVAILABILITY AND IMPLEMENTATION:
Source code available for download at https://github.com/leylabmpi/Struo. Custom GTDB databases available at http://ftp.tue.mpg.de/ebio/projects/struo/.

### 2019-03-04

https://pubmed.ncbi.nlm.nih.gov/30836089/
Comment Curr Biol
. 2019 Mar 4;29(5):R169-R172. doi: 10.1016/j.cub.2019.01.033.
Systematics: The Cohesive Nature of Bacterial Species Taxa
Frederick M Cohan 1
https://www.cell.com/current-biology/fulltext/S0960-9822(19)30062-4
https://www.sciencedirect.com/science/article/pii/S0960982219300624

### 2018-01-03

https://pubmed.ncbi.nlm.nih.gov/30285620/
BMC Genomics
. 2018 Oct 3;19(1):724. doi: 10.1186/s12864-018-5099-6.
Exclusivity offers a sound yet practical species criterion for bacteria despite abundant gene flow
Erik S Wright 1 2, David A Baum 3
https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5099-6
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6171291/

7:46 AM Â· Oct 4, 2018
https://twitter.com/digitalwright/status/1047618983497818114
Erik Wright on X: "Is "species" a real thing for bacteria? @baobabbaum and I did a deep dive into thousands of bacterial genomes to figure it out. Turns out the answer is yes and no. Check it out in #BMCGenomics: https://t.co/COJRzSpbs5 https://t.co/FmYMJSuKQF" / X

## 2017

https://www.ncbi.nlm.nih.gov/pubmed/28731467
ISME J. 2017 Nov;11(11):2399-2406. doi: 10.1038/ismej.2017.113. Epub 2017 Jul 21.
Uncultivated microbes in need of their own taxonomy.
Konstantinidis KT1, RossellÃ³-MÃ³ra R2, Amann R3.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5649169/
Table 1
Proposed standards for high-quality taxa descriptions of uncultivated Bacteria and Archaea
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5649169/table/tbl1/?report=objectonly


January 16, 2017
http://schaechter.asmblog.org/schaechter/2017/01/do-bacterial-species-really-exist-and-why-should-we-care.html
Small Things Considered: Do bacterial species really exist and why should we care?

## 2015

https://www.ncbi.nlm.nih.gov/pubmed/26150420
Nucleic Acids Res. 2015 Aug 18;43(14):6761-71. doi: 10.1093/nar/gkv657. Epub 2015 Jul 6.
Microbial species delineation using whole genome sequences.
Varghese NJ1, Mukherjee S1, Ivanova N1, Konstantinidis KT2, Mavrommatis K3, Kyrpides NC4, Pati A4.

----------
## gtdb
Genome Taxonomy Database

https://x.com/ace_gtdb

- https://gtdb.ecogenomic.org/
- https://gtdb.ecogenomic.org/about
Cite GTDB
- https://gtdb.ecogenomic.org/downloads
- https://data.gtdb.ecogenomic.org/
- https://data.gtdb.ecogenomic.org/releases/
- https://data.gtdb.ecogenomic.org/releases/latest/
- https://data.gtdb.ecogenomic.org/releases/latest/genomic_files_reps/

https://data.gtdb.ecogenomic.org/releases/latest/FILE_DESCRIPTIONS.txt
```
* bac120_<release>.tree
    Bacterial reference tree inferred from the concatenation of 120 proteins and spanning the representative
    genomes for each bacterial species cluster. This tree is used to curate the GTDB taxonomy. The provided
    tree is in Newick format, decorated with the GTDB taxonomy, and contains non-parametric bootstrap support
    values.

* Files in genomic_files_reps are specific to GTDB species representatives:
    * bac120_marker_genes_reps_<release>.tar.gz
        Untrimmed and unaligned marker genes used in the concatenated alignment used to infer
        the bacterial reference tree. Genomes without a marker gene indicate either multiple hits or 
        no hits were found for that gene. Genes are provided as both nucleotide and amino acid
        sequences. Trimmed and aligned hits can be found in bac120_msa_marker_genes_reps_<release>.tar.gz.

    * bac120_msa_marker_genes_reps_<release>.tar.gz
        Trimmed and aligned marker genes for GTDB representative for each of the 120 bacterial proteins.
        
    * bac120_msa_reps_<release>.faa.gz
        Multiple sequence alignment used to infer the bacterial reference tree.
```

https://pubmed.ncbi.nlm.nih.gov/36218463/
Bioinformatics
. 2022 Nov 30;38(23):5315-5316. doi: 10.1093/bioinformatics/btac672.
GTDB-Tk v2: memory friendly classification with the genome taxonomy database
Pierre-Alain Chaumeil 1 2, Aaron J Mussig 1, Philip Hugenholtz 1, Donovan H Parks 1
https://academic.oup.com/bioinformatics/article/38/23/5315/6758240?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9710552/

https://pubmed.ncbi.nlm.nih.gov/34520557/
Nucleic Acids Res
. 2022 Jan 7;50(D1):D785-D794. doi: 10.1093/nar/gkab776.
GTDB: an ongoing census of bacterial and archaeal diversity through a phylogenetically consistent, rank normalized and complete genome-based taxonomy
Donovan H Parks 1, Maria Chuvochina 1, Christian Rinke 1, Aaron J Mussig 1, Pierre-Alain Chaumeil 1, Philip Hugenholtz 1
https://academic.oup.com/nar/article/50/D1/D785/6370255
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8728215/
DATA AVAILABILITY
The GTDB can be accessed at https://gtdb.ecogenomic.org/ and data files for each GTDB release are available from https://data.gtdb.ecogenomic.org/. Genomes comprising the GTDB are obtained from the NCBI Assembly database (www.ncbi.nlm.nih.gov/assembly). GTDB species clusters are updated using the Python code at https://github.com/Ecogenomics/gtdb-species-clusters which is made available under the GNU GPL v3.0.

https://pubmed.ncbi.nlm.nih.gov/31730192/
Bioinformatics
. 2019 Nov 15;36(6):1925-1927. doi: 10.1093/bioinformatics/btz848. Online ahead of print.
GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database
Pierre-Alain Chaumeil 1, Aaron J Mussig 1, Philip Hugenholtz 1, Donovan H Parks 1
https://academic.oup.com/bioinformatics/article/36/6/1925/5626182?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7703759/

https://pubmed.ncbi.nlm.nih.gov/30148503/
Nat Biotechnol
. 2018 Nov;36(10):996-1004. doi: 10.1038/nbt.4229. Epub 2018 Aug 27.
A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life
Donovan H Parks 1, Maria Chuvochina 1, David W Waite 1, Christian Rinke 1, Adam Skarshewski 1, Pierre-Alain Chaumeil 1, Philip Hugenholtz 1
https://www.nature.com/articles/nbt.4229

relative evolutionary divergence (RED)

Figure 1: Rank normalization through RED.

https://kazumaxneo.hatenablog.com/entry/2022/04/13/025100
GTDB-Tkã®version 2 - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

http://kazumaxneo.hatenablog.com/entry/2019/10/29/143024
GTDBã®ã‚ªãƒ³ãƒ©ã‚¤ãƒ³ç³»çµ±æ¨¹ AnnoTree - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

7:30 AM Â· Oct 28, 2019
https://twitter.com/kazumachack/status/1188583676772093952
Kazuma Uesaka on Twitter: "å¾“æ¥ã®åˆ†é¡æ³•ã§ã¯ãªãã€å…±é€šã—ã¦å­˜åœ¨ã™ã‚‹éºä¼å­ç¾¤ã‚’å…ƒã«ã—ãŸãƒã‚¯ãƒ†ãƒªã‚¢ã€ã‚¢ãƒ¼ã‚­ã‚¢ã®åˆ†é¡ã‚’ææ¡ˆã™ã‚‹ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã¨ãƒ„ãƒ¼ãƒ«ã®ç´¹ä»‹ã§ã™ã€‚ãƒ„ãƒ¼ãƒ«ã®æ–¹ã¯æ—¢ã«è¤‡æ•°ã®ãƒ„ãƒ¼ãƒ«ã«çµ„ã¿è¾¼ã¾ã‚Œã¦ã„ã¾ã™ã€‚
https://kazumaxneo.hatenablog.com/entry/2019/07/13/235653
æ™®éçš„ãª single-copy proteinsã«åŸºã¥ã„ã¦ãƒã‚¯ãƒ†ãƒªã‚¢ã¨ã‚¢ãƒ¼ã‚­ã‚¢ã‚’åˆ†é¡ã™ã‚‹Genome Taxonomy Database (GTDB) ã¨ãã®åˆ†é¡ãƒ„ãƒ¼ãƒ« GTDB-Tk - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------
## PhyloCode

http://phylonames.org/
Welcome to the website of the
International Society for Phylogenetic Nomenclature.

The ISPN encourages the development and use of phylogenetic nomenclature and oversees the implementation of a code of rules to govern phylogenetic nomenclature â€” the PhyloCode.
http://phylonames.org/code/

https://ja.wikipedia.org/wiki/PhyloCode
PhyloCode (ãƒ•ã‚¡ã‚¤ãƒ­ã‚³ãƒ¼ãƒ‰) ã¯ã€æ­£å¼åç§°ã‚’ International Code of Phylogenetic Nomenclatureï¼ˆå›½éš›ç³»çµ±å‘½åè¦ç´„ï¼‰ã¨ã„ã„ã€ç³»çµ±ç™ºç”Ÿã«åŸºã¥ã„ãŸå‘½åã«é–¢ã™ã‚‹è¦ç´„ã§ã‚ã‚‹[1]ã€‚ç‰¹ã«ã‚¯ãƒ¬ãƒ¼ãƒ‰ã®å‘½åã‚’è¦å®šã™ã‚‹ã“ã¨ã«ä¸»çœ¼ãŒç½®ã‹ã‚Œã¦ã„ã‚‹ã€‚

https://twitter.com/search?q=phylocode%20lang%3Aja&src=typed_query

6:38 PM Â· Feb 4, 2024
https://twitter.com/kaisekiriu/status/1754076889566859658
ã‚«ã‚¤ã‚»ã‚­ãƒªãƒ¥ã‚¦ on X: ""ç³»çµ±é–¢ä¿‚ã®ã¿ã«åŸºã¥ãå®šç¾©"ï¼"ç³»çµ±é–¢ä¿‚ä»¥å¤–ã®æƒ…å ±ã«ã‚‚åŸºã¥ãå®šç¾©"ã¯è‰¯ã„ç·šã„ã£ã¦ã‚‹æ°—ã‚‚ã™ã‚‹ã‘ã©ã€ã˜ã‚ƒã‚PhyloCodeã®ã„ã†"apomorphy modified crown clade"ã¯ã©ã£ã¡ãªã‚“ã§ã™ã‹ï¼Ÿã¿ãŸã„ãªã¨ã“ã‚ã‚‚ã‚ã‚‹ã€‚" / X

3:41 PM Â· Jan 26, 2021
https://twitter.com/leeswijzer/status/1353956139587629058
Minaka Nobuhiro ã€ˆã¿ãªã‹é£Ÿå ‚ã€‰åº—ä¸» on X: "ï¼»è’æ›¸æ—¥èªŒï¼½Philip D. Cantino and Kevin de Queiroz 2020. International Code of Phylogenetic Nomenclature (PhyloCode). CRC Press, Boca Raton. https://t.co/v432Uigndy â€»åºæ–‡40ãƒšãƒ¼ã‚¸ã‚’èª­ã‚“ã ãŒï¼Œ1990å¹´ã®ç«‹ã¡ä¸Šã’ä»¥æ¥ï¼Œãƒ•ã‚¡ã‚¤ãƒ­ã‚³ãƒ¼ãƒ‰ãŒãŸã©ã£ãŸ â€œè‹¦é›£ã®é“â€ ãŒã—ã®ã°ã‚Œã‚‹ï¼ https://t.co/TTNKjuol1x" / X

ç¶šï¼‰å…ˆæ—¥ã®æœ¬å±‹B&Bã§ã®å¤œå™ºã§ã‚‚è¨€åŠã—ãŸãŒï¼Œåˆ†é¡ã‚’ç³»çµ±ã« â€œå¾“å±â€ ã•ã›ã‚ˆã†ã¨ã™ã‚‹ãƒ•ã‚¡ã‚¤ãƒ­ã‚³ãƒ¼ãƒ‰æ´¾ã®é‡æœ›ã¯å†…å¤–ã®æŠµæŠ—ã«ã™ã‚Šæ¸›ã£ã¦ãã¦ã„ã‚‹ã‚ˆã†ãªæ°—ãŒã™ã‚‹ã‚“ã ã‘ã©ï¼ãƒ¯ã‚¿ã‚¯ã‚·ãŒçŸ¥ã£ã¦ã„ã‚‹1990å¹´ä»£åˆé ­ã®ãƒ•ã‚¡ã‚¤ãƒ­ã‚³ãƒ¼ãƒ‰èƒå‹•æœŸã¯æ—¢å­˜ã®ãƒªãƒ³ãƒå‘½åè¦ç´„ã‚’ç½®ãæ›ãˆã‚ˆã†ã¨ã™ã‚‹é¼»æ¯ã®è’ã•ã‚’æ„Ÿã˜ãŸï¼

ç¶šï¼‰ãã†ã„ãˆã°ï¼Œã‚­ãƒ£ãƒ­ãƒ«ãƒ»ã‚±ã‚¹ã‚¯ãƒ»ãƒ¦ãƒ³ï¼»ä¸‰ä¸­ä¿¡å®ãƒ»é‡ä¸­é¦™æ–¹å­è¨³ï¼½ã€è‡ªç„¶ã‚’åã¥ã‘ã‚‹ï¼šãªãœç”Ÿç‰©åˆ†é¡ã§ã¯ç›´æ„Ÿã¨ç§‘å­¦ãŒè¡çªã™ã‚‹ã®ã‹ ã€ http://leeswijzer.org/files/NamingNature.html ã«ã‚‚ï¼Œãƒ•ã‚¡ã‚¤ãƒ­ã‚³ãƒ¼ãƒ‰æ´¾ã‚’ã‚ãã‚‹æŠ—äº‰ã¸ã®è¨€åŠãŒã‚ã£ãŸãªï¼

ç¶šï¼‰ã€Œãƒ•ã‚¡ã‚¤ãƒ­ã‚³ãƒ¼ãƒ‰ã¨ã„ã†ãƒ—ãƒ­ã‚¸ã‚§ã‚¯ãƒˆã¯ï¼Œãƒ˜ãƒ‹ãƒƒã‚¯ãŒä¸»å¼µã—ãŸå³å¯†ãªåˆ†å²å­¦çš„ãªå‘½åæ–¹æ³•ã‚’ã™ã¹ã¦ã®ç”Ÿç‰©ã«å½“ã¦ã¯ã‚ã‚ˆã†ã¨ã™ã‚‹ã€ï¼ˆp. 321ï¼‰ï¼›ã€ŒçœŸã®å•é¡Œç‚¹ã¯ãƒ•ã‚¡ã‚¤ãƒ­ã‚³ãƒ¼ãƒ‰ãŒåˆ†é¡å­¦è€…ã«ã‚ˆã‚‹å‘½åæ³•ã«æ ¹æœ¬çš„ãªå¤‰æ›´ã‚’è¿«ã£ãŸã ã‘ã§ã¯ãªãï¼Œç’°ä¸–ç•Œã‚»ãƒ³ã‚¹ã¸ã®é…æ…®ãŒã„ã£ã•ã„ãªã‹ã£ãŸç‚¹ã«ã‚ã‚‹ã€ï¼ˆp. 322ï¼‰ï¼


2002 å¹´ 12 å·» p. 2-7
https://www.jstage.jst.go.jp/article/taxa/12/0/12_KJ00003894952/_article/-char/ja/
ã‚³ãƒŸãƒ¥ãƒ‹ã‚±ãƒ¼ã‚·ãƒ§ãƒ³åª’ä½“ã¨ã—ã¦ã®ã‚¿ã‚¯ã‚½ãƒ³å : PhyloCodeã®æ½®ã¯å¼•ã„ãŸã‹?
å°å³¶ ç´”ä¸€
https://www.jstage.jst.go.jp/article/taxa/12/0/12_KJ00003894952/_pdf/-char/ja

2001 å¹´ 11 å·» p. 1-7
https://www.jstage.jst.go.jp/article/taxa/11/0/11_KJ00003894894/_article/-char/ja/
PhyloCodeã¨"ä¼çµ±çš„"åˆ†é¡å­¦
å°å³¶ ç´”ä¸€
https://www.jstage.jst.go.jp/article/taxa/11/0/11_KJ00003894894/_pdf/-char/ja

https://pubmed.ncbi.nlm.nih.gov/34892874/
Cladistics
. 2006 Apr;22(2):186-197. doi: 10.1111/j.1096-0031.2006.00097.x.
The PhyloCode: a critical discussion of its theoretical foundation
https://onlinelibrary.wiley.com/doi/10.1111/j.1096-0031.2006.00097.x

----------
## tag

2025-07-16
Googleæ¤œç´¢: ã‚¿ã‚° åˆ†é¡ éšå±¤ ç”Ÿç‰©

https://www.jstage.jst.go.jp/article/taxa/47/0/47_1/_article/-char/ja/
J-STAGEãƒˆãƒƒãƒ—/ã‚¿ã‚¯ã‚µï¼šæ—¥æœ¬å‹•ç‰©åˆ†é¡å­¦ä¼šèªŒ/47 å·» (2019)/æ›¸èªŒ
Articles
çˆ¬è™«é¡ã®åˆ†é¡å­¦ãƒ»ç³»çµ±å­¦ãƒ»ç”Ÿç‰©åœ°ç†å­¦â€•åˆ†å²åˆ†é¡å­¦ã®å•é¡Œç‚¹
ç–‹ç”° åŠª
https://www.jstage.jst.go.jp/article/taxa/47/0/47_1/_pdf/-char/ja
https://www.jstage.jst.go.jp/article/taxa/47/0/47_1/_pdf
æ®µéšç¾¤ã«ã‚ˆã‚‹åˆ†é¡ä½“ç³»
åˆ† å² ç¾¤ï¼ˆcladeï¼‰ ã‹ æ®µ éš ç¾¤ï¼ˆgradeï¼‰ ã¨ ã„ ã† ã® ã‚‚ï¼Œ
åˆ†å²åˆ†é¡å­¦è€…ã¨é€²åŒ–åˆ†é¡å­¦è€…ã®è«–äº‰ã®ç„¦ç‚¹ã®ä¸€ã¤ã§
ã‚ã£ãŸï¼

ã“ã®æ–¹æ³•ã«ãªã‚‰ã£ã¦ï¼Œç¥–å…ˆç¾¤ã®ä¸­ã«å­å­«ç¾¤
ã®å‡ºç¾ã—ãŸä½ç½®ã‚’ã‚¿ã‚°ã«ã‚ˆã£ã¦ç¤ºã™ã“ã¨ã§ï¼Œãƒªãƒ³ãƒ
å¼ã®æ®µéšç¾¤åˆ†é¡ã§ï¼Œå­å­«ç¾¤ã‚’å«ã‚€ç³»çµ±é–¢ä¿‚ã‚’ç¤ºã™ã“
ã¨ãŒã§ãã‚‹ï¼è¡¨ 1ã«çˆ¬è™«é¡ã®ç³»çµ±æ¨¹ã‚’æ®µéšåˆ†é¡ã«
ã‚ˆã£ã¦è¡¨ã—ãŸï¼ç¥–å…ˆç¾¤ã®çˆ¬è™«ç¶±ã«ã¯é³¥ç¶±ã‚‚å“ºä¹³ç¶±ã‚‚
å«ã‚ãšï¼Œãã®ã‚¿ã‚°ã‚’ï¼ˆé³¥ç¶±ï¼‰ã¨ï¼ˆå“ºä¹³ç¶±ï¼‰ã¨ã—ã¦ä½
ç½®ã ã‘ã‚’ç¤ºã™ã®ã§ã‚ã‚‹ï¼ã“ã®æ–¹æ³•ã§ã‚ã‚Œã°ï¼Œç§‘ãƒ¬ãƒ™
ãƒ«ã§ã‚‚å±ãƒ¬ãƒ™ãƒ«ã§ã‚‚ã©ã®éšç´šã§ã‚‚ç¥–å…ˆç¾¤ã®ä¸­ã§ã®å­
å­«ç¾¤ã®èµ·æºã¨ç³»çµ±é–¢ä¿‚ã‚’ç¤ºã™ã“ã¨ãŒã§ãã‚‹ï¼



2019.2.17
https://yuyakinoshita.com/blog/2019/02/17/classification-method/
WordPressã®è¨˜äº‹åˆ†é¡ã‚’ã‚«ãƒ†ã‚´ãƒªãƒ¼ã‹ã‚‰ã‚¿ã‚°ã«å¤‰æ›´ã—ãŸç†ç”±ã¨ãƒ¡ãƒªãƒƒãƒˆãƒ»ãƒ‡ãƒ¡ãƒªãƒƒãƒˆ | Blog | Yuya Kinoshita

https://docs.alfresco.com/using-ja/tasks/site-content-tag.html
ã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã®ã‚¿ã‚°ä»˜ã‘ã¨åˆ†é¡ | Alfresco Documentation
ã‚¿ã‚°ã§ã¯ã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã‚’éšå±¤åŒ–ã§ãã¾ã›ã‚“ãŒã€æ¤œç´¢ã«ä¾¿åˆ©ã§ã™ã€‚ä¸€æ–¹ã€ã‚«ãƒ†ã‚´ãƒªã§ã¯ã€ã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã‚’åˆ†é¡ã—ã¦éšå±¤åŒ–ã§ãã¾ã™ã€‚

Googleæ¤œç´¢: éšå±¤ åˆ†é¡ ã‚¿ã‚°ä»˜ã‘

http://hp.vector.co.jp/authors/VA027560/notes/tag.html
åˆ†é¡ã‚¿ã‚°ã¨ã¯
å¤šé¢æ€§
ä¸€ã¤ã®é …ç›®ã«ã¯ã€è¤‡æ•°ã®ç•°ãªã‚‹ã‚¿ã‚°ã‚’ä»˜ã‘ã‚‹ã“ã¨ãŒã§ãã¾ã™ã€‚ã“ã‚Œã«ã‚ˆã‚Šã€ã€ŒA ã§ã‚ã‚Š B ã§ã‚‚ã‚ã‚‹ã€ã¨ã„ã†ã‚ˆã†ãªã€å¤šé¢çš„ã§æŸ”è»Ÿãªåˆ†é¡ãŒå¯èƒ½ã«ãªã‚Šã¾ã™ã€‚ã“ã®é …ç›®ã¯ã€åˆ†é¡ A ã®é›†åˆã¨åˆ†é¡ B ã®é›†åˆã®ä¸¡æ–¹ã«åŒæ™‚ã«å­˜åœ¨ã—ã¾ã™ã€‚
ã¾ã¨ã‚
ç‰¹ã«æ–‡æ›¸ã‚„ç”»åƒãªã©ã®å‰µä½œç‰©ã¯ã€ä¸€ã¤ã®é …ç›®å˜ä½ã«è¤‡æ•°ã®åˆ†é‡ã‚’å†…åŒ…ã—ã¦ã„ã‚‹ã“ã¨ãŒã‚ã‚Šã¾ã™ã€‚ãŸã¨ãˆã°ã€æ—¥è¨˜ã€æ–‡æ›¸ãƒ»ç”»åƒç®¡ç†ã€ãƒ–ãƒƒã‚¯ãƒãƒ¼ã‚¯ãªã©ã§ã€ã“ã®ã‚ˆã†ãªåˆ†é¡æ–¹æ³•ãŒåŠ¹æœçš„ã§ã—ã‚‡ã†ã€‚
â€¦ éšå±¤æ§‹é€ ã®å•é¡Œç‚¹ã¯ã€ã™ã¹ã¦ã®çŸ¥è­˜ã¯ãã‚Œãã‚Œé©åˆ‡ãªå ´æ‰€ã«åã‚ã‚‰ã‚Œã¦ã„ã‚‹ã¨ã„ã†å‰æã®ä¸Šã«æˆã‚Šç«‹ã£ã¦ã„ã‚‹ã“ã¨ã ã€‚å®Ÿéš›ã¯ã€1ã¤ã®å ´æ‰€ã«é©åˆã™ã‚‹çŸ¥è­˜ãªã©ãªã„ã€‚éå¸¸ã«ç´›ã‚‰ã‚ã—ã„ã‚‚ã®ã 

http://piggydb.jp/d/28
ã‚¿ã‚°ã‚’ã‚¿ã‚°ã§åˆ†é¡ã™ã‚‹ï¼ˆéšå±¤ã‚¿ã‚°ï¼‰
ã‚¿ã‚®ãƒ³ã‚°ã®å•é¡Œç‚¹ã¯ã€ã‚¿ã‚°ã®æ•°ãŒå¤šããªã£ãŸå ´åˆã«åæ‹¾ãŒã¤ã‹ãªããªã£ã¦ã—ã¾ã†ã“ã¨ã§ã™ã€‚ã¾ãŸã€æœ¬æ¥åˆ†é¡ã«ã¯ä¸Šä¸‹é–¢ä¿‚ãŒã‚ã‚Šã¾ã™ãŒã€å¾“æ¥ã®ã‚¿ã‚°ã‚·ã‚¹ãƒ†ãƒ ã§ãã‚Œã‚’è¡¨ç¾ã™ã‚‹ã®ã¯éå¸¸ã«æ‰‹é–“ãŒã‹ã‹ã‚Šã¾ã™ã€‚


2020å¹´2æœˆ11æ—¥
https://8vivid.net/folder-management-limit/
ãƒ•ã‚©ãƒ«ãƒ€ãƒ¼ã‚’åˆ©ç”¨ã—ãŸãƒ•ã‚¡ã‚¤ãƒ«ç®¡ç†ã®é™ç•Œèª¬ï½œ8vivid
ã€ç©é›†åˆã€ã¨ã¯ï¼Ÿ
Aã«ã‚‚Bã«ã‚‚å±ã™ã‚‹ã‹ã‚‰ã€ã©ã£ã¡ã®ãƒ•ã‚©ãƒ«ãƒ€ãƒ¼ã«ã‚‚æ ¼ç´ã—ãŸã„â€¦
å‚ç›´çš„ãªåˆ†é¡æ³•ã§ã‚ã‚‹ã‚«ãƒ†ã‚´ãƒªãƒ¼ã€‚æ°´å¹³çš„ãªåˆ†é¡æ³•ã§ã‚ã‚‹ã‚¿ã‚°ã€‚
ãƒ•ã‚©ãƒ«ãƒ€ãƒ¼ã®éšå±¤åŒ–ã§ã®ç®¡ç†ã«ã¯é™ç•ŒãŒã‚ã‚‹
ãƒ•ã‚©ãƒ«ãƒ€ãƒ¼ã¯å‚ç›´çš„ç®¡ç†ãƒ»ã‚¿ã‚°ã¯æ°´å¹³çš„ç®¡ç†


2015.11.10
https://ocw.kyoto-u.ac.jp/ja/general-education-jp/library2015/pdf/No.5-7_note.pdf
ã€æƒ…å ±ã®çµ„ç¹”åŒ–ã¨æ¤œç´¢ï¼šç†è«–ã¨å®Ÿéš›ã€
é»’æ©‹ç¦å¤«æ•™æˆ(æƒ…å ±å­¦ç ”ç©¶ç§‘)
- å‹•æ¤ç‰©ã®åˆ†é¡ï¼šã‚¢ãƒªã‚¹ãƒˆãƒ†ãƒ¬ã‚¹/äººç‚ºåˆ†é¡, ãƒªãƒ³ãƒ/éšå±¤çš„ã‚«ãƒ†ã‚´ãƒª, ã‚¢ãƒ€ãƒ³ã‚½ãƒ³/é¡å‹åˆ†é¡,
ãƒ©ãƒãƒ«ã‚¯/ç³»çµ±åˆ†é¡
- ãƒ•ã‚©ãƒ¼ã‚¯ã‚½ãƒãƒŸãƒ¼ï¼šå…±åŒä½œæ¥­ã«ã‚ˆã‚‹åˆ†é¡ã€‚ã‚¦ã‚§ãƒ–ä¸Šã®æƒ…å ±ã¸ã®ã‚¿ã‚°ä»˜ã‘ã€‚
ã‚½ãƒ¼ã‚·ãƒ£ãƒ«ãƒ–ãƒƒã‚¯ãƒãƒ¼ã‚¯ã€å†™çœŸï½¥å‹•ç”»å…±æœ‰ã‚µã‚¤ãƒˆ


2005/12/26 2015/5/8
http://d.zeromemory.info/forksonomy-taxsonomy2
éšå±¤å‹åˆ†é¡ã¨ã‚¿ã‚°å‹åˆ†é¡ã®é•ã„ã«ã¤ã„ã¦(Folksonomy)ãã®ï¼’ - suVeneã®ã‚¢ãƒ¬
2005/12/25 2015/5/8
http://d.zeromemory.info/forksonomy-taxsonomy
éšå±¤å‹åˆ†é¡ã¨ã‚¿ã‚°å‹åˆ†é¡ã®é•ã„ã«ã¤ã„ã¦(Folksonomy) - suVeneã®ã‚¢ãƒ¬
ãã—ã¦ã€éšå±¤å‹ã®ä¸€ç•ªã®å•é¡Œã¨ã—ã¦ è¤‡æ•°ã®å´é¢ã‚’æŒã¤ã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã‚’ã©ã†æ‰±ã†ã‹ã ã€‚
ãã“ã§ã€æ–°ãŸã«ã¨ã£ã¦ä»£ã‚ã‚‹ã‚‚ã®ã¨ã—ã¦è€ƒãˆã‚‰ã‚Œã¦ã„ã‚‹ã®ãŒ ã‚¿ã‚°ä»˜ã‘ã«ã‚ˆã‚‹æ•´ç†ã ã€‚
Folksonomy(ãƒ•ã‚©ãƒ¼ã‚¯ã‚½ãƒãƒŸãƒ¼)ã‚¢ãƒ¼ã‚­ãƒ†ã‚¯ãƒãƒ£ã¨å‘¼ã°ã‚Œã‚‹ã€‚
Folksonomyã¨ã¯ã€Folksï¼ˆçš†ã®ï¼‰taxonomyï¼ˆåˆ†é¡å­¦ï¼‰ã¨ã„ã†äºŒã¤ã®è¨€è‘‰ã‚’çµ„ã¿åˆã‚ã›ãŸé€ èªã€‚



Googleæ¤œç´¢: taxonomy tag classification hierarchical

https://www.thepolymathlab.com/categories-and-tags-explained

Categories
taxonomy
classification
taxonomic hierarchy

Tags
non-hierarchical keyword about an item.


B Blackburn è‘— - â€2012
https://onlinelibrary.wiley.com/doi/pdf/10.1002/9781118433829.app1
Information Organization and Classifi cation: Taxonomies and Metadata - Information Governance - Wiley Online Library
Social Tagging and Folksonomies
metadata tags.



2010
https://dbsj.org/journal/dbsj_journal/dbsj_journal_vol_9_no_1_47_52/
DBSJ Journal, Vol.9, No.1, pp.47-52 éšå±¤çš„ã‚ªãƒ¼ãƒˆã‚¿ã‚®ãƒ³ã‚°ã«ã‚ˆã‚‹Q&Aã‚³ãƒŸãƒ¥ãƒ‹ãƒ†ã‚£ã®çŸ¥è­˜æ•´ç† | æ—¥æœ¬ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹å­¦ä¼š

https://db-event.jpn.org/deim2010/proceedings/files/D3-4.pdf
DEIM Forum 2010 D3-4
éšå±¤çš„ã‚ªãƒ¼ãƒˆã‚¿ã‚®ãƒ³ã‚°ã«ã‚ˆã‚‹Q&Aã‚³ãƒŸãƒ¥ãƒ‹ãƒ†ã‚£ã®çŸ¥è­˜æ•´ç†
è¿‘å¹´ã§ã¯ï¼Œã‚«ãƒ†ã‚´ãƒªã®é™ç•Œã‚’æ‰“ç ´ã™ã‚‹ã‚‚ã®ã¨ã—ã¦ï¼Œã‚¿ã‚°ã¨å‘¼
ã°ã‚Œã‚‹ééšå±¤çš„ãªçŸ­ã„ç´¢å¼•èªå¥ã‚’æ•´ç†å¯¾è±¡ã«è¤‡æ•°ä»˜ä¸ã™ã‚‹æ–¹
æ³•ã«æ³¨ç›®ãŒé›†ã¾ã£ã¦ã„ã‚‹ [1]ï¼ã‚¿ã‚°ã«ã‚ˆã‚‹æƒ…å ±æ•´ç†ã¯ï¼Œä»˜ä¸ã™
ã‚‹ã‚¿ã‚°ã®å€‹æ•°ã‚„ç¨®é¡ã«åˆ¶é™ã‚’é€šå¸¸è¨­ã‘ãªã„ãŸã‚ï¼Œæ–°å¥‡ãªèªå¥ã«
ã‚ˆã‚‹æ¤œç´¢ã‚„ï¼Œã‚«ãƒ†ã‚´ãƒªã‚ˆã‚Šã‚‚å…·ä½“çš„ãªæ¤œç´¢ã®æ”¯æ´ã‚’è¡Œãˆã‚‹åˆ©ç‚¹
ãŒã‚ã‚‹ï¼ã“ã®åˆ©ç‚¹ã‚’ç”Ÿã‹ã—ã¦ï¼Œå†™çœŸå…±æœ‰ã‚µã‚¤ãƒˆã® Flickrï¼ˆæ³¨3ï¼‰ã‚„
ã‚½ãƒ¼ã‚·ãƒ£ãƒ«ãƒ–ãƒƒã‚¯ãƒãƒ¼ã‚¯ã® Deliciousï¼ˆæ³¨4ï¼‰ãªã©ã§ã¯ï¼Œå¤šæ•°ã®ãƒ¦ãƒ¼
ã‚¶ãŒè¤‡æ•°ã®ã‚¿ã‚°ã‚’è‡ªç”±ã«ä»˜ã‘åŠ ãˆã¦ã„ãæ–¹æ³•ã§ã‚ã‚‹ãƒ•ã‚©ãƒ¼ã‚¯ã‚½
ãƒãƒŸãƒ¼ï¼ˆfolksonomy = folks + taxonomy; æ°‘è¡†ã«ã‚ˆã‚‹åˆ†é¡æ³•ï¼‰ã‚’
æ¡ç”¨ã—ï¼Œå¤§ããªæˆåŠŸã‚’åã‚ã¦ã„ã‚‹ [2], [3]ï¼

### folksonomy

https://ja.wikipedia.org/wiki/ãƒ•ã‚©ãƒ¼ã‚¯ã‚½ãƒãƒŸãƒ¼
folksonomy
ãƒ•ã‚©ãƒ¼ã‚¯ã‚½ãƒãƒŸãƒ¼ã®ä¾¿ç›Š
æƒ…å ±æ¤œç´¢ã‚„å›³æ›¸é¤¨æƒ…å ±å­¦ã®ãƒ—ãƒ­ãŒæ„å‘³ã‚’æ˜ç¢ºã«ã—ãªãŒã‚‰æ§‹ç¯‰ã—ãŸçµ±åˆ¶èªå½™ï¼ˆçµ±åˆ¶èªã€çµ±åˆ¶ç´¢å¼•èªã€ controlled vocabularies ï¼‰ã«ã‚ˆã‚‹åˆ†é¡æ³•ã¨é•ã„ã€ãƒ•ã‚©ãƒ¼ã‚¯ã‚½ãƒãƒŸãƒ¼ã¯ã‚·ã‚¹ãƒ†ãƒãƒãƒƒã‚¯ã§ã¯ãªãã€æƒ…å ±ç§‘å­¦è€…ã®ç›®ã‹ã‚‰è¦‹ã‚Œã°æ´—ç·´ã•ã‚Œã¦ã„ãªã„åˆ†é¡æ³•ã§ã‚ã‚‹ã€‚ã—ã‹ã—ã€ã‚¤ãƒ³ã‚¿ãƒ¼ãƒãƒƒãƒˆã®ãƒ¦ãƒ¼ã‚¶ãƒ¼ã«ã¨ã£ã¦ã¯ã€è¤‡é›‘ãªã€éšå±¤çš„ã«çµ„ç¹”ã•ã‚ŒãŸå‘½åæ³•ã‚’ç¿’ã‚ãšã«ã™ã‚€ãŸã‚ã€ã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã®åˆ†é¡ã®ã‚³ã‚¹ãƒˆãŒãã‚ã‚ã¦ä½ããªã‚‹ã€‚ãƒ¦ãƒ¼ã‚¶ãƒ¼ã¯ãã®å ´ã§ã‚¿ã‚°ã‚’ä½œã‚Šã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã«ã¤ã‘ã‚Œã°ã„ã„ã ã‘ã§ã‚ã‚‹ã€‚

2009
https://ipsj.ixsq.nii.ac.jp/ej/?action=pages_view_main&active_action=repository_view_main_item_detail&item_id=71659&item_no=1&page_id=13&block_id=8
æƒ…å ±å­¦åºƒå ´ï¼šæƒ…å ±å‡¦ç†å­¦ä¼šé›»å­å›³æ›¸é¤¨
ä¸»é¡Œã«å¿œã˜ãŸè¦–ç‚¹åˆ†æã«åŸºã¥ãç”»åƒã®è‡ªå‹•ã‚¿ã‚°ä»˜ã‘æ–¹å¼ã®ææ¡ˆ
ãƒ¦ãƒ¼ã‚µã‚™ã‹ã‚™å”èª¿ã—ã¦ã‚³ãƒ³ãƒ†ãƒ³ãƒ„ç¾¤ã«ã‚¿ã‚¯ã‚™ä»˜ã‘ã‚’è¡Œã†ã“ã¨ ã¦ã‚™ç”Ÿæˆã•ã‚ŒãŸã‚¿ã‚¯ã‚™é›†åˆã‹ã‚™ç¤ºã™åˆ†é¡ä½“ç³»ã‚’ folksonomy[4]ã¨ å‘¼ã¯ã‚™ã‚Œæ³¨ç›®ã•ã‚Œã¦ã„ã‚‹.ã“ã‚Œã«ã‚ˆã‚Šï¼Œå„ãƒ¦ãƒ¼ã‚µã‚™ã‹ã‚™è‡ªåˆ†ã® è¦–ç‚¹ã¦ã‚™ã‚³ãƒ³ãƒ†ãƒ³ãƒ„å¯¾ã—ã¦ï¼Œè‡ªç”±ã«ã‚¿ã‚¯ã‚™ã¨å‘¼ã¯ã‚™ã‚Œã‚‹ãƒ¡ã‚¿ãƒ†ã‚™ ãƒ¼ã‚¿ã‚’ä»˜ä¸ã—ï¼Œãã‚Œã‚‰ã‚’å…±æœ‰ã—ãã®ã‚¿ã‚¯ã‚™è‡ªä½“ã‚’å”èª¿ç·¨é›† ã™ã‚‹ã“ã¨ã«ã‚ˆã‚Šï¼Œ ã‚¿ã‚¯ã‚™ã«ã‚ˆã‚Šã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã‹ã‚™åˆ†é¡ã•ã‚Œã‚‹. ã“ã‚Œã«ã‚ˆã‚Šï¼Œãƒ¡ã‚¿ãƒ†ã‚™ãƒ¼ã‚¿ã‚’å–å¾—ã™ã‚‹ãŸã‚ã®äººçš„ã‚³ã‚¹ãƒˆã‚’ æŠ‘ãˆï¼Œæ—¥ã€…å¢—å¤§ã™ã‚‹ã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã«å¯¾ã™ã‚‹ã‚¢ãƒãƒ†ãƒ¼ã‚·ãƒ§ãƒ³ ã‹ã‚™å¯èƒ½ã¨ãªã‚‹.
December 2004
https://adammathes.com/academic/computer-mediated-communication/folksonomies.html
Folksonomies - Cooperative Classification and Communication Through Shared Metadata

https://www.infosta.or.jp/journal/200802j.html
ã€Œæƒ…å ±ã®ç§‘å­¦ã¨æŠ€è¡“ã€ æŠ„éŒ² ã€€ Vol. 58 (2008), No.2
ç‰¹é›† :ã€Œåˆ†é¡ã‚’ã¿ã¤ã‚ãªãŠã™ã€ã®ç·¨é›†ã«ã‚ãŸã£ã¦
ã€€è¿‘å¹´Webã®ä¸–ç•Œã§"Folksonomy"ãŒè©±é¡Œã«ãªã£ã¦ã„ã¾ã™ã€‚ã“ã‚Œã¯ï¼Œå¾“æ¥ã®å˜ç´”ãªæ¤œç´¢ã§ã¯ã™ãã„åˆ‡ã‚Œãªã„ï¼Œæƒ…å ±æ¢ç´¢ã®ãƒ‹ãƒ¼ã‚ºã‚’æº€ãŸã™ãŸã‚ã«ã€Œåˆ†é¡ã€ãŒå¿…è¦ã ã¨ã„ã†ã“ã¨ã®è¡¨ã‚Œã¨ã¨ã‚‰ãˆã‚‹ã“ã¨ãŒã§ãã‚‹ã‹ã¨æ€ã„ã¾ã™ã€‚
ã€€ã—ã‹ã—ï¼ŒFolksonomyã«ã¯ã€Œåˆ†é¡ä½“ç³»ã€ã¨ã„ã†ã‚‚ã®ã¯ã‚ã‚Šã¾ã›ã‚“ã€‚

https://wired.jp/2005/02/07/ãƒãƒƒãƒˆä¸–ç•Œã‚’ã‚¿ã‚°ã§åˆ†é¡ã™ã‚‹ã€Œãƒ•ã‚©ãƒ¼ã‚¯ã‚½ãƒãƒŸãƒ¼/
ãƒãƒƒãƒˆä¸–ç•Œã‚’ã‚¿ã‚°ã§åˆ†é¡ã™ã‚‹ã€Œãƒ•ã‚©ãƒ¼ã‚¯ã‚½ãƒãƒŸãƒ¼ã€ | WIRED.jp
ã€€ã“ã®ã¨ã“ã‚ã€ãƒ¦ãƒ¼ã‚¶ãƒ¼ãŒã‚³ãƒ³ãƒ†ãƒ³ãƒ„ã‚’ä½œã£ã¦ã„ãã‚¿ã‚¤ãƒ—ã®ã‚µã‚¤ãƒˆã§ã¯ã€ãƒã‚¿ãƒ¼ãƒ•ã‚£ãƒ¼ãƒ«ãƒ‰æ°ãŒè¨€åŠã™ã‚‹ã‚ˆã†ãªä»˜åŠ ä¾¡å€¤ã®ãŸã‚ã«ã€ã‚¿ã‚°ãƒ»ã‚·ã‚¹ãƒ†ãƒ â€•â€•ã€Œãƒ•ã‚©ãƒ¼ã‚¯ã‚½ãƒãƒŸãƒ¼ã€(folksonomy)[folks(äººã€…)ã¨taxonomy(åˆ†é¡æ³•)ã‚’åˆã‚ã›ãŸé€ èª]ã¨ã‚‚è¨€ã‚ã‚Œã‚‹â€•â€•ã‚’æŸ±ã«ã™ã‚‹ã¨ã“ã‚ãŒå¢—ãˆã¦ã„ã‚‹ã€‚

----------


