# mgsa.rRNA

## Table of Contents
- [unclassified](#unclassified)
- [microbiome](#microbiome)
- [phage](#phage)
- [plasmid](#plasmid)
- [updates](#updates)
- [people](#people)
  - [anda](#anda)
  - [miyazaki-kentaro](#miyazaki-kentaro)
- [book](#book)
- [data](#data)
  - [Greengenes](#greengenes)
  - [RDP](#rdp)
- [](#)
- [hypervariable region](#hypervariable-region)

----------

----------
## unclassified

https://ja.wikipedia.org/wiki/16S_rRNAç³»çµ±è§£æ
å¾®ç”Ÿç‰©ç³»çµ±å­¦ã®åˆ†é‡ã«ãŠã„ã¦ã€ã“ã®éºä¼å­é…åˆ—ã¯ç³»çµ±é€²åŒ–è§£æã«ã‚ˆãåˆ©ç”¨ã•ã‚Œã¦ã„ã‚‹[2][3][4][5]ã€‚

è§£æã«ãŠã‘ã‚‹æ³¨æ„ç‚¹
ç´°èŒãŒæŒã¤16S rRNAéºä¼å­é…åˆ—ã¯ä¸€ã¤ã¨ã¯é™ã‚‰ãšã€è¤‡æ•°ã®16S rRNAéºä¼å­ãŒã‚²ãƒãƒ ä¸­ã«ãƒãƒ«ãƒã‚³ãƒ”ãƒ¼ã§å«ã¾ã‚Œã‚‹ã“ã¨ãŒå¤šã„[35]ã€‚

16S rRNAéºä¼å­ã®äº¤é›‘
é€²åŒ–ãŒå‚ç›´ä¼é”ã«ã‚ˆã£ã¦é§†å‹•ã•ã‚Œã‚‹ã¨ã„ã†ä»®å®šã®ä¸‹ã§ã¯ã€16S rRNAéºä¼å­ã¯ç¨®ç‰¹ç•°çš„ã§ã‚ã‚‹ã¿ãªã™ã“ã¨ãŒã§ãã€åŸæ ¸ç”Ÿç‰©é–“ã®ç³»çµ±é–¢ä¿‚ã‚’æ¨æ¸¬ã™ã‚‹ç¢ºå®Ÿãªéºä¼çš„ãƒãƒ¼ã‚«ãƒ¼ã§ã‚ã‚‹ã¨é•·å¹´è€ƒãˆã‚‰ã‚Œã¦ããŸã€‚ã—ã‹ã—ãªãŒã‚‰ã€ç ”ç©¶ãŒé€²ã‚€ã«é€£ã‚Œã€ã“ã‚Œã‚‰ã®éºä¼å­ã«ãŠã„ã¦ã‚‚éºä¼å­ã®æ°´å¹³ä¼æ’­ãŒç™ºç”Ÿã—ã¦ã„ã‚‹ã“ã¨ãŒåˆ†ã‹ã£ã¦ããŸã€‚

2022/12/12

https://rpubs.com/hiroiwata/981939
ãƒã‚¤ã‚ªãƒ¡ãƒˆãƒªãƒƒã‚¯ã‚¹ç¬¬9å›ã€€ç³»çµ±æ¨¹ä½œæˆ
å²©ç”°æ´‹ä½³
ã¯ã˜ã‚ã«
ã€€ã“ã“ã§ã¯ã€å¾®ç”Ÿç‰©å¢ã®ãƒ¡ã‚¿ã‚²ãƒãƒ ãƒ‡ãƒ¼ã‚¿ã®è§£æï¼ˆhttps://www.biorxiv.org/content/10.1101/2021.10.04.462986v2.full.pdf ï¼‰ã«ç”¨ã„ã‚‰ã‚ŒãŸã€ãƒ€ã‚¤ã‚ºã®æ ¹åœå¾®ç”Ÿç‰©ã®16S rRNAï¼ˆãƒªãƒœã‚¾ãƒ¼ãƒãƒ«RNAï¼‰ã®ãƒ‡ãƒ¼ã‚¿ã®ä¸€éƒ¨ã‚’ç”¨ã„ã¦è§£æã‚’è¡Œã†ã€‚ãªãŠã€rRNAã¯ã€ã‚¦ã‚£ãƒ«ã‚¹ã‚’é™¤ãå…¨ç”Ÿç‰©ã«å­˜åœ¨ã—ã¦ã€é€²åŒ–é€Ÿåº¦ãŒæ¯”è¼ƒçš„é…ã„ã“ã¨ã‹ã‚‰ã€ç¨®åˆ†é¡ãªã©ã«ã‚ˆãç”¨ã„ã‚‰ã‚Œã¦ã„ã‚‹ã€‚ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã§ã¯ã€å¾®ç”Ÿç‰©å¢ã®å¤šæ§˜æ€§ã¨ç³»çµ±é–“ã®é–¢ä¿‚ã‚’æŠŠæ¡ã™ã‚‹ãŸã‚ã«16S rRNAã‚’ç”¨ã„ã‚‰ã‚Œã‚‹ã“ã¨ãŒå¤šã„ï¼ˆ
https://bifidus-fund.jp/keyword/kw009.shtml
16S rRNAãƒ»16S rDNAï½œç”¨èªé›†ï½œè…¸å†…ç´°èŒå­¦ä¼š

----------
## microbiome

https://github.com/haruosuz/mgsa/blob/master/references/mgsa.metagenome.md

The Madness of Microbiome: Attempting To Find Consensus "Best Practice" for 16S Microbiome Studies.

----------
## hypervariable region

### 2024-10-23
https://pubmed.ncbi.nlm.nih.gov/39597509/
Microorganisms
. 2024 Oct 23;12(11):2119. doi: 10.3390/microorganisms12112119.
Effect of the 16S rRNA Gene Hypervariable Region on the Microbiome Taxonomic Profile and Diversity in the Endangered Fish Totoaba macdonaldi
Itzel Soledad PÃ©rez-Bustamante 1, Roberto Cruz-Flores 1, JesÃºs Antonio LÃ³pez-Carvallo 2, Samuel SÃ¡nchez-Serrano 3
https://www.mdpi.com/2076-2607/12/11/2119
https://pmc.ncbi.nlm.nih.gov/articles/PMC11596169/
Figure 1.
Representation of the 16S rRNA gene hypervariable regions. The positions of the utilized primers (forward and reverse) for the amplification of four hypervariable regions are shown. The start and finish of the V1â€“V2 (aqua), V2â€“V3 (purple), V3â€“V4 (pink), and V5â€“V7 (green) regions are highlighted. The positioning is based on the E. coli strain U 5/41 16S rRNA gene.
https://pmc.ncbi.nlm.nih.gov/articles/PMC11596169/figure/microorganisms-12-02119-f001/

### 2024-03-08
https://pubmed.ncbi.nlm.nih.gov/38559569/
ISME Commun
. 2024 Mar 8;4(1):ycae034. doi: 10.1093/ismeco/ycae034. eCollection 2024 Jan.
Taxonomic resolution of different 16S rRNA variable regions varies strongly across plant-associated bacteria
Katarina Hrovat 1, Bas E Dutilh 2 3 4, Marnix H Medema 1, Chrats Melkonian 1 2
https://academic.oup.com/ismecommun/article/4/1/ycae034/7624640
https://pmc.ncbi.nlm.nih.gov/articles/PMC10980831/

### 2023-03-09
https://pubmed.ncbi.nlm.nih.gov/36894603/
Sci Rep
. 2023 Mar 9;13(1):3974. doi: 10.1038/s41598-023-30764-z.
Determining the most accurate 16S rRNA hypervariable region for taxonomic identification from respiratory samples
Ruben LÃ³pez-Aladid # 1 2 3, Laia FernÃ¡ndez-Barat # 4 5 6, Victoria Alcaraz-Serrano 1 2 3, Leticia Bueno-Freire 1 2 3, Nil VÃ¡zquez 1 2 3, Roque Pastor-IbÃ¡Ã±ez 7, Andrea Palomeque 1 2 3, Patricia Oscanoa 1 2 3, Antoni Torres 8 9 10
https://www.nature.com/articles/s41598-023-30764-z
https://www.nature.com/articles/s41598-023-30764-z/figures/1
https://pmc.ncbi.nlm.nih.gov/articles/PMC9998635/
Figure 1.
Hypervariable region strategy on 16S rRNA. A summary of the four hypervariable region strategies for amplification of variable regions V1â€“V2, V3â€“V4, V5â€“V7, and V7â€“V9 in the 16S rRNA gene (positioning is based on the E.coli 16S rRNA gene).
https://pmc.ncbi.nlm.nih.gov/articles/PMC9998635/figure/Fig1/

----------
## phage

https://www.ncbi.nlm.nih.gov/pubmed/22902729
Microbiology. 2012 Nov;158(Pt 11):2789-95. doi: 10.1099/mic.0.057943-0. Epub 2012 Aug 17.
Evidence of bacteriophage-mediated horizontal transfer of bacterial 16S rRNA genes in the viral metagenome of the marine sponge Hymeniacidon perlevis.
Harrington C1, Del Casale A, Kennedy J, Neve H, Picton BE, Mooij MJ, O'Gara F, Kulakov LA, Larkin MJ, Dobson AD.

https://www.ncbi.nlm.nih.gov/pubmed/21666016
Appl Environ Microbiol. 2011 Aug;77(15):5529-32. doi: 10.1128/AEM.00457-11. Epub 2011 Jun 10.
Extent and variation of phage-borne bacterial 16S rRNA gene sequences in wastewater environments.
Del Casale A1, Flanagan PV, Larkin MJ, Allen CC, Kulakov LA.

----------
## plasmid

https://pubmed.ncbi.nlm.nih.gov/26534993/
Proc Natl Acad Sci U S A
. 2015 Nov 17;112(46):14343-7. doi: 10.1073/pnas.1514326112. Epub 2015 Nov 3.
Bacterial clade with the ribosomal RNA operon on a small plasmid rather than the chromosome
Mizue Anda 1, Yoshiyuki Ohtsubo 1, Takashi Okubo 1, Masayuki Sugawara 1, Yuji Nagata 1, Masataka Tsuda 1, Kiwamu Minamisawa 2, Hisayuki Mitsui 2
https://www.pnas.org/doi/10.1073/pnas.1514326112
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4655564/
rRNA is essential for life because of its functional importance in protein synthesis. The rRNA (rrn) operon encoding 16S, 23S, and 5S rRNAs is located on the â€œmainâ€ chromosome in all bacteria documented to date and is frequently used as a marker of chromosomes. Here, our genome analysis of a plant-associated alphaproteobacterium, Aureimonas sp. AU20, indicates that this strain has its sole rrn operon on a small (9.4 kb), high-copy-number replicon. 

Therefore, pAU20rrn represents a distinct class of replicons, which we designated the rrn-plasmid, which is coupled with an rrn-lacking chromosome (RLC).

https://www.ncbi.nlm.nih.gov/pubmed/14663090
Microbiology. 2003 Dec;149(Pt 12):3587-93.
A functional plasmid-borne rrn operon in soil isolates belonging to the genus Paracoccus.
Battermann A1, Disse-KrÃ¶mker C, Dreiseikelmann B.

https://www.ncbi.nlm.nih.gov/pubmed/11251820
Mol Microbiol. 2001 Feb;39(4):1010-21.
Analysis of the replicon region and identification of an rRNA operon on pBM400 of Bacillus megaterium QM B1551.
Kunnimalaiyaan M1, Stevenson DM, Zhou Y, Vary PS.

----------
## book

### 2020.02
https://sakuhinsha.com/nonfiction/27969.html
ç”Ÿå‘½ã®ã€ˆç³»çµ±æ¨¹ã€‰ã¯ã‹ã‚‰ã¿ã‚ã†
ã‚²ãƒãƒ ã«åˆ»ã¾ã‚ŒãŸã¾ã£ãŸãæ–°ã—ã„é€²åŒ–å²

https://www.ivis.co.jp/text/20221012.pdf
2022å¹´10æœˆ12æ—¥ ã‚ã‹ã¿ãšä¼š
éˆ´æœ¨ å‹

â€¢ ã‚«ãƒ¼ãƒ«ãƒ»ã‚¦ãƒ¼ã‚ºï¼ˆç±³å›½ã®å¾®ç”Ÿç‰©å­¦è€…ã€1928-2012ï¼‰ã¯ã€ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã‚’åˆæˆã™ã‚‹ãƒªãƒœã‚½ãƒ¼ãƒ 
ã‚’æ§‹æˆã™ã‚‹RNAï¼ˆåŸæ ¸ç”Ÿç‰©ã®â€16S rRNAâ€ã¨çœŸæ ¸ç”Ÿç‰©ã®â€18S rRNAâ€ï¼‰ã«ç€ç›®ã—ã€ç¨®ã€…ã®
ç´°èŒï¼ˆåŸæ ¸ç”Ÿç‰©ï¼‰ã‚„å˜ç´°èƒç”Ÿç‰©ï¼ˆçœŸæ ¸ç”Ÿç‰©ï¼‰ã®é–“ã®é•ã„ã‚’èª¿æŸ»ã—ãŸï¼ˆå›³2-1ï¼‰ã€‚ãã®çµæœã€
å¾“æ¥ã¯ç´°èŒã¨æ€ã‚ã‚Œã¦ã„ãŸãƒ¡ã‚¿ãƒ³ç”ŸæˆèŒãŒã€ä»–ã®ç”Ÿç‰©ã®ç´°èƒå£ãŒå«ã‚€ãƒšãƒ—ãƒãƒ‰ã‚°ãƒªã‚«
ãƒ³ã¨ã„ã†è„‚è³ªã‚’å«ã¾ãªã„ãªã©ã€åŸæ ¸ç”Ÿç‰©ã‚„çœŸæ ¸ç”Ÿç‰©ã¨ã¯åˆ¥å½¢æ…‹ã®ç‰¹å¾´ã‚’æŒã¤ã“ã¨ã‚’ç™º
è¦‹ã—ãŸã€‚

â€¢ ã‚«ãƒ¼ãƒ«ãƒ»ã‚¦ãƒ¼ã‚ºã¨å…±åŒç ”ç©¶è€…ã®ã‚¸ãƒ§ãƒ¼ã‚¸ãƒ»ãƒ•ã‚©ãƒƒã‚¯ã‚¹ï¼ˆç±³å›½ã®ç”Ÿç‰©å­¦è€…ã€1945-ï¼‰ã‚‰
ã¯1980å¹´ã®è«–æ–‡ã§ã€ã™ã¹ã¦ã®ç”Ÿç‰©ã‚’ã€Œç´°èŒ/ã‚¢ãƒ¼ã‚­ã‚¢/çœŸæ ¸ç”Ÿç‰©ã€ã®ä¸‰ã¤ã«åˆ†é¡ ã™ã‚‹ã€Œãƒ“ãƒƒã‚°ãƒ»ãƒ„ãƒªãƒ¼ã€ã‚’æå”±ã—ãŸï¼ˆå›³4-4ï¼‰ãŒã€ãã‚Œã‚‰ã®æ ¹å…ƒã¯ã€Œå…±é€šç¥–å…ˆçŠ¶æ…‹ã€ã¨
æ›–æ˜§ã«è¡¨ç¾ã•ã‚Œã¦ã„ãŸã€‚
â€¢ ãã®å¾Œã®ç ”ç©¶ã§ã€ã™ã¹ã¦ã®ç”Ÿç‰©ã®ç¥–å…ˆã¯RNAã«ã‚ˆã‚‹è‡ªå·±è¤‡è£½æ©Ÿèƒ½ã«ç”±æ¥ã™ã‚‹
ã¨ã„ã†è€ƒãˆæ–¹ãŒæ‹¡ãŒã£ãŸã€‚ã¾ãŸã€ã‚¢ãƒ¼ã‚­ã‚¢ã¯ç´°èŒã‚ˆã‚Šã‚‚çœŸæ ¸ç”Ÿç‰©ã¨ã®å…±é€šæ€§ãŒé«˜ ã„ã¨æ€ã‚ã‚Œã‚‹ã‚ˆã†ã«ãªã‚Šã€ã‚¦ãƒ¼ã‚ºã‚‰ã¯1990å¹´ã®è«–æ–‡ã§ã“ã‚Œã‚‰ã‚’åæ˜ ã—ãŸç³»çµ±æ¨¹ã‚’
ææ¡ˆã—ãŸï¼ˆå›³4-6ï¼‰ã€‚ã€Œå¤ç´°èŒã€ã¯ã€Œã‚¢ãƒ¼ã‚­ã‚¢ã€ã«æ”¹åã•ã‚Œã€ã¾ãŸã€ŒåŸæ ¸ç”Ÿç‰©ã€ã¨ã„ã†
åˆ†é¡ã¯æ¶ˆæ»…ã—ã¦ã„ã‚‹ã€‚

### 2018
https://davidquammen.com/the-tangled-tree/
The Tangled Tree - David Quammen
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/188bto4/alma9926392103104034
The Tangled Tree : A Radical New History of Life / David Quammen.

The_Tangled_Tree_A_Radical_New_History_of_Life_----_(Chapter_47).pdf

Woeseâ€™s unrooted tree, 1987, and the â€œprogenote.â€ Redrawn and modified, from Woese (1987), by Patricia J. Wynne.

The_Tangled_Tree_A_Radical_New_History_of_Life_----_(Chapter_48).pdf

The â€œNatural Systemâ€ tree of Woese, Kandler, and Wheelis, 1990.

### 2018-10-06
06 October 2018

https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/188bto4/alma99342392104031
Statistical Analysis of Microbiome Data with R / by Yinglin Xia, Jun Sun, Ding-Geng Chen.
Xia, Yinglin. author. Sun, Jun. author. Chen, Ding-Geng. author.
1st ed. 2018.
Singapore : Springer Nature Singapore : Imprint: Springer, 2018.
2018
ICSA Book Series in Statistics,

Ebook Central Perpetual, DDA and Subscription Titles

SpringerLink Books (Series & Reference Works)

9.5 Compare Microbiome Communities Using the GUniFrac Package
325

Step 3: Read Phylogenetic Tree
The GUniFrac package requires a rooted tree as input data. We can use the function midpoint() from the phangorn package to obtain the rooted tree.
```
> library(phangorn)
> otu_tree <- midpoint(otu_tree)
> otu_tree
```

### 2015
https://yalebooks.yale.edu/book/9780300223507/welcome-to-the-microbiome/
Welcome to the Microbiome
Getting to Know the Trillions of Bacteria and Other Microbes In, On, and Around You
by Rob DeSalle and Susan L. Perkins
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/188bto4/alma99275963904031
Ebook Central Perpetual, DDA and Subscription Titles

----------
## data

2021-12-13
https://staffblog.amelieff.jp/entry/metagenome.db
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ16S rRNAé…åˆ—ã®ï¼’å¤§ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ - ã‚¢ãƒ¡ãƒªã‚¨ãƒ•ã®æŠ€è¡“ãƒ–ãƒ­ã‚°
SILVA
Greengenes

----------
### ltp

https://pubmed.ncbi.nlm.nih.gov/34111737/
Syst Appl Microbiol
. 2021 Jul;44(4):126218. doi: 10.1016/j.syapm.2021.126218. Epub 2021 May 24.
Release LTP_12_2020, featuring a new ARB alignment and improved 16S rRNA tree for prokaryotic type strains
Wolfgang Ludwig 1, Tomeu Viver 2, Ralf Westram 3, Juan Francisco Gago 2, Esteban Bustos-Caparros 2, Katrin Knittel 1, Rudolf Amann 1, Ramon Rossello-Mora 4
https://www.sciencedirect.com/science/article/pii/S0723202021000412
https://imedea.uib-csic.es/mmg/ltp/#Downloads

----------
### Greengenes

### 2024-05
https://pubmed.ncbi.nlm.nih.gov/37500913/
Nat Biotechnol
. 2024 May;42(5):715-718. doi: 10.1038/s41587-023-01845-1. Epub 2023 Jul 27.
Greengenes2 unifies microbial data in a single reference tree
Daniel McDonald 1, Yueyu Jiang 2, Metin Balaban 3, Kalen Cantrell 4, Qiyun Zhu 5 6, Antonio Gonzalez 1, James T Morton 7, Giorgia Nicolaou 8, Donovan H Parks 9, SÃ¸ren M Karst 10, Mads Albertsen 11, Philip Hugenholtz 9, Todd DeSantis 12, Se Jin Song 13, Andrew Bartko 13, Aki S Havulinna 14 15, Pekka Jousilahti 14, Susan Cheng 16 17, Michael Inouye 18 19, Teemu Niiranen 14 20, Mohit Jain 21, Veikko Salomaa 14, Leo Lahti 22, Siavash Mirarab 2, Rob Knight 23 24 25 26
https://www.nature.com/articles/s41587-023-01845-1
https://pmc.ncbi.nlm.nih.gov/articles/PMC10818020/

However, UniFrac18, a phylogenetic method, used with our Greengenes2 tree provides better concordance (Fig. 1f and Supplementary Fig. 1d).

Integrating the GTDB and LTP taxonomies

----------
### RDP

### 2024-04-11
https://pubmed.ncbi.nlm.nih.gov/38436268/
Microbiol Resour Announc
. 2024 Apr 11;13(4):e0106323. doi: 10.1128/mra.01063-23. Epub 2024 Mar 4.
Updated RDP taxonomy and RDP Classifier for more accurate taxonomic classification
Qiong Wang 1, James R Cole 2
https://journals.asm.org/doi/full/10.1128/mra.01063-23
https://pmc.ncbi.nlm.nih.gov/articles/PMC11008197/

2:00 AM Â· Apr 11, 2024
https://twitter.com/ASMicrobiology/status/1778105814617604253
ASM on X: "The RDP Classifier is one of the most popular machine learning approaches for taxonomic classification. Researchers updated the RDP taxonomy to incorporate newly described taxa &amp; recent changes in nomenclature for prokaryotes. Learn more in #MRAJournal: https://t.co/2hAFX3KfBj https://t.co/XzIYQuEnpo" / X

### 2014-01
Ribosomal Database Project: data and tools for high throughput rRNA analysis.
Cole JR, Wang Q, Fish JA, Chai B, McGarrell DM, Sun Y, Brown CT, Porras-Alfaro A, Kuske CR, Tiedje JM Nucleic Acids Res. 2014 Jan;42(Database issue):D633-42. PMID:24288368
    is cited by the following papers.
åˆè¨ˆ: 1052, 
https://pubmed.ncbi.nlm.nih.gov/24288368/
Nucleic Acids Res
. 2014 Jan;42(Database issue):D633-42. doi: 10.1093/nar/gkt1244. Epub 2013 Nov 27.
Ribosomal Database Project: data and tools for high throughput rRNA analysis
James R Cole 1, Qiong Wang, Jordan A Fish, Benli Chai, Donna M McGarrell, Yanni Sun, C Titus Brown, Andrea Porras-Alfaro, Cheryl R Kuske, James M Tiedje
https://academic.oup.com/nar/article/42/D1/D633/1063201
https://pmc.ncbi.nlm.nih.gov/articles/PMC3965039/

https://integbio.jp/dbcatalog/record/nbdc00184

----------
### VITCOMIC2

https://github.com/haruosuz/bioinfo/blob/master/2022/CaseStudy.md#vitcomic2

### 2018-03-19
https://pubmed.ncbi.nlm.nih.gov/29560821/
BMC Syst Biol
. 2018 Mar 19;12(Suppl 2):30. doi: 10.1186/s12918-018-0545-2.
VITCOMIC2: visualization tool for the phylogenetic composition of microbial communities based on 16S rRNA gene amplicons and metagenomic shotgun sequencing
Hiroshi Mori 1, Takayuki Maruyama 2, Masahiro Yano 2, Takuji Yamada 2, Ken Kurokawa 3
https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-018-0545-2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861490/
Construction of a high-quality 16S rRNA gene sequence reference database including uncultured taxa
A high-quality 16S rRNA gene sequence reference database that includes uncultured taxa was constructed using 16S rRNA gene sequences obtained from the Ribosomal Database Project (RDP) [12].

----------
## updates

### 2025
1:03 AM Â· Jan 30, 2025
https://x.com/ASMicrobiology/status/1884633497681211661
ASM on X: "The MultiTax-human database integrates data from multiple sources and uses advanced classification techniques to provide a a high-resolution reference tool specifically designed for human microbiome research. Learn more in @JournalSpectrum: https://t.co/bdTdjbhAXz https://t.co/iKzqx1KmeC" / X
https://pubmed.ncbi.nlm.nih.gov/39817732/
Microbiol Spectr
. 2025 Jan 16:e0131224. doi: 10.1128/spectrum.01312-24. Online ahead of print.
MultiTax-human: an extensive and high-resolution human-related full-length 16S rRNA reference database and taxonomy
Zhiwei Bao 1 2, Bin Zhang 1, Jianhua Yao 2, Ming D Li 1
https://journals.asm.org/doi/10.1128/spectrum.01312-24
The databases of Greengenes2 (9), SILVA (release 138.1) (10), RDP (11), and GTDB (release 207) (12) were all downloaded from their official websites.
The code used to build the web application is available at https://github.com/zwbao/MultiTax-database. 

### 2024


### 2024-11

9:08 AM Â· Nov 5, 2024
https://x.com/nkjmu/status/1853590048253599786
Yu Nakajima on X: "@M_Banba_ ãã‚Œã®ç°¡ä¾¿ãªåˆ¤å®šã¨ã—ã¦ã€16S rRNAã®é¡ä¼¼åº¦ãŒ97%(è¿‘å¹´ã€98.65ã¨ã—ã¦ã„ã‚‹ã‚‚ã®ã‚‚ã‚ã‚Šã¾ã™ãŒ)ã§ã™ã‹ã­ ãŸã ã€97%ä»¥ä¸‹ãªã‚‰åˆ¥ç¨®(DNADNAãƒã‚¤ãƒ–ãƒª70%ä»¥ä¸‹)ã‚’æº€ãŸã—ã¾ã™ãŒã€é€†ã¯ãã†ã§ã¯ãªã„ã¨ã„ã†â€¦ (ä»¥å‰ä½¿ã£ãŸã‚¹ãƒ©ã‚¤ãƒ‰ã§ã™) https://t.co/wC7FklXep9" / X



### 2024-08

4:56 PM Â· Aug 5, 2024
https://x.com/iMetaScience/status/1820368375962317232
iMeta on X: "Long journey of #16S rRNAâ€amplicon #sequencing toward cellâ€based functional bacterial #microbiota characterization #iMetaOmics https://t.co/pkDy8HRnok #microbiome #SingleCell @wileymicrobio @wileyinresearch" / X
https://onlinelibrary.wiley.com/doi/full/10.1002/imo2.9
Long journey of 16S rRNAâ€amplicon sequencing toward cellâ€based functional bacterial microbiota characterization - Jin - iMetaOmics - Wiley Online Library

### 2024-07-22

3:56 PM Â· Jul 4, 2024
https://x.com/DaganLab/status/1808756699001495656
DaganLab on X: "Happy to share our results on #plasmid #evolution. We show that plasmid contribution to gene transfer corresponds mostly to transfer of the plasmid entity rather than gene transfer between plasmids and chromosomes. https://t.co/23FWndfWAj Great work by Samer Kadibalban! https://t.co/XsWiaIgNoE" / X

https://pubmed.ncbi.nlm.nih.gov/38964320/
Curr Biol
. 2024 Jul 22;34(14):3189-3200.e5. doi: 10.1016/j.cub.2024.06.030. Epub 2024 Jul 3.
The extent and characteristics of DNA transfer between plasmids and chromosomes
A Samer Kadibalban 1, Giddy Landan 1, Tal Dagan 2
https://www.cell.com/current-biology/fulltext/S0960-9822(24)00814-5
https://www.sciencedirect.com/science/article/pii/S0960982224008145
In addition to protein-coding genes, we examined evidence for the transfer of RNA genes between plasmids and chromosomes, which is considered as rare (but exceptions have been reported, e.g., for rRNA genes in Aureimonas plasmids42,43). Our analysis uncovered multiple segments that putatively correspond to transfer of RNA genes. The dataset analyzed here includes overall 197 rRNA genes in 42 plasmids, all of which are shared with the chromosome (Table S4). Out of the total 88 segments comprising rRNA genes, 61 segments correspond to a complete gene set of the rRNA operon. Example is sharing of an rRNA operon between plasmids and chromosomes in several isolates of the sugarcane endophyte Azospirillum brasilense, in agreement with previous reports.44 

Posted June 21, 2024.
https://www.biorxiv.org/content/10.1101/2024.06.21.600076v1
RecA is a reliable marker for bacterial taxonomy, even in the Candidate Phyla Radiation | bioRxiv

Culture-independent approaches are commonly used to characterise the taxonomic composition of bacterial communities. Among these approaches, the amplicon-based metagenomics relies on specific genetic markers, such as the 16S rRNA gene, while the shotgun metagenomics annotates the whole bacterial DNA. Despite the 16S being the gold standard marker, studies highlighted its inefficiency in characterising and quantifying divergent bacterial groups such as the Candidate Phyla Radiation.

In perspective, RecA could be considered as a broad-spectrum marker for amplicon-based studies to overcome the limits of 16S rRNA.

### 2024-04-20
https://pubmed.ncbi.nlm.nih.gov/38643216/
Sci Rep
. 2024 Apr 20;14(1):9067. doi: 10.1038/s41598-024-59667-3.
The overlooked evolutionary dynamics of 16S rRNA revises its role as the "gold standard" for bacterial species identification
OldÅ™ich BartoÅ¡ 1, Martin Chmel 2 3, Iva SwierczkovÃ¡ 2
https://www.nature.com/articles/s41598-024-59667-3
https://pmc.ncbi.nlm.nih.gov/articles/PMC11032355/

saturation

### 2024-04-11

2:00 AM Â· Apr 11, 2024
https://twitter.com/ASMicrobiology/status/1778105814617604253
ASM on X: "The RDP Classifier is one of the most popular machine learning approaches for taxonomic classification. Researchers updated the RDP taxonomy to incorporate newly described taxa &amp; recent changes in nomenclature for prokaryotes. Learn more in #MRAJournal: https://t.co/2hAFX3KfBj https://t.co/XzIYQuEnpo" / X
https://pubmed.ncbi.nlm.nih.gov/38436268/
Microbiol Resour Announc
. 2024 Apr 11;13(4):e0106323. doi: 10.1128/mra.01063-23. Epub 2024 Mar 4.
Updated RDP taxonomy and RDP Classifier for more accurate taxonomic classification
Qiong Wang 1, James R Cole 2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11008197/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861490/
VITCOMIC2: visualization tool for the phylogenetic composition of microbial communities based on 16S rRNA gene amplicons and metagenomic shotgun sequencing

A high-quality 16S rRNA gene sequence reference database that includes uncultured taxa was constructed using 16S rRNA gene sequences obtained from the Ribosomal Database Project (RDP) [12].

### 2023-11-14

https://pubmed.ncbi.nlm.nih.gov/37963895/
Nat Commun
. 2023 Nov 14;14(1):7232. doi: 10.1038/s41467-023-42681-w.
Bacteria can maintain rRNA operons solely on plasmids for hundreds of millions of years
Mizue Anda 1 2, Shun Yamanouchi 3, Salvatore Cosentino 4 3, Mitsuo Sakamoto 5, Moriya Ohkuma 5, Masako Takashima 5, Atsushi Toyoda 6, Wataru Iwasaki 7 8 9 10 11 12
https://www.nature.com/articles/s41467-023-42681-w
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10645730/

[ 23.12.28 ]
https://www.sgmj.org/index.php?page=news_index&nid=516
æ—¥æœ¬ã‚²ãƒãƒ å¾®ç”Ÿç‰©å­¦ä¼š - ãŠçŸ¥ã‚‰ã›ä¸€è¦§
https://www.sgmj.org/pdf/newsletter/sgmj_no28.pdf
ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¯å­å­«ã«å®‰å®šã«ä¼ã‚ã‚Šã«ãã„ã“ã¨ã‹ã‚‰ã€å¾®ç”Ÿ
ç‰©ã®ç”Ÿå­˜ã«å¿…é ˆãªéºä¼å­ã¯åŸå‰‡ã¨ã—ã¦æŸ“è‰²ä½“ä¸Šã«å­˜åœ¨ã—ã¾ã™ã€‚
ã¨ã‚Šã‚ã‘ã€å¾®ç”Ÿç‰©ã®åŸºæœ¬æ§‹æˆè¦ç´ ã®ä¸€ã¤ã§ã‚ã‚‹ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®
åˆæˆã«å¿…é ˆãªãƒªãƒœã‚½ãƒ¼ãƒ  RNAï¼ˆrRNAï¼‰éºä¼å­ã¯ã€æŸ“è‰²ä½“ã¨
ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã‚’åŒºåˆ¥ã™ã‚‹å®šç¾©ã®ä¸€éƒ¨ã¨ã—ã¦ä½¿ã‚ã‚Œã¦ããŸã“ã¨ã‚‚
ã‚ã‚‹ã»ã©ã€å¿…ãšæŸ“è‰²ä½“ä¸Šã«å­˜åœ¨ã™ã‚‹ã¨ã•ã‚Œã¦ãã¾ã—ãŸã€‚ã“ã‚Œ
ã¾ã§ã®ç ”ç©¶ã§ã€ã‚·ãƒ¥ãƒ¼ãƒ‰ãƒ¢ãƒŠãƒ‰ãƒ¼ã‚¿é–€ï¼ˆæ—§åç§°ï¼šãƒ—ãƒ­ãƒ†ã‚ªãƒ
ã‚¯ãƒ†ãƒªã‚¢é–€ï¼‰ã«å±ã™ã‚‹ Aureimonas å±ã®ãƒã‚¯ãƒ†ãƒªã‚¢ã¯ãƒ—ãƒ©ã‚¹ãƒŸ
ãƒ‰ã ã‘ã« rRNA éºä¼å­ã‚’æŒã¤ã“ã¨ãŒæ˜ã‚‰ã‹ã«ãªã£ã¦ã„ã¾ã—ãŸ
ãŒã€ã“ã†ã—ãŸç¾è±¡ãŒç‰¹å®šã®ãƒã‚¯ãƒ†ãƒªã‚¢ã«é™ã‚‰ã‚Œã‚‹ã‚‚ã®ãªã®ã‹ã€
ã¾ãŸã€ã“ã†ã—ãŸç¾è±¡ãŒä¸€æ™‚çš„ãªã‚‚ã®ãªã®ã‹ï¼ˆã™ãªã‚ã¡ã€æŸ“è‰²
ä½“ã‹ã‚‰ rRNA éºä¼å­ã‚’å¤±ã£ãŸçŠ¶æ…‹ã§å¾®ç”Ÿç‰©ãŒçµ¶æ»…ã‚’é•·æœŸé–“é€ƒ
ã‚Œã‚‹ã“ã¨ãŒã§ãã‚‹ã®ã‹ï¼‰ã¯åˆ†ã‹ã£ã¦ã„ã¾ã›ã‚“ã§ã—ãŸã€‚

11:00 AM Â· Nov 15, 2023
https://twitter.com/NIG_idenken/status/1724608158663098856
å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ on X: "ã€ç ”ç©¶æˆæœã€‘æ–°ã—ã„ç ”ç©¶æˆæœã‚’ç™ºè¡¨ã—ã¾ã—ãŸã€‚â– æ¯”è¼ƒã‚²ãƒãƒ è§£æç ”ç©¶å®¤ã€Œå¿…é ˆéºä¼å­ãŒæŸ“è‰²ä½“ã«ç„¡ãã¦ã‚‚ç”Ÿç‰©ã¯çµ¶æ»…ã—ãªã„â€•â€•æ•°å„„å¹´å‰ã‹ã‚‰ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã ã‘ã§ãƒªãƒœã‚½ãƒ¼ãƒ RNAéºä¼å­ã‚’ ç¶­æŒã™ã‚‹ãƒã‚¯ãƒ†ãƒªã‚¢ã®ç™ºè¦‹â€•â€•ã€ ğŸ‘‡è«–æ–‡ï¼ˆè‹±èªï¼‰ https://t.co/jbDxrg3nfR ğŸ‘‡è§£èª¬è¨˜äº‹ï¼ˆæ—¥æœ¬èªï¼‰ https://t.co/KGL8NhBpqB" / X
https://www.nig.ac.jp/nig/ja/2023/11/research-highlights_ja/pr20231114.html
å¿…é ˆéºä¼å­ãŒæŸ“è‰²ä½“ã«ç„¡ãã¦ã‚‚ç”Ÿç‰©ã¯çµ¶æ»…ã—ãªã„â€•â€•æ•°å„„å¹´å‰ã‹ã‚‰ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã ã‘ã§ãƒªãƒœã‚½ãƒ¼ãƒ RNAéºä¼å­ã‚’ ç¶­æŒã™ã‚‹ãƒã‚¯ãƒ†ãƒªã‚¢ã®ç™ºè¦‹â€•â€• | å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€

2023/11/14
https://www.k.u-tokyo.ac.jp/information/category/press/10606.html
å¿…é ˆéºä¼å­ãŒæŸ“è‰²ä½“ã«ç„¡ãã¦ã‚‚ç”Ÿç‰©ã¯çµ¶æ»…ã—ãªã„ â€• æ•°å„„å¹´å‰ã‹ã‚‰ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã ã‘ã§ãƒªãƒœã‚½ãƒ¼ãƒ RNAéºä¼å­ã‚’ ç¶­æŒã™ã‚‹ãƒã‚¯ãƒ†ãƒªã‚¢ã®ç™ºè¦‹ â€•ï½œè¨˜è€…ç™ºè¡¨ï½œãŠçŸ¥ã‚‰ã›ï½œæ±äº¬å¤§å­¦å¤§å­¦é™¢æ–°é ˜åŸŸå‰µæˆç§‘å­¦ç ”ç©¶ç§‘

https://nbrp.jp/public/news/press/news-231115/
ã€ç ”ç©¶è«–æ–‡ç™ºè¡¨ï½œä¸€èˆ¬å¾®ç”Ÿç‰©ã€‘å¿…é ˆéºä¼å­ãŒæŸ“è‰²ä½“ã«ç„¡ãã¦ã‚‚ç”Ÿç‰©ã¯çµ¶æ»…ã—ãªã„ ï¼æ•°å„„å¹´å‰ã‹ã‚‰ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã ã‘ã§ãƒªãƒœã‚½ãƒ¼ãƒ RNAéºä¼å­ã‚’ç¶­æŒã™ã‚‹ãƒã‚¯ãƒ†ãƒªã‚¢ã®ç™ºè¦‹ï¼ï½œãŠçŸ¥ã‚‰ã›ãƒ»ãƒ—ãƒ¬ã‚¹ãƒªãƒªãƒ¼ã‚¹ï½œNBRP

### 2022

### 2022-08-03

https://pubmed.ncbi.nlm.nih.gov/36304264/
Front Bioinform
. 2022 Aug 3:2:905489. doi: 10.3389/fbinf.2022.905489. eCollection 2022.
16S-ITGDB: An Integrated Database for Improving Species Classification of Prokaryotic 16S Ribosomal RNA Sequences
Yu-Peng Hsieh 1, Yuan-Mao Hung 2, Mong-Hsun Tsai 3 4, Liang-Chuan Lai 5 4, Eric Y Chuang 1 2 6 4
https://www.frontiersin.org/journals/bioinformatics/articles/10.3389/fbinf.2022.905489/full
https://pmc.ncbi.nlm.nih.gov/articles/PMC9580931/
RDP, SILVA, and Greengenes are the most widely used 16S rRNA databases. Many 16S rRNA classifiers have used these databases as a reference for taxonomic assignment tasks. However, some of the prokaryotic taxonomies only exist in one of the three databases. Furthermore, Greengenes and SILVA include a considerable number of taxonomies that do not have the resolution to the species level, which has limited the classifiers' performance.

https://github.com/yphsieh/16S-ITGDB

### 2022-07-08

https://pubmed.ncbi.nlm.nih.gov/35799218/
Microbiome
. 2022 Jul 8;10(1):104. doi: 10.1186/s40168-022-01295-y.
Phylogenies of the 16S rRNA gene and its hypervariable regions lack concordance with core genome phylogenies
Hayley B Hassler 1, Brett Probert 1, Carson Moore 1, Elizabeth Lawson 1, Richard W Jackson 2, Brook T Russell 3, Vincent P Richards 4
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01295-y
https://pmc.ncbi.nlm.nih.gov/articles/PMC9264627/


### 2002-03

https://colil.dbcls.jp/browse/papers/11931166/1/100/year
Evolution of the gyrB gene and the molecular phylogeny of Enterobacteriaceae: a model molecule for molecular systematic studies.
Dauga C Int J Syst Evol Microbiol. 2002 Mar;52(Pt 2):531-47. PMID:11931166
    is cited by the following papers.
â† å‰ã¸  åˆè¨ˆ: 18, 
https://pubmed.ncbi.nlm.nih.gov/11931166/
Comparative Study Int J Syst Evol Microbiol
. 2002 Mar;52(Pt 2):531-47. doi: 10.1099/00207713-52-2-531.
Evolution of the gyrB gene and the molecular phylogeny of Enterobacteriaceae: a model molecule for molecular systematic studies
Catherine Dauga 1
https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/00207713-52-2-531#tab2
Saturation of gyrB genes was observed by the pairwise comparison of Proteus spp., Providencia alcalifaciens and Morganella morganii sequences.

### 2021

### 2021-07-21

11:23 PM Â· Jul 21, 2021
https://twitter.com/mSphereJ/status/1417852717200969733
mSphere on Twitter: "Amplicon sequence variants artificially split bacterial genomes into separate clusters, by @PatSchloss. Special shout-out to the Code Club and #Riffomonas Project. #bioinformatics #microbiome https://t.co/qlIREIB4Xu @ASMicrobiology #mSphere https://t.co/cQE5mYAONv" / Twitter
https://journals.asm.org/doi/10.1128/mSphere.00191-21
Amplicon Sequence Variants Artificially Split Bacterial Genomes into Separate Clusters | mSphere

12:51 AM Â· Jul 22, 2021
https://x.com/genolib_19/status/1417874862970867717
Kei on X: "OTUã®ä»£ã‚ã‚Šã«ASVã‚’åˆ©ç”¨ã™ã‚‹ã“ã¨ã¯å˜ä¸€ã®ã‚²ãƒãƒ  (ç‰¹ã«16Sã‚³ãƒ”ãƒ¼ãŒè¤‡æ•°ã®å ´åˆ) ã‚’éåº¦ã«åˆ¥ã®ã‚‚ã®ã¨ã—ã¦èªè­˜ã—ã¦ã—ã¾ã†å¯èƒ½æ€§ãŒã‚ã‚‹ã¨ã®ã“ã¨ | Amplicon Sequence Variants Artificially Split Bacterial Genomes into Separate Clusters | mSphere https://t.co/IBkP5jxUfP" / X

### 2021-05

6:55 AM Â· May 11, 2021
https://twitter.com/mikerappe/status/1391874644332802052
Michael Rappe on Twitter: "Looking for good citations that reveal genomic heterogeneity between bacteria with identical 16S rRNA genes. Anyone have a favorite to share?" / Twitter

https://pubmed.ncbi.nlm.nih.gov/24505072/
Int J Syst Evol Microbiol
. 2014 Feb;64(Pt 2):346-351. doi: 10.1099/ijs.0.059774-0.
Towards a taxonomic coherence between average nucleotide identity and 16S rRNA gene sequence similarity for species demarcation of prokaryotes
Mincheol Kim 1, Hyun-Seok Oh 2, Sang-Cheol Park 2, Jongsik Chun 2 1
https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.059774-0#tab2

2021-02-09
å¤§è…¸èŒãƒªãƒœã‚½ãƒ¼ãƒ ã‚ªãƒšãƒ­ãƒ³ã®éºä¼å­å¤‰æ›
https://pubmed.ncbi.nlm.nih.gov/33585862/
G3 (Bethesda)
. 2021 Feb 9;11(2):jkaa002. doi: 10.1093/g3journal/jkaa002.
Rates of gene conversions between Escherichia coli ribosomal operons
Isaac Gifford 1, Aurko Dasgupta 1, Jeffrey E Barrick 1
https://academic.oup.com/g3journal/article/11/2/jkaa002/5974039
To more directly measure rates of rRNA intragenomic recombination, we sequenced the seven Escherichia coli rRNA operons in 15 lineages that were evolved for âˆ¼13,750 generations with frequent single-cell bottlenecks that reduce the effects of selection.

### 2020

### 2020-06-09
https://pubmed.ncbi.nlm.nih.gov/32518240/
Nat Commun
. 2020 Jun 9;11(1):2900. doi: 10.1038/s41467-020-16694-8.
Ribosome engineering reveals the importance of 5S rRNA autonomy for ribosome assembly
Shijie Huang # 1 2, Nikolay A Aleksashin # 1 3 4, Anna B Loveland # 5, Dorota Klepacki 1, Kaspar Reier 6, Amira Kefi 1, Teresa Szal 1, Jaanus Remme 6, Luc Jaeger 7, Nora VÃ¡zquez-Laslop 1 3, Andrei A Korostelev 8, Alexander S Mankin 9 10
https://www.nature.com/articles/s41467-020-16694-8
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7283268/

2:36 PM Â· Jun 10, 2020
https://twitter.com/puni2azarashi/status/1270590549414707205
å¤§è…¸èŒã§ãƒªãƒœã‚½ãƒ¼ãƒ RNA 5S rRNA ã®æ©Ÿèƒ½è§£æ˜ã«æŒ‘ã‚“ã è«–æ–‡ãŒå‡ºã¾ã—ãŸï¼ˆNature Communï¼‰!
ãƒªãƒœã‚½ãƒ¼ãƒ RNAã¯ã€åŸºæœ¬çš„ã«ã‚²ãƒãƒ ã‹ã‚‰è»¢å†™ã•ã‚ŒãŸå¾Œã€ãã‚Œãã‚Œã®ãƒ‘ãƒ¼ãƒ„ã«åˆ‡ã‚Šå‡ºã•ã‚Œã¾ã™ã€‚ä¾‹ãˆã°å¤§è…¸èŒãªã‚‰ã°ã€ãƒªãƒœã‚½ãƒ¼ãƒ RNAå‰é§†ä½“ãŒè»¢å†™ã•ã‚ŒãŸã®ã¡ã€5Sã€16Sã€23Sã«åˆ†ã‹ã‚Œã‚‹ã®ã§ã™ï¼ˆç¶š
ãã‚Œãªã‚‰5Sã‚‚16Sã‚‚23Sã‚‚1:1:1ã§å­˜åœ¨ã™ã‚‹ã¯ãšã€ã¨ãªã‚‹ã®ã§ã™ãŒã€èˆˆå‘³æ·±ã„ã“ã¨ã«ã€å¤šãã®ç”Ÿç‰©ï¼ˆçœŸæ ¸ç”Ÿç‰©ã‚’å«ã‚€ï¼‰ã¯ã€5Sã ã‘åˆ¥ã«ã€ä½™åˆ†ãªã‚³ãƒ”ãƒ¼ã‚’ã‚²ãƒãƒ ä¸Šã«æŒã£ã¦ã„ã¾ã™ã€‚ã—ã‹ã—ã€ãã®ç†ç”±ãŒã‚ã‹ã£ã¦ã„ã¾ã›ã‚“ã§ã—ãŸã€‚
ãƒªãƒœã‚½ãƒ¼ãƒ ã¯ã€åˆ‡ã‚Šå‡ºã•ã‚ŒãŸ5S 16S 23SãŒã™ã¹ã¦1:1:1ã§çµåˆã—ã¦å½¢æˆã•ã‚Œã¾ã™ï¼ˆç¶š
ã¤ã¾ã‚Šç†è«–ä¸Šã€5Sã¯ä½™ã‚‹ã®ã§ã™ã€‚ãã‚Œãªã‚‰ã€5Sã¯ãƒªãƒœã‚½ãƒ¼ãƒ æ§‹æˆè¦ç´ ã¨ã„ã†ä»•äº‹ä»¥å¤–ã«ã‚‚ä½•ã‹å¤§äº‹ãªå½¹ç›®ãŒã‚ã‚‹ã‹ã‚‚çŸ¥ã‚Œã¾ã›ã‚“ã€‚ã—ã‹ã—æ™®é€šã«ã™ã¹ã¦ã®5Sã‚’æ¬ æã•ã›ã‚‹ã¨ã€å¤§è…¸èŒã¯æ­»ã‚“ã§ã—ã¾ã„ã€ã†ã¾ãèª¿ã¹ã‚‰ã‚Œã¾ã›ã‚“ã€‚
ãã“ã§ç­†è€…ã‚‰ã¯ã€å¤§è…¸èŒã‚²ãƒãƒ ç·¨é›†ã—ã€éŠé›¢5S rRNAã ã‘ã‚’æ’é™¤ã™ã‚‹ã“ã¨ã«ã—ã¾ã—ãŸï¼ˆç¶š
ç­†è€…ã‚‰ã¯ã€23Sã¨5SãŒã€åˆ‡ã‚Šå‡ºã•ã‚ŒãŸã®ã¡çµåˆã™ã‚‹ã“ã¨ã«ç€ç›®ã—ã¦ã€23Sã®5Sçµåˆéƒ¨ä½ä»˜è¿‘ã‚’ç·¨é›†ã—ã¦ã€5Sé…åˆ—ã‚’å…¥ã‚Œã¦ã‚„ã‚Šã¾ã—ãŸï¼ˆçµ„ã¿è¾¼ã¿23Sã¨å‘¼ã³ã¾ã™ï¼‰ã€‚
ã“ã‚Œã§ã€ãƒªãƒœã‚½ãƒ¼ãƒ ã¯æ©Ÿèƒ½ã™ã‚‹ã‘ã‚Œã©ã‚‚ã€éŠé›¢5Sã®ãªã„å¤§è…¸èŒã‚’ç”Ÿã¿å‡ºã™ã“ã¨ã«æˆåŠŸã—ã¾ã—ãŸï¼ˆç¶š
é¢ç™½ã„ã“ã¨ã«ã€éŠé›¢5S rRNAã®ãªã„å¤§è…¸èŒã§ã‚‚ã€ã¡ã‚ƒã‚“ã¨å¢—æ®–ã™ã‚‹ã“ã¨ãŒã§ãã‚‹ã“ã¨ãŒã‚ã‹ã‚Šã¾ã—ãŸã€‚ã¤ã¾ã‚Šã€5S rRNAãŒä½™åˆ†ã«éŠé›¢ã•ã‚Œã‚‹ã“ã¨ã¯ã€ãã“ã¾ã§å¤§äº‹ã§ã¯ãªã‹ã£ãŸã®ã§ã™ï¼
ã—ã‹ã—ä¸€æ–¹ã€ãƒªãƒœã‚½ãƒ¼ãƒ ã®æ©Ÿèƒ½ãã®ã‚‚ã®ã¯å¤§ããä½ä¸‹ã—ã¦ã„ã‚‹ã“ã¨ã‚‚åˆ†ã‹ã‚Šã¾ã—ãŸï¼ˆç¶š
è©³ã—ãæ§‹é€ è§£æã‚’ã™ã‚‹ã¨ã€ãƒªãƒœã‚½ãƒ¼ãƒ ã®æ§‹é€ ã¯ã€çµ„ã¿è¾¼ã¿23Sã‚’å«ã‚€ãƒªãƒœã‚½ãƒ¼ãƒ ã¨æ™®é€šã®ãƒªãƒœã‚½ãƒ¼ãƒ ï¼ˆéŠé›¢5Sã‚’å«ã‚€ï¼‰ã§å¤§ããªå·®ã¯ã‚ã‚Šã¾ã›ã‚“ã§ã—ãŸã€‚
ä¸€æ–¹ã€ãƒªãƒœã‚½ãƒ¼ãƒ æ§‹æˆå‰ã®ã‚µãƒ–ãƒ¦ãƒ‹ãƒƒãƒˆã®æ§‹é€ ã«ã¯å·®ãŒã‚ã‚Šã€çµ„ã¿è¾¼ã¿23Sã‚’å«ã‚€æ–¹ã§ã¯ã€ãƒšãƒ—ãƒã‚¸ãƒ«ãƒˆãƒ©ãƒ³ã‚¹ãƒ•ã‚§ãƒ©ãƒ¼ã‚¼ä¸­å¿ƒã®å½¢æˆã«éšœå®³ãŒã‚ã‚Šã¾ã—ãŸï¼ˆç¶š
ã¤ã¾ã‚Šã€éŠé›¢5S rRNAã¯ã€ãƒªãƒœã‚½ãƒ¼ãƒ ã‚µãƒ–ãƒ¦ãƒ‹ãƒƒãƒˆãŒæ­£ã—ã„æ§‹é€ ã‚’å–ã‚‹ãŸã‚ã«ã€éŠé›¢ã•ã‚Œã¦ãŠã‹ã­ã°ãªã‚‰ãªã„ã€ã¨ã„ã†ã“ã¨ãŒã‚ã‹ã‚Šã¾ã—ãŸã€‚

### 2020-02

https://colil.dbcls.jp/browse/papers/31712737/1/100/year
https://pubmed.ncbi.nlm.nih.gov/31712737/
ISME J
. 2020 Feb;14(2):597-608. doi: 10.1038/s41396-019-0552-3. Epub 2019 Nov 11.
Unlinked rRNA genes are widespread among bacteria and archaea
Tess E Brewer 1 2, Mads Albertsen 3, Arwyn Edwards 4, Rasmus H Kirkegaard 3, Eduardo P C Rocha 5, Noah Fierer 6 7
https://academic.oup.com/ismej/article/14/2/597/7474940?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6976660/

Fig. 1
In most bacteria and archaea, rRNA genes are arranged in the order 16S-23S-5S, and are transcribed and regulated as a single unit. However, in some cases, the 16S is separated from the 23S and 5S, and is referred to as â€œunlinkedâ€

In addition, tRNA are the most common genes found in the space between the 16S and 23S rRNA genes, and range from only 75â€“90â€‰bp in length [38].

After classifying each rRNA gene pair as linked or unlinked based on the distance between the 16S and 23S rRNA genes,

### 2019

### 2019-01-28
https://pubmed.ncbi.nlm.nih.gov/30691394/
Comparative Study BMC Genomics
. 2019 Jan 28;20(1):92. doi: 10.1186/s12864-018-5389-z.
Comparative genomics of Bacteria commonly identified in the built environment
Nancy Merino 1 2, Shu Zhang 3 4, Masaru Tomita 5 6, Haruo Suzuki 7 8
https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5389-z
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6350394/

The diversity level of the 16S rRNA varied greatly between genera. 

Since the advent of high-throughput sequencing, several studies have used amplicon sequencing to gain more information about the microbial community of the BE, including the ribosomal RNA region (e.g., 16S rRNA) for Bacteria and Archaea and the internal transcribed spacer (ITS) region for Fungi [29].

Diversity among common BE genera
To assess the diversity of the â€œCommon BE genera,â€ we calculated the mean distance (Dmean) between all pairs of taxa within each genus based on 16S rRNA gene sequences available in the LTP datasets of the SILVA v128 release [58]. The SILVA database was selected over other 16S rRNA databases (e.g. Greengenes [59, 60] and RDP [61]) due to greater alignment quality [62] and because it is continuously updated [63]. 

However, the 16S rRNA gene has its limitations (e.g., sequence heterogeneity [66] and horizontal gene transfer [67]), even though it is widely used as a molecular clock to understand evolution [67â€“70].

Fig. 1
Diversity levels in 16S rRNA gene sequences for each bacterial genus commonly found in the built environment. The mean distance (Dmean) between all pairs of bacteria was used as a diversity index [58]

In addition, 16S rRNA amplicon sequencing was the dominant method used to determine the microbial community amongst the 54 publications used in this study. Some publications also conducted culture-based studies (e.g. study on airborne bacteria in Tokyo [112]). This introduces bias from the range of protocols used across publications, including sample collection methods (e.g. swab, wipe, air, and storage method), DNA extraction methods, primers used, 16S rRNA target region (e.g. V3â€“V4, V4, V6â€“V8), and sequencing methods [113â€“115]. With advances in sequencing for 16S rRNA (e.g., full-length [116]), genomes, and metagenomes (e.g., longer contigs, accurate base calling) and increased global research collaboration (e.g., MetaSUB [117]), more specific classification of BE microorganisms can be obtained at the species level, allowing for more accurate descriptions in future studies.

### 2018-06-08
https://colil.dbcls.jp/browse/papers/29937760/1/100/year
MINI REVIEW ARTICLE
https://pubmed.ncbi.nlm.nih.gov/29937760/
Front Microbiol
. 2018 Jun 8:9:1232. doi: 10.3389/fmicb.2018.01232. eCollection 2018.
Multiple Ribosomal RNA Operons in Bacteria; Their Concerted Evolution and Potential Consequences on the Rate of Evolution of Their 16S rRNA
Romilio T Espejo 1, NicolÃ¡s Plaza 1 2
https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2018.01232/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6002687/
Abstract
Bacterial species differ greatly in the number and location of the rRNA operons which may be present in the bacterial chromosomes and plasmids. Most bacterial species contain more than one ribosomal RNA operon copy in their genomes, with some species containing up to 15 such copies. We review the number and location of the rRNA operons and discuss evolution of 16S rRNA (rrs) genes -which are considered as ultimate chronometers for phylogenetic classification- in bacteria with multiple copies of these genes.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6002687/#sec-a.f.atitle
Ribosomal Rna Operons in Bacteria

FIGURE 1
Median of the number of rrs copies in 3,070 bacterial species according to data reported in rrnDB database (Stoddard et al., 2015).

In addition, rrn copies are also found in both chromosomes and plasmids as in B. megaterium (Kunnimalaiyaan et al., 2001) or Paracoccus species (Battermann et al., 2003). A special case is Aureimonas sp. AU20 in which the only rRNA operon is in the plasmid (Anda et al., 2015).

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6002687/#sec-a.f.btitle
Concerted Evolution of rRna Genes

Furthermore, a comparison of rrs sequences between closely related species and between rrs copies in the same genome indicated that as with most other genes, recombination with horizontally transferred rrs is relatively frequent (reviewed by Kitahara and Miyazaki, 2013). 

Recently, analysis of 2,143 genomes distinguished 28 genomes with high intragenomic heterogeneity revealing horizontal gene transfer (HGT) events and their potential donors (Tian et al., 2015). In all these 28 cases HGT of the 16S rRNA gene only occurred at intrageneous or intraspecies levels. More recently, a conserved, highly divergent rrs (7.3â€“9.0%) was found in one of the five ribosomal operons in the species complex Scytonema hyalinum (Johansen et al., 2017).

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6002687/#sec-a.f.etitle
Conclusion

The copy contributing part of the genome may be acquired by horizontal transfer. 16S rRNA sequences have long been used as an â€œultimate chronometerâ€ for phylogenetic classification of bacterial species but the effect of multiple rrs in bacterial genomes and their location in chromosomes or plasmids has not been considered.

### 2017-10-26
https://colil.dbcls.jp/browse/papers/29073157/1/100/year
https://pubmed.ncbi.nlm.nih.gov/29073157/
PLoS One
. 2017 Oct 26;12(10):e0186393. doi: 10.1371/journal.pone.0186393. eCollection 2017.
Highly divergent 16S rRNA sequences in ribosomal operons of Scytonema hyalinum (Cyanobacteria)
Jeffrey R Johansen 1 2, Jan MareÅ¡ 2 3 4, Nicole Pietrasiak 5, MarkÃ©ta BohunickÃ¡ 4, Jan Zima Jr 4, Lenka Å tenclovÃ¡ 2 3, TomÃ¡Å¡ Hauer 2 4
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0186393
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5658200/

A nearly complete genome scaffold of Scytonema hyalinum HK-05 was sequenced by other authors (NCBI accession AP018194), and reported with seven separate plasmids, including a 16.3 kb plasmid containing a full ribosomal operon that is discussed in this paper (NCBI accession AP018198).

The localization of the Type 1 operon on a plasmid in HK-05 further supports the HGT hypothesis, as vector-facilitated HGT is more common than direct transfer of chromosomal segments. It is intriguing that this is the first report of an rrn-plasmid in cyanobacteria, and only the second report of an rrn-plasmid in the bacterial domain [48].


https://www.ncbi.nlm.nih.gov/nuccore/AP018198
```
LOCUS       AP018198               16296 bp    DNA     circular BCT 15-DEC-2022
DEFINITION  Scytonema sp. HK-05 plasmid plasmid4 DNA, complete genome.

     rRNA            complement(10947..13764)
                     /locus_tag="SAMD00079811_82740"
                     /product="23S ribosomal RNA"

     rRNA            complement(14188..15678)
                     /locus_tag="SAMD00079811_82750"
                     /product="16S ribosomal RNA"
```

### 2017-03-14
https://pubmed.ncbi.nlm.nih.gov/28361695/
Comparative Study BMC Genomics
. 2017 Mar 14;18(Suppl 2):114. doi: 10.1186/s12864-017-3501-4.
SILVA, RDP, Greengenes, NCBI and OTT - how do these taxonomies compare?
Monika BalvoÄiÅ«tÄ— 1, Daniel H Huson 2
https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3501-4
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5374703/

### 2015-07-27
https://colil.dbcls.jp/browse/papers/26220935/1/100/year
https://pubmed.ncbi.nlm.nih.gov/26220935/
Genome Biol Evol
. 2015 Jul 27;7(8):2310-20. doi: 10.1093/gbe/evv143.
Rare Events of Intragenus and Intraspecies Horizontal Transfer of the 16S rRNA Gene
Ren-Mao Tian 1, Lin Cai 1, Wei-Peng Zhang 1, Hui-Luo Cao 1, Pei-Yuan Qian 2
https://academic.oup.com/gbe/article/7/8/2310/558055?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4558861/
The 2,143 finished prokaryotic genome sequences (including both chromosomes and plasmids) 

### 2013-01-01
https://pubmed.ncbi.nlm.nih.gov/23734299/
Mob Genet Elements
. 2013 Jan 1;3(1):e24210. doi: 10.4161/mge.24210.
Revisiting bacterial phylogeny: Natural and experimental evidence for horizontal gene transfer of 16S rRNA
Kei Kitahara 1, Kentaro Miyazaki
https://www.tandfonline.com/doi/full/10.4161/mge.24210
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3661144/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3661144/#sec-a.f.ftitle
Evidence of HGT of 16S rRNA in Nature: Counterexamples to the Complexity Hypothesis

In E. coli, differences in 16S rRNA sequences among seven rrn operons (rrnAâ€“H) are less than 1%. Interestingly, several isolated species of prokaryotes contains rrn operons that show significant diversity. For example, the thermophilic bacterium Thermoanaerobacter tengcongensis contains four rrn operons, and the 16S rRNA sequence of rrnC shows only 88.4% homology with other rrn operons (rrnA, rrnB and rrnD).13 The 16S rRNA gene of rrnC is not a pseudogene because it correctly forms the conserved secondary structure of 16S rRNA. The sequence is more closely related (95%) to that of other Thermoanaerobacter species (T. subterraneus SL9 and T. keratinophilus 2KXI), suggesting that this gene was transferred into the ancestor of T. tengcongensis and became fixed as a functional gene in this organism over evolution.13 Similar interoperonic diversity is observed in Desulfotomaculum kuznestovii (91.7% homology), Thermobispora bispora R51 (93.6%), Thermonospora chromogena (94%) and Halobacterium marismortui (95%).13 These examples can be regarded as snapshots of ongoing HGT of 16S rRNA in nature and are thought to be clear counterexamples of the complexity hypothesis. Chimeric 16S rRNA genes have also been reported between two distantly related species24 or closely related lineages,25-27 suggesting that partial fragments of the 16S rRNA genes are also amenable to HGT.

It remains unclear how HGT of 16S rRNA is driven in the environment. 16S rRNA genes may be propagated by transduction because, according to a previous report, this gene can be incorporated into a broad-host-range, generalized transducing phage (SN-T) at a frequency of 1 Ã— 109 transductants/PFU.28

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3661144/#sec-a.f.gtitle
Through DNA sequencing, we found that most 16S rRNA genes were derived from Gammaproteobacteria, but some were derived from Betaproteobacteria with 80.9% identity to the E. coli sequence (note that the phylogenetic class of E. coli is Gammaproteobacteria; Fig. 3).34
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3661144/figure/F3/?report=objectonly
Figure 3. Neighbor-joining phylogenetic tree of 16S rRNA genes. Environmental 16S rRNA genes that were functional in E. coli are shown as the clone ID (A01â€“H03) with their closest relatives (nomenclature). Several other relevant strains are also shown.

### 2016

https://www.ncbi.nlm.nih.gov/pubmed/27617693
Nat Microbiol. 2016 Sep 12;1(11):16160. doi: 10.1038/nmicrobiol.2016.160.
Exploiting rRNA operon copy number to investigate bacterial reproductive strategies.
Roller BR1,2, Stoddard SF1, Schmidt TM1,3,4.

### 2014

### 2014-01-14
https://pubmed.ncbi.nlm.nih.gov/24572375/
Review RNA Biol
. 2014;11(3):176-85. doi: 10.4161/rna.27505. Epub 2014 Jan 14.
Molecular phylogenetics before sequences: oligonucleotide catalogs as k-mer spectra
Mark A Ragan 1, Guillaume Bernard 1, Cheong Xin Chan 1
https://www.tandfonline.com/doi/full/10.4161/rna.27505
https://pmc.ncbi.nlm.nih.gov/articles/PMC4008546/
Table 1. All 16S ribosomal rRNA sequences used in this study, their GenBank accession numbers, and their inclusion in our re-analysis of rRNAs from (A) three kingdoms74 and (B) proteobacteria (Fig. 4 of ref. 40). For proteobacteria in our analysis B, we identify class (Î±, Î², Î³, or Î”-proteobacteria).
Source organism	GenBank accession	Analysis

### 2013

https://www.ncbi.nlm.nih.gov/pubmed/23872556
Appl Environ Microbiol. 2013 Oct;79(19):5962-9. doi: 10.1128/AEM.01282-13. Epub 2013 Jul 19.
Intragenomic heterogeneity of 16S rRNA genes causes overestimation of prokaryotic diversity.
Sun DL1, Jiang X, Wu QL, Zhou NY.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3811346/
The pseudogene previously reported in Borrelia afzelii, with a high difference of 20.38% (6), was not included in this study, as it is not predicted to be a 16S rRNA gene by RNAmmer.


### 2012

http://d.hatena.ne.jp/hoxo_m/20121031/p1
16S rRNAè§£æã®ã‚¯ã‚ªãƒªãƒ†ã‚£ãƒ•ã‚£ãƒ«ã‚¿ãƒªãƒ³ã‚°ã«ã¤ã„ã¦ - ã»ããç¬‘ã‚€
Sliding Window ãƒ•ã‚£ãƒ«ã‚¿

### 2011

### 2011-05-27
https://pubmed.ncbi.nlm.nih.gov/21619604/
BMC Evol Biol
. 2011 May 27:11:146. doi: 10.1186/1471-2148-11-146.
Potential pitfalls of modelling ribosomal RNA data in phylogenetic tree reconstruction: evidence from case studies in the Metazoa
Harald O Letsch 1, Karl M Kjer
https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-11-146
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3123606/

### 2011-02-20
review
https://www.ncbi.nlm.nih.gov/pubmed/20223646
Microbiol Res. 2011 Feb 20;166(2):99-110. doi: 10.1016/j.micres.2010.02.003. Epub 2010 Mar 11.
Microbial phylogeny and diversity: small subunit ribosomal RNA sequence analysis and beyond.
Rajendhran J1, Gunasekaran P.

### 2010

https://www.ncbi.nlm.nih.gov/pubmed/20418441
Appl Environ Microbiol. 2010 Jun;76(12):3886-97. doi: 10.1128/AEM.02953-09. Epub 2010 Apr 23.
Diversity of 16S rRNA genes within individual prokaryotic genomes.
Pei AY1, Oberdorf WE, Nossa CW, Agarwal A, Chokshi P, Gerz EA, Jin Z, Lee P, Yang L, Poles M, Brown SM, Sotero S, Desantis T, Brodie E, Nelson K, Pei Z.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2893482/
, while the diversified gene in B. afzelii (20.38%) appears to be a pseudogene.

RESULTS

Diversity of 16S rRNA genes.
, and Borrelia afzelii (20.38%) (15, 19, 20, 22, 39, 46, 55). The sequence diversity observed in these species appears to follow five patterns: intervening sequences, regional diversity, random diversity, pseudogene, and gene truncation/partial rRNA operon.

TABLE 2.
Species with significant diversity among paralogous 16S rRNA genes

FIG. 2.
Homologs of the 28- and 31-bp inserts in T. tengcongensis rrnC 16S rRNA genes also are present in 16S rRNA genes of T. kivui and T. siderophilus. The 24- and 28-bp inserts were separated by a 6-nucleotide conserved sequence shown in red.

Species with nonfunctional rRNA gene.
Borrelia afzelii contains two 16S rRNA genes found on two separate chromosomes that differ by 20.38%. The lengths of B. afzelii rrnA and rrnB are 1,509 nt and 1,537 nt, respectively, when compared to E. coli as a template. While rrnB is closely related to 16S rRNA genes in other species of Borrelia, rrnA is significantly different. Alignment of the two genes results in a 1,570-nt overlap, of which 320 bp are mismatched (225 substitutions and 95 indels). These mismatches do not follow any type of aforementioned patterns and are distributed throughout the alignment evenly. This level of random diversity may have occurred due to the escape from constraint in a nonfunctional 16S rRNA pseudogene. There has not been any experiment designed to examine the expression of rrnA of B. afzelii. It does not appear to be horizontally transferred into B afzelii from other species, as it is closer to rrnB of B. afzelii than to a species in any other genus.

DISCUSSION

Of the two 16S rRNA genes in B. afzelii, the pseudogene has a much lower GC content (38.1%) than the functional copy (46.5%). It appears the random mutations in the pseudogene have been bringing its GC content toward the baseline for the whole genome (28%).

### 2009-07-01

Published: 01 July 2009
https://academic.oup.com/jcb/article/29/3/400/2548067
16S and 28S rDNA Sequences in Phylogenetic Analyses of Freshwater Prawns (Macrobrachium Bate, 1868) from Taiwan | Journal of Crustacean Biology | Oxford Academic

Table 1
The sources of 16S rDNA and 28S rDNA sequences of freshwater prawns (Macrobrachium) used in this study (NCBI accession numbers: EU from this study, DQ from GenBank).

saturation

### 2003

### 2003-10-20
20-Oct-2003
https://repository.kulib.kyoto-u.ac.jp/dspace/handle/2433/97613
æœ€å°¤æ³•ã«ã‚ˆã‚‹ç”Ÿç‰©ã®ç³»çµ±é–¢ä¿‚æ¨å®šã«ã¤ã„ã¦(æ•°å­¦è€…ã®ãŸã‚ã®åˆ†å­ç”Ÿç‰©å­¦å…¥é–€,ç ”ç©¶ä¼šå ±å‘Š)
åŠ è—¤, å’Œè²´
https://repository.kulib.kyoto-u.ac.jp/dspace/bitstream/2433/97613/1/KJ00004705532.pdf
ï½¡ã“ã“ã§ã¯ï½¤1970å¹´ä»£ã«Wooseã‚‰ã«ã‚ˆã£ã¦è¡Œã‚ã‚ŒãŸï½¤SSUrRNA(ribosomalRNAã®smallsub-unit)ã‚’ä½¿ã£ãŸçµæœã‚’ç´¹ä»‹ã™ã‚‹ï½¡rRNAã«ã¯éºä¼å­é‡è¤‡ãŒã»ã¨ã‚“ã©ãªãï½¤ç”Ÿç‰©ã®ç³»çµ±é–¢ä¿‚ã‚’æ¨å®šã™ã‚‹ä¸Šã§éå¸¸ã«éƒ½åˆã®ã‚ˆã„åˆ†å­ã¨è¨€ãˆã‚‹(

SSUrRNAã‚’ç”¨ã„ãŸç³»çµ±æ¨å®šã®å•é¡Œç‚¹ã¨ã—ã¦ï½¤ï½¢é€²åŒ–é€Ÿåº¦ãŒä»–ç¨®ã«æ¯”ã¹æ—©ã„ç¨®ï½£ï½¢GCå«é‡3ã®é«˜ã„ç¨®ï½£ã®ç³»çµ±çš„ä½ç½®ã®æ¨å®šã‚’é–“é•ãˆã‚„ã™ã„ã“ã¨ãŒæŒ™ã’ã‚‰ã‚Œã‚‹ï½¡

å…ˆã®ä¾‹ã«ãŠã„ã¦ï½¤SSUrRNAã‚’ç”¨ã„ãŸç³»çµ±æ¨¹ã§ã¯çœŸæ ¸ç”Ÿç‰©ã«è‡³ã‚‹æã¯ï½¤çœŸæ­£ç´°èŒ/å¤ç´°èŒã«æ¯”ã¹ã¦æ˜ã‚‰ã‹ã«é•·ããªã‚‹ï½¡ã“ã®ã“ã¨ã¯ï½¤SSUrRNAã‚’ç”¨ã„ãŸæ¨å®šãŒèª¤ã£ãŸçµè«–ã‚’å°ã„ã¦ã„ã‚‹å¯èƒ½æ€§ã‚’ç¤ºå”†ã™ã‚‹ï½¡

å›³ 3:ãƒ‘ãƒ©ãƒ­ã‚¬ã‚¹ãªéºä¼å­ã‚’ç”¨ã„ãŸãƒ«ãƒ¼ ãƒˆã®ç‰¹å®š

ã€€ä¾‹ãˆã°ï½¤æœ€å°¤æ³•ã«ã‚ˆã£ã¦ç¥–å…ˆå‹ã‚’æ¨å®šã™ã‚‹ã¨å…±é€šç¥–å…ˆã®SSUrRNAã®GCå«é‡ã¯ä½ã„ã¨çµè«–ã•ã‚Œã‚‹5ï½¡ã¾ãŸï½¤ãã®è«–ã¸ã®åè«–ã¨ã—ã¦æœ€ç¯€ç´„æ³•ã«ã‚ˆã‚‹ã¨ç¥–å…ˆå‹ã®rRNAã®GCå«é‡ã¯é«˜ã„ã¨æ¨å®šã•ã‚Œã‚‹ã¨ã®ä¸»å¼µ6 ã‚‚ã‚ã‚Šï½¤æœªã è§£æ±ºã«ã¯è‡³ã£ã¦ã„ãªã„ï½¡

5Galtier(1999) 
6DiGiulioTheunwersalancestorlivedinathermophilicorhyperthermophilicenvironment.(2000)

https://colil.dbcls.jp/browse/papers/9880254/1/100/year
A nonhyperthermophilic common ancestor to extant life forms.
Galtier N, Tourasse N, Gouy M Science. 1999 Jan 8;283(5399):220-1. PMID:9880254
    is cited by the following papers.
  åˆè¨ˆ: 42, 
https://pubmed.ncbi.nlm.nih.gov/9880254/
Science
. 1999 Jan 8;283(5399):220-1. doi: 10.1126/science.283.5399.220.
A nonhyperthermophilic common ancestor to extant life forms
N Galtier 1, N Tourasse, M Gouy

https://fire.biol.wwu.edu//cmoyer/zztemp_fire/biol345_S99/cold.pdf

Substitution rates are highly variable among sites in rRNA molecules as a consequence of unequal selective constraints (6).

Fig. 1. Maximum likelihood tree reconstructed from 40 LSU rRNA sequences (17). 

Fig. 2. Correlation between optimal growth temperature and rRNA G+C content in prokaryotes.

The rRNA ancestral G+C contents as estimated above are close to the average G+C contents in the compared sequences (LSU, 53.2%; SSU, 55.3%).

### 1990
https://pubmed.ncbi.nlm.nih.gov/2112744/
Proc Natl Acad Sci U S A
. 1990 Jun;87(12):4576-9. doi: 10.1073/pnas.87.12.4576.
Towards a natural system of organisms: proposal for the domains Archaea, Bacteria, and Eucarya
C R Woese 1, O Kandler, M L Wheelis
https://pmc.ncbi.nlm.nih.gov/articles/PMC54159/
https://pmc.ncbi.nlm.nih.gov/articles/instance/54159/pdf/pnas01037-0173.pdf

ä»¥ä¸‹ã®æ–‡ç« ã¯ã€rRNA ã«é–¢é€£ã™ã‚‹éƒ¨åˆ†ã‚’æŠœç²‹ã—ã€é©åˆ‡ã«ã‚¹ãƒšãƒ¼ã‚¹ã‚’å…¥ã‚Œã¦å¯èª­æ€§ã‚’å‘ä¸Šã•ã›ãŸã‚‚ã®ã§ã™ï¼š

It is only on the molecular level that we see the living world divide into three distinct primary groups. For every well-characterized molecular system, there exists a characteristic eubacterial, archaebacterial, and eukaryotic version, which all members of each group share. Ribosomal RNAs provide an excellent example (in part because they have been so thoroughly studied). 

One structural feature in the small subunit rRNA by which the eubacteria can be distinguished from archaebacteria and eukaryotes is the hairpin loop lying between positions 500 and 545 (18), which has a side bulge protruding from the stalk of the structure. In all eubacterial cases (over 400 known), the side bulge comprises six nucleotides (of a characteristic composition), and it protrudes from the "upstream" strand of the stalk between the fifth and sixth base pair. In both archaebacteria and eukaryotes, however, the corresponding bulge comprises seven nucleotides (of a different characteristic composition), and it protrudes from the stalk between the sixth and seventh pair (18, 19). 

The small subunit rRNA of eukaryotes, on the other hand, is readily identified by the region between positions 585 and 655 (E. coli numbering), because both prokaryotic groups exhibit a common characteristic structure here that is never seen in eukaryotes (18, 19). Finally, archaebacterial 16S rRNAs are readily identified by the unique structure they show in the region between positions 180 and 197 or that between positions 405 and 498 (18, 19). 

Many other examples of group-invariant rRNA characteristics exist; see refs. 2, 18, and 19. [The reader wishing to gain a broader and more detailed appreciation for the molecular definition of the three groups can consult refs. 2, 20, and 21 and the proceedings of the most recent conference on archaebacteria (2).]

Fig. 1 is a universal phylogenetic tree, showing the relationships among the primary groups. The root of the tree is seen to separate the eubacteria from the other two primary groups, making the archaebacteria and eukaryotes specific (but distant) relatives.

A relationship between archaebacteria and eukaryotes is not overly surprising, for with few exceptions (the rRNA being one), the archaebacterial versions of molecules resemble their eukaryotic homologs more than their eubacterial ones (24, 29, 30).

FIG. 1. Universal phylogenetic tree in rooted form, showing the three domains. Branching order and branch lengths are based upon rRNA sequence comparisons (and have been taken from figure 4 of ref. 2).

### 1987
https://pubmed.ncbi.nlm.nih.gov/2439888/
Review Microbiol Rev. 1987 Jun;51(2):221-71. doi: 10.1128/mr.51.2.221-271.1987.
Bacterial evolution
C R Woese
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC373105/

### 1980
https://pubmed.ncbi.nlm.nih.gov/6771870/
Science. 1980 Jul 25;209(4455):457-63. doi: 10.1126/science.6771870.
The phylogeny of prokaryotes
G E Fox, E Stackebrandt, R B Hespell, J Gibson, J Maniloff, T A Dyer, R S Wolfe, W E Balch, R S Tanner, L J Magrum, L B Zablen, R Blakemore, R Gupta, L Bonen, B J Lewis, D A Stahl, K R Luehrsen, K N Chen, C R Woese

----------
## people

----------

### anda
æŒ‰ç”°ã€€ç‘æµï¼ˆMizue ANDAï¼‰

2019/03/01 
https://www.jstage.jst.go.jp/browse/jsmeja/list/-char/ja
rRNAéºä¼å­ã‚’ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã ã‘ã§ç¶­æŒã™ã‚‹ãƒã‚¯ãƒ†ãƒªã‚¢ã®ã‚²ãƒãƒ é€²åŒ–ã‚’è€ƒãˆã‚‹
https://www.jstage.jst.go.jp/article/jsmeja/34/1/34_5/_pdf/-char/ja

2017-05-20
https://katosei.jsbba.or.jp/view_html.php?aid=800
https://katosei.jsbba.or.jp/download_pdf.php?aid=800
https://www.jstage.jst.go.jp/article/kagakutoseibutsu/55/6/55_378/_pdf
æŸ“è‰²ä½“ã«ãƒªãƒœã‚½ãƒ¼ãƒ RNAéºä¼å­ã®ç„¡ã„ç’°å¢ƒç´°èŒã®ç™ºè¦‹
äºˆæƒ³å¤–ãªã‚²ãƒãƒ æ§‹é€ ã¨é€²åŒ–
å”¯ä¸€ã®rrnã¯9.4â€‰kbã®ãƒ¬ãƒ—ãƒªã‚³ãƒ³ï¼ˆpAU20rrnï¼‰ã«å­˜åœ¨ã—ã¦ã„ãŸï¼ˆå›³1Aï¼‰ï¼

https://schaechter.asmblog.org/schaechter/2015/11/your-operating-system-on-a-thumbdrive-a-snippet.html
Small Things Considered: Your Operating System on a Thumbdrive â€“ A Snippet

24 11æœˆ 2015
http://www.microbial-ecology.jp/?p=7070
æ—¥æœ¬å¾®ç”Ÿç‰©ç”Ÿæ…‹å­¦ä¼š Â» æŸ“è‰²ä½“ä¸Šã‹ã‚‰ãƒªãƒœã‚½ãƒ¼ãƒ RNAéºä¼å­ãŒæ¶ˆãˆãŸç´°èŒã‚’ç™ºè¦‹ ï½ã‚²ãƒãƒ ã®å¸¸è­˜ã‚’è¦†ã™ï½ (PNAS 112)

2015å¹´11æœˆ4æ—¥
https://www.tohoku.ac.jp/japanese/2015/11/press20151104-01.html
æŸ“è‰²ä½“ä¸Šã‹ã‚‰ãƒªãƒœã‚½ãƒ¼ãƒ RNAéºä¼å­ãŒæ¶ˆãˆãŸç´°èŒã‚’ç™ºè¦‹ ï½ã‚²ãƒãƒ ã®å¸¸è­˜ã‚’è¦†ã™ï½

### miyazaki-kentaro
å®®å´ å¥å¤ªéƒ

https://staff.aist.go.jp/miyazaki-kentaro/group/
ãƒˆãƒƒãƒ—|æ±å¤§æ–°é ˜åŸŸãƒ¡ãƒ‡ã‚£ã‚«ãƒ«æƒ…å ±ç”Ÿå‘½-ç”£ç·ç ”å®®å´ç ”ç©¶å®¤

2017/08/30
https://www.aist.go.jp/aist_j/press_release/pr2017/pr20170830/pr20170830.html
ç”£ç·ç ”ï¼šé€²åŒ–ç³»çµ±åˆ†é¡ã®æŒ‡æ¨™ã¨ãªã‚‹16S rRNAéºä¼å­ã®é€²åŒ–çš„ãªä¸­ç«‹æ€§ã‚’å®Ÿé¨“çš„ã«è¨¼æ˜
ï¼æŒ‡æ¨™ã¨ã—ã¦ã®é©æ€§ã‚’æ¤œè¨ã™ã‚‹ãŸã‚ã®é‡è¦ãªäº‹å®Ÿã‚‚åŒæ™‚ã«ç™ºè¦‹ï¼

2015
https://seikagaku.jbsoc.or.jp/10.14952/SEIKAGAKU.2015.870475/data/index.html
16Sãƒªãƒœã‚½ãƒ¼ãƒ RNAã®æ°´å¹³ä¼æ’­å®Ÿé¨“ã‹ã‚‰ã¿ãˆã¦ãã‚‹ãƒªãƒœã‚½ãƒ¼ãƒ ã®å¯å¡‘æ€§
- ã‚²ãƒãƒ å†…ã«é…åˆ—ç›¸åŒæ€§ã®ä½ã„2ç¨®ã®16S rRNAéºä¼å­ã‚’ä½µã›æŒã¤ã‚‚ã®ã‚„ï¼Œã‚­ãƒ¡ãƒ©æ§˜16S rRNAéºä¼å­ã‚’æŒã¤ã‚‚ã®ãªã©ãŒçŸ¥ã‚‰ã‚Œã¦ã„ã‚‹ã®ã§ã‚ã‚‹ï¼
- ãŸã¨ãˆã°ï¼Œä¸€ã¤ã®æŸ“è‰²ä½“ä¸Šã«äº’ã„ã«ç›¸åŒæ€§ã®ä½ã„è¤‡æ•°ã®16S rRNAéºä¼å­ã‚’ä½µã›æŒã¤å¾®ç”Ÿç‰©ã‚„ï¼Œè¤‡æ•°ã®ç´°èŒç”±æ¥ã®16S rRNAéºä¼å­ãŒãƒ¢ã‚¶ã‚¤ã‚¯çŠ¶ã«ãªã£ã¦ã„ã‚‹ã‚‚ã®ãŒè¦‹ã¤ã‹ã£ã¦ã„ã‚‹3â€“7)ï¼

https://www.jstage.jst.go.jp/article/kagakutoseibutsu/52/2/52_70/_article/-char/ja/
16S rRNAéºä¼å­ã®ã€Œæ°´å¹³ä¼æ’­ã€
ç•°ç¨®16S rRNAã«ã‚ˆã‚‹éºä¼çš„ç›¸è£œ
2014 å¹´ 52 å·» 2 å· p. 70-72
https://katosei.jsbba.or.jp/download_pdf.php?aid=19

2014
https://ci.nii.ac.jp/naid/110009872169
Ribosomal RNAã®äººå·¥æ°´å¹³ä¼æ’­ã«ã‚ˆã‚‹å¤§è…¸èŒå®¿ä¸»ãƒ‡ã‚¶ã‚¤ãƒ³ (ç‰¹é›† ç”£æ¥­å¾®ç”Ÿç‰©ã®åŠ›ã‚’æ·±åŒ–ã•ã›ã‚‹æ—¥æœ¬ã®æŠ€~è‚²ç¨®ã®æœ€å‰ç·š~æˆ‘ãŒå›½ç‹¬è‡ªã®èµ·ç‚¹ãƒ»æŠ€è¡“ã‹ã‚‰)
ç”Ÿç‰©å·¥å­¦ä¼šèªŒ 92(11), 607-611, 2014
https://www.sbj.or.jp/wp-content/uploads/file/sbj/9211/9211_tokushu_5.pdf

2014
ç·èª¬
https://www.jseb.jp/wordpress/wp-content/uploads/14-01-003.pdf
ãƒªãƒœã‚½ãƒ¼ãƒ æ”¹å¤‰ã«ã‚ˆã‚‹å¤§è…¸èŒå®¿ä¸»ãƒ‡ã‚¶ã‚¤ãƒ³
2.3 è‡ªç„¶ç•Œã¦ã‚™è¦‹ã‚‰ã‚Œã‚‹ 16S rRNA éºä¼å­ã®ã‚±ã‚™ãƒãƒ å†…å¤š æ§˜æ€§ã¨ãƒªãƒ›ã‚™ã‚½ãƒ¼ãƒ ã®å¯å¡‘æ€§
ãƒ­ãƒ³)ã¯è¤‡æ•°å€‹å­˜åœ¨ã—ã¦ã„ã‚‹ã€‚ç”Ÿç‰©ç¨®ã«ã‚ˆã‚Šæ§˜ã€…ã¦ã‚™ã‚ã‚‹ ã‹ã‚™ï¼Œä¾‹ãˆã¯ã‚™å¤§è…¸èŒã¦ã‚™ã¯ 7 å€‹ï¼Œæ¯è‰èŒã¦ã‚™ã¯ 13 å€‹ã® rRNA ã‚ªãƒ˜ã‚šãƒ­ãƒ³ã‹ã‚™ã‚³ãƒ¼ãƒˆã‚™ã•ã‚Œã¦ã„ã‚‹ã€‚èˆˆå‘³æ·±ã„ã“ã¨ã«ï¼Œã“ã‚Œã‚‰ ã¯å®Œå…¨ã«åŒã—ã‚™ã¨ã„ã†ã‚ã‘ã¦ã‚™ã¯ãªãï¼Œã‚ªãƒ˜ã‚šãƒ­ãƒ³é–“ã«æ•°å¡©åŸº ã®å¤šå‹ã‹ã‚™è¦‹ã‚‰ã‚Œã‚‹ã“ã¨ã‹ã‚™å¤šã„ã€‚ã€Œä¾‹å¤–ã€ã¨ã¯ï¼Œã‚±ã‚™ãƒãƒ ä¸­ ã«è¤‡æ•°å­˜åœ¨ã™ã‚‹ 16S rRNA ã®å·®ã‹ã‚™ã€Œæ•°å¡©åŸºã€ã¦ã‚™ã¯ãªãï¼Œ ã‚ãŸã‹ã‚‚ç•°ç¨®ç”Ÿç‰©ã® 16S rRNA éºä¼å­ã‹ã‚™ä¸¸ã“ã‚™ã¨é£›ã²ã‚™è¾¼ã‚“ ã¦ã‚™ããŸã‹ã®ã‚ˆã†ã«ï¼Œç›¸åŒæ€§ã®ä½ã„ 16S rRNA éºä¼å­ã‹ã‚™å…± å­˜ã—ã¦ã„ã‚‹äº‹ä¾‹ã‚’æŒ‡ã™ã€‚ã¨ã‚™ã¡ã‚‰ã‹ä¸€æ–¹ã‹ã‚™å½éºä¼å­ã¨ã„ã† ã‚ã‘ã¦ã‚™ã¯ãªãï¼Œä¸¡æ–¹å…±ã«æ©Ÿèƒ½ã—ã¦ã„ã‚‹ã“ã¨ã‚‚ç¢ºã‹ã‚ã‚‰ã‚Œ ã¦ã„ã‚‹ 4)ã€‚éºä¼å­ä¸¸ã“ã‚™ã¨ã¦ã‚™ã¯ãªãï¼Œéƒ¨åˆ†çš„ãªæ–­ç‰‡ã¨ã—ã¦ é£›ã²ã‚™è¾¼ã‚“ã¦ã‚™ãã¦ã„ã‚‹ã‚ˆã†ã«ã¿ãˆã‚‹äº‹ä¾‹ã‚‚è¦‹ã¤ã‹ã£ã¦ã„ ã‚‹ã€‚ç›¸åŒéºä¼å­é–“ã¦ã‚™çµ„æ›ãˆã‹ã‚™èµ·ããŸã‹ã®ã‚ˆã†ãªã€Œã‚­ãƒ¡ãƒ© å‹ã€ã®16S rRNAã¦ã‚™ã‚ã‚‹5â€“8)ã€‚

4) Acinas, S.G., L.A. Marcelino, V. Klepac-Ceraj, and M.F. Polz.
2004. Divergence and redundancy of 16S rRNA sequences in genomes with multiple rrn operons. J. Bacteriol. 186: 2629â€“ 2635.
5) Wang, Y. and Z. Zhang. 2000. Comparative sequence analyses reveal frequent occurrence of short segments containing an abnormally high number of non-random base variations in bacterial rRNA genes. Microbiology 146: 2845â€“2854.
6) Schouls, L.M., C.S. Schot, and J.A. Jacobs. 2003. Horizontal transfer of segments of the 16S rRNA genes between species of the Streptococcus anginosus group. J. Bacteriol. 185: 7241â€“7246.
7) Miller, S.R., S. Augustine, T.L. Olson, R.E. Blankenship, J. Selker, and A.M. Wood. 2005. Discovery of a free-living chlo- rophyll d-producing cyanobacterium with a hybrid proteobac- terial/cyanobacterial small-subunit rRNA gene. Proc. Natl. Acad. Sci. USA 102: 850â€“855.
8) Eardly, B.D., S.M. Nour, P. van Berkum, and R.K. Selander.
2005. Rhizobial 16S rRNA and dnaK genes: mosaicism and the uncertain phylogenetic placement of Rhizobium galegae. Appl. Environ. Microbiol. 71: 1328â€“1335.


2012/10/30
https://www.aist.go.jp/aist_j/press_release/pr2012/pr20121030/pr20121030.html
ç”£ç·ç ”ï¼šç”Ÿç‰©ç¨®ã‚’è¶ŠãˆãŸ16S rRNAéºä¼å­ã®æ©Ÿèƒ½ç›¸è£œæ€§ã‚’ç¢ºèª
ï¼ãƒã‚¯ãƒ†ãƒªã‚¢ã®ç³»çµ±åˆ†é¡å­¦ã®æ ¹æœ¬ã«ç–‘å•ã‚’æŠ•ã’ã‹ã‘ã‚‹ï¼

----------



