# mgsa.metagenomics
**ãƒ¡ã‚¿ã‚²ãƒãƒ **

## Table of Contents
- [codon](https://github.com/haruosuz/codon/blob/master/README.md#metagenome)
- [](#)
- [unclassified](#unclassified)
- [updates](#updates)
[2019](#2019)
[2018](#2018)
[2017](#2017)
- [review](#review)
- [Metagenomic Assembly]
- [book](#book)
- [video](#video)
- [CAMI_challenge](#cami) Critical Assessment of Metagenome Interpretation
- [batch effect](#batch-effect)
- [meta-analysis](#meta-analysis)
  - [qiita](#qiita)
- [metabolic inference](#metabolic-inference)
  - [PICRUSt](#picrust)
  - [Tax4Fun](#tax4fun)
  - [themetagenomics](#themetagenomics) removed from CRAN
  - [paprica](#paprica)
  - [Piphillin](#piphillin)
- [HGT](#hgt)
- [MGnify](#mgnify) EBI metagenomics
- [kazumaxneo](#kazumaxneo)
- [](#)
- [benchmark](#benchmark)
- [](#)
- [kaiju](#kaiju)
- [](#)
- [web-tool](#web-tool)
  - [MANTA](#manta) æ°´å£è³¢å¸ç ”ç©¶å®¤
  - [pavian](#pavian)
  - [taxonomer](#taxonomer)
- [MEDUSA](#medusa)
- [recentrifuge](#recentrifuge)
- [centrifuge](#centrifuge)
- [unifrac](#unifrac)
- [LEfSe](#lefse)
- [diversity index](#diversity-index) å¤šæ§˜æ€§æŒ‡æ•°
- [EzBioCloud](#ezbiocloud)
- [PhyloSift](#PhyloSift)
- [](#)
- [mothur-qiime](#mothur-qiime)
- [16S](#16s)
- [amplicon-shotgun](#amplicon-shotgun)
- [rRNA](#rrna)
- [Hi-C](#hi-c)
- [](#)
- [people](#people)
[duvallet](#duvallet) Claire Duvallet
[Mori](#mori)
[Rocha](#rocha)
- [bromberglab](#bromberglab)
[mi-faser](#mi-faser)
[rebean](#rebean)
- [CoverM](#Coverm)
- [](#)
- [MAG](#mag) Metagenome Assembled Genome
  - [MAGNETO](#magneto)
  - [MDMcleaner](#mdmcleaner)
  - [MetaWRAP](#metawrap)
  - [](#)
- [](#)

----------

----------
### 

https://github.com/haruosuz/microbe/blob/master/references/microbe.urban.md

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238498/
A global metagenomic map of urban microbiomes and antimicrobial resistance

Applying these thresholds, we identified 1,145 microbial species (Figure 1B), as defined by the NCBI annotation in KrakenUniq, that make up the sub-core urban microbiome with 31 species in the true core microbiome (Figure 1A).

----------
### SingleM

https://github.com/wwood/singlem

https://wwood.github.io/singlem/tools/pipe

Please use raw metagenomic reads, not quality trimmed reads, if possible. Quality trimming with e.g. Trimmomatic reads can make them too short for SingleM to use, particularly if they are trimmed to be shorter than 100 bp. Adapter trimming is unlikely to be detrimental, but is not needed.

https://pubmed.ncbi.nlm.nih.gov/40670710/
Nat Biotechnol
. 2025 Jul 16. doi: 10.1038/s41587-025-02738-1. Online ahead of print.
Comprehensive taxonomic identification of microbial species in metagenomic data using SingleM and Sandpiper
Ben J Woodcroft 1, Samuel T N Aroney 2, Rossen Zhao 2, Mitchell Cunningham 3, Joshua A M Mitchell 2, Rizky Nurdiansyah 2, Linda Blackall 3, Gene W Tyson 2
https://www.nature.com/articles/s41587-025-02738-1

https://kazumaxneo.hatenablog.com/entry/2022/05/04/131449
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®OTUè§£æã‚’è¡Œã† singleM - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

å®Ÿè¡Œæ–¹æ³•

ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚·ãƒ³ã‚°ãƒªãƒ¼ãƒ‰ã‚’æŒ‡å®šã™ã‚‹ã€‚QualityãƒˆãƒªãƒŸãƒ³ã‚°ã•ã‚Œã¦çŸ­ããªã£ãŸãƒªãƒ¼ãƒ‰ã§ã¯ãªãç”Ÿã®ãƒªãƒ¼ãƒ‰ã‚’ä½¿ã†ã“ã¨ãŒæ¨å¥¨ã•ã‚Œã¦ã„ã‚‹ã€‚

----------

### 2025-09-26
### MetaCompass
https://github.com/marbl/MetaCompass

https://pubmed.ncbi.nlm.nih.gov/41015028/
Cell Rep Methods
. 2025 Sep 26:101186. doi: 10.1016/j.crmeth.2025.101186. Online ahead of print.
Reference-guided assembly of metagenomes with MetaCompass
Tu Luan 1, Victoria P Cepeda-Espinoza 1, Bo Liu 1, Zac Bowen 2, Ujjwal Ayyangar 2, Mathieu Almeida 3, Sergey Koren 4, Todd J Treangen 5, Adam Porter 6, Mihai Pop 7
https://www.cell.com/cell-reports-methods/fulltext/S2667-2375(25)00222-X

Quality of MetaCompass assemblies of individual genomes

Metagenomics-assembled contigs (MAGs)

8:35 AM Â· Sep 27, 2025
https://x.com/CellRepMethods/status/1971720328146784643
Cell Reports Methods on X: "Reference-guided assembly of metagenomes with MetaCompass https://t.co/hLmYQEUMhM" / X

3 Mar 2024
https://arxiv.org/abs/2403.01578
[2403.01578] MetaCompass: Reference-guided Assembly of Metagenomes

https://kazumaxneo.hatenablog.com/entry/2018/05/25/200236
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ãƒªãƒ•ã‚¡ãƒ¬ãƒ³ã‚¹ã‚¬ã‚¤ãƒ‰ã‚¢ã‚»ãƒ³ãƒ–ãƒªã‚’è¡Œã† MetaCompass - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------
### kazumaxneo

http://kazumaxneo.hatenablog.com/archive/category/metagenome
metagenome ã‚«ãƒ†ã‚´ãƒªãƒ¼ã®è¨˜äº‹ä¸€è¦§ - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

http://kazumaxneo.hatenablog.com/entry/2017/05/22/164017
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ„ãƒ¼ãƒ« - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

http://kazumaxneo.hatenablog.com/entry/2019/02/15/073000
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ã‚¢ã‚»ãƒ³ãƒ–ãƒªcontig.fastaã«ç²¾åº¦ã®é«˜ã„ç³»çµ±æƒ…å ±ã‚’ã‚¢ã‚µã‚¤ãƒ³ã™ã‚‹CATã¨ã€binned.fastaã«ç²¾åº¦ã®é«˜ã„ç³»çµ±æƒ…å ±ã‚’ã‚¢ã‚µã‚¤ãƒ³ã™ã‚‹BAT - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹



----------

## MAG
Metagenome Assembled Genome (MAG)

https://www.ddbj.nig.ac.jp/ddbj/qualifiers.html
DDBJ Qualifier key
metagenome_source
å®šç¾©	Metagenome Assembled Genome (MAG) ã¨ã—ã¦å¾—ã‚‰ã‚ŒãŸé…åˆ—ã€ã™ãªã‚ã¡ã€metagenome ã‹ã‚‰å˜ä¸€ taxon ã¨ã—ã¦ assemble ã•ã‚ŒãŸå ´åˆã«ã€ãã®ç”±æ¥ã¨ãªã£ãŸ metagenome ã‚’è¨˜è¼‰ã—ã¾ã™ã€‚

----------


----------

### 2022-06-15
### MAGNETO

https://gitlab.univ-nantes.fr/bird_pipeline_registry/magneto

https://kazumaxneo.hatenablog.com/entry/2022/06/17/013547
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã®ãŸã‚ã®è‡ªå‹•åŒ–ã•ã‚ŒãŸãƒ¯ãƒ¼ã‚¯ãƒ•ãƒ­ãƒ¼ MAGNETO - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

https://pubmed.ncbi.nlm.nih.gov/35703559/
mSystems
. 2022 Aug 30;7(4):e0043222. doi: 10.1128/msystems.00432-22. Epub 2022 Jun 15.
MAGNETO: An Automated Workflow for Genome-Resolved Metagenomics
Benjamin Churcheward 1, Maxime Millet 1, Audrey BihouÃ©e 2 3, Guillaume Fertin 1, Samuel Chaffron 1 4
https://europepmc.org/article/MED/35703559
https://journals.asm.org/doi/10.1128/msystems.00432-22

11:20 PM Â· Jun 24, 2022
https://twitter.com/schaffron/status/1540338989240033282
Samuel Chaffron on Twitter: "Have you ever wondered how to group metagenomic samples for co-assembly to improve MAGs recovery? Check out our solution implemented in MAGNETO: An Automated Workflow for Genome-Resolved Metagenomics | mSystems https://t.co/zykKasJwIq" / Twitter

----------
### 2022-05-10
### MDMcleaner

https://pubmed.ncbi.nlm.nih.gov/35536293/
Nucleic Acids Res
. 2022 May 10;gkac294. doi: 10.1093/nar/gkac294. Online ahead of print.
How clear is our current view on microbial dark matter? (Re-)assessing public MAG & SAG datasets with MDMcleaner
John Vollmers 1, Sandra Wiegand 1, Florian Lenk 1, Anne-Kristin Kaster 1
https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac294/6583244

8:05 PM Â· May 20, 2022
https://twitter.com/kasterlab/status/1527606371872546818
Kaster Lab on Twitter: "Our genome assessment pipeline for microbial dark matter, MDMcleaner, is now available via Pip & Conda! Also check out the corresponding publication at NAR: https://t.co/8lqdLnNQfD" / Twitter

6:13 PM Â· May 16, 2022
https://twitter.com/BioMickWatson/status/1526128600617046022
Mick W@tson on Twitter: "Another taxonomy-based approach to MAG contamination: How clear is our current view on microbial dark matter? (Re-)assessing public MAG & SAG datasets with MDMcleaner | Nucleic Acids Research | Oxford Academic https://t.co/Fpl1iHhR4K" / Twitter

In our experience these really don't work very well especially for MAGs distant from the reference DB

And if your MAGs are not distant from the reference DB, why are you creating them>

I mean if your microbiome contains known genomes (which you can find out using containment queries), you don't need to create MAGs


----------
### 2018-09-15
### MetaWRAP

https://kazumaxneo.hatenablog.com/entry/2018/10/07/073000
è¤‡æ•°ã®biningãƒ„ãƒ¼ãƒ«ã‚’çµ±åˆã—ã€åŒ…æ‹¬çš„ãªãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã‚’è¡Œã†ãƒ‘ã‚¤ãƒ—ãƒ©ã‚¤ãƒ³ metaWRAP - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
https://github.com/bxlab/metaWRAP
https://pubmed.ncbi.nlm.nih.gov/30219103/
Microbiome
. 2018 Sep 15;6(1):158. doi: 10.1186/s40168-018-0541-1.
MetaWRAP-a flexible pipeline for genome-resolved metagenomic data analysis
Gherman V Uritskiy 1, Jocelyne DiRuggiero 2, James Taylor 3
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0541-1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138922/




2019å¹´4æœˆ25æ—¥
https://www.natureasia.com/ja-jp/nature/highlights/98543
Nature ãƒã‚¤ãƒ©ã‚¤ãƒˆï¼šãƒ’ãƒˆè…¸å†…å¾®ç”Ÿç‰©ç›¸ã®æ§‹æˆã‚’ã“ã‚Œã¾ã§ã«ãªã„åˆ†è§£èƒ½ã§æ˜ã‚‰ã‹ã«ã™ã‚‹
ã€1ä¸‡1850ã®ãƒ’ãƒˆè…¸ã®ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ã‹ã‚‰ã€9ä¸‡2143ã®ãƒ¡ã‚¿ã‚²ãƒãƒ ã‚¢ã‚»ãƒ³ãƒ–ãƒªãƒ¼ã«ã‚ˆã‚‹å†æ§‹ç¯‰ã‚²ãƒãƒ ï¼ˆMAGï¼‰ã‚’å¾—ã¦ã€1952ç¨®ã®æœªåŸ¹é¤Šã®å€™è£œç´°èŒã‚’ç‰¹å®šã—ãŸã€‚
ã€ç³»çµ±ç™ºç”Ÿå­¦çš„å¤šæ§˜æ€§ã‚’281ï¼…å¢—å¤§ã•ã›ãŸã€‚
ã€æ•°ç™¾ã®æ–°ã—ã„ç”Ÿåˆæˆéºä¼å­ã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼ã‚’ã‚³ãƒ¼ãƒ‰ã—ã¦ã„ã‚‹ã“ã¨ãŒåˆ†ã‹ã£ãŸã€‚
https://www.nature.com/articles/s41586-019-0965-1
A new genomic blueprint of the human gut microbiota | Nature
Phylogenetic diversity was quantified by the sum of branch lengths using the phytools R package59.


https://twitter.com/jgi/status/1106429999974772736
Joint Genome Inst. on Twitter: "Read "Novel insights from uncultivated genomes of the global human gut #microbiome" in @nature from @BerkeleyLab @LBNLBioSci @jgi and @GladstoneInst @UCSF @czbiohub researchers: https://t.co/5dUrX7zHzd #BioEGSB #DataScienceâ€¦ https://t.co/ExoEkSqpRp"
1:40 AM - 15 Mar 2019
![](https://pbs.twimg.com/media/D1rUODCUgAA8bDz.jpg)

https://twitter.com/TrendsMicrobiol/status/1106251713122578433
Trends Microbiol on Twitter: "Sam Minot spotlights how de novo assembly vastly expands the known microbial universe of the gut microbiome https://t.co/kyUgu6tu1Q"
1:52 PM - 14 Mar 2019
https://www.sciencedirect.com/science/article/pii/S0966842X19300587

https://www.ncbi.nlm.nih.gov/pubmed/30867587
Nature. 2019 Mar 13. doi: 10.1038/s41586-019-1058-x. [Epub ahead of print]
Novel insights from uncultivated genomes of the global human gut microbiome.
Nayfach S1,2, Shi ZJ3,4, Seshadri R5,6, Pollard KS3,4,7, Kyrpides N8,9.

2019Feb28
https://bibgraph.hpcr.jp/abst/pubmed/30816235
ãƒ¡ã‚¿ã‚²ãƒãƒ é›†åˆã‚²ãƒãƒ ã¯ãƒ­ã‚·ã‚¢ã®ã‚«ãƒ ãƒãƒ£ãƒ„ã‚«ã«ãŠã‘ã‚‹2ã¤ã®æ¸©æ³‰ãƒ—ãƒ¼ãƒ«ã®å¾®ç”Ÿç‰©å¤šæ§˜æ€§ã¸ã®æ–°ã—ã„æ´å¯Ÿã‚’æä¾›ã™ã‚‹ - Bibgraphï¼ˆãƒ“ãƒ–ã‚°ãƒ©ãƒ•ï¼‰| PubMedã‚’æ—¥æœ¬èªã§è«–æ–‡æ¤œç´¢

å†ç¾æ€§ã®ã‚ã‚‹ãƒ‘ã‚¤ãƒ—ãƒ©ã‚¤ãƒ³
https://github.com/WatsonLab/MAGpy
https://www.ncbi.nlm.nih.gov/pubmed/30418481
Bioinformatics. 2018 Nov 10. doi: 10.1093/bioinformatics/bty905. [Epub ahead of print]
MAGpy: a reproducible pipeline for the downstream analysis of metagenome-assembled genomes (MAGs).
Stewart RD1, Auffret M2, Snelling TJ3, Roehe R2, Watson M1.

æœªåŸ¹é¤ŠåŸæ ¸ç”Ÿç‰©ã®ã‚²ãƒãƒ 
https://www.ncbi.nlm.nih.gov/pubmed/30266101
Microbiome. 2018 Sep 28;6(1):173. doi: 10.1186/s40168-018-0550-0.
Genomes from uncultivated prokaryotes: a comparison of metagenome-assembled and single-amplified genomes.
Alneberg J1, Karlsson CMG2, Divne AM3, Bergin C3, Homa F3, Lindh MV2,4, Hugerth LW1,5, Ettema TJG3, Bertilsson S6, Andersson AF7, Pinhassi J8.

http://kazumaxneo.hatenablog.com/entry/2018/10/09/003000
NGSãƒ‡ãƒ¼ã‚¿ã¾ãŸã¯ã‚¢ã‚»ãƒ³ãƒ–ãƒªã‹ã‚‰ãƒã‚¯ãƒ†ãƒªã‚¢ã‚„ã‚¢ãƒ¼ã‚­ã‚¢ã®taxanomic assignmentã‚’è¡Œã„ã€ã‚²ãƒãƒ ã®noveltyãªã©ã‚’è©•ä¾¡ã™ã‚‹ MIGA - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
ã€€å¾®ç”Ÿç‰©ã‚²ãƒãƒ ã‚¢ãƒˆãƒ©ã‚¹ï¼ˆMiGAï¼‰ãƒ—ãƒ­ã‚¸ã‚§ã‚¯ãƒˆã¯ã€ANIã«åŸºã¥ã„ã¦ã‚¯ã‚¨ãƒªãƒ¼ã‚²ãƒãƒ ã¾ãŸã¯ã‚¢ã‚»ãƒ³ãƒ–ãƒªã•ã‚ŒãŸã‚³ãƒ³ãƒ†ã‚£ã‚°é…åˆ—ã®é ‘å¼·ãªåˆ†é¡å­¦çš„åˆ†é¡ã‚’æä¾›ã™ã‚‹ã€‚
ã€€ã•ã‚‰ã«ã€ã‚¯ã‚¨ãƒªãƒ¼ã‚²ãƒãƒ é…åˆ—ã¯ã€å˜é›¢ã•ã‚ŒãŸãŒåˆ†é¡ã•ã‚Œã¦ã„ãªã„ã‚²ãƒãƒ ã‚„ã€metagenome-assembled genomesï¼ˆMAGï¼‰ãŠã‚ˆã³single-cell amplified genomeï¼ˆSAGï¼‰ã‚³ãƒ¬ã‚¯ã‚·ãƒ§ãƒ³ã«å¯¾ã—ã¦æ¤œç´¢ã—ã€

http://kazumaxneo.hatenablog.com/entry/2018/05/08/212758
ã‚¯ãƒ©ã‚¹ã‚¿ã‚’è‡ªå‹•ã§æ±ºã‚ã¦ãƒ“ãƒ‹ãƒ³ã‚°ã™ã‚‹ BinSanity - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹
ã€‚Metagenome-assembled genomes ï¼ˆMAGï¼‰ã«contigã‚’åˆ†é¡ã™ã‚‹ã“ã¨ã¯ã€ãƒ¡ã‚¿ã‚²ãƒãƒ ãƒ‡ãƒ¼ã‚¿ã‚’åˆ†æã™ã‚‹éš›ã«ç›´é¢ã™ã‚‹ãƒãƒ¼ãƒ‰ãƒ«ã®1ã¤ã§ã‚ã‚‹ã€‚

https://www.ncbi.nlm.nih.gov/pubmed/29234139
Nat Microbiol. 2018 Feb;3(2):253. doi: 10.1038/s41564-017-0083-5.
Author Correction: Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life.
Parks DH1, Rinke C1, Chuvochina M1, Chaumeil PA1, Woodcroft BJ1, Evans PN1, Hugenholtz P2, Tyson GW3.

https://www.ncbi.nlm.nih.gov/pubmed/28894102
Nat Microbiol. 2017 Nov;2(11):1533-1542. doi: 10.1038/s41564-017-0012-7. Epub 2017 Sep 11.
Recovery of nearly 8,000 metagenome-assembled genomes substantially expands the tree of life.
Parks DH1, Rinke C1, Chuvochina M1, Chaumeil PA1, Woodcroft BJ1, Evans PN1, Hugenholtz P2, Tyson GW3.

2017å¹´09æœˆ12æ—¥
http://crisp-bio.blog.jp/archives/3772216.html
ãƒ¡ã‚¿ã‚²ãƒãƒ ã‹ã‚‰ã‚¢ãƒƒã‚»ãƒ³ãƒ–ãƒ«ã—ãŸã‚²ãƒãƒ ã«ã‚ˆã‚Šå¾®ç”Ÿç‰©ç³»çµ±æ¨¹30ï¼…ä»¥ä¸Šæ‹¡å¼µ

2017-03
https://repository.kulib.kyoto-u.ac.jp/dspace/bitstream/2433/227988/1/scr_2017_56.pdf
Title çµç¶æ¹–æ²–ã«ç”Ÿæ¯ã™ã‚‹ç´°èŒãƒ»ã‚¦ã‚¤ãƒ«ã‚¹ã®ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ
Author(s) å²¡å´, å‹è¼”
ç´°èŒãƒ¡ã‚¿ã‚²ãƒãƒ ã«ãŠã„ã¦ã¯ã€SPAdes ã§ã‚¢ã‚»ãƒ³ãƒ–ãƒªã•ã‚ŒãŸå¡©åŸºé…åˆ—æ–­ç‰‡ã«å¯¾ã—ã€4-mer ãŠã‚ˆã³
ã‚µãƒ³ãƒ—ãƒ«é–“ã®ã‚«ãƒãƒ¬ãƒƒã‚¸ã®å·®ç•°ã‚’ç”¨ã„ã¦ã‚¯ãƒ©ã‚¹ã‚¿ãƒªãƒ³ã‚°ã™ã‚‹æ‰‹æ³•(binning) ã‚’ç”¨ã„ãŸã€‚ã“ã‚Œã«ã‚ˆ
ã‚Šã€å„ç´°èŒç³»çµ±ã® MAG(Metagenome Assembled Genome)ã‚’æ§‹ç¯‰ã—ã€çµç¶æ¹–ã®è¡¨æ°´å±¤ãŠã‚ˆã³
æ·±æ°´å±¤ã«ç”Ÿæ¯ã™ã‚‹å„ªå çš„ãªç´°èŒç³»çµ±ã®ã‚²ãƒãƒ ã‚’ 90ï¼…ä»¥ä¸Šã® Completeness ã§å¾—ã‚‹ã“ã¨ãŒã§ã
ãŸã€‚


----------

### 2023-10-25
https://github.com/taowenmicro/EasyMicrobiomeR
2 years ago
https://pubmed.ncbi.nlm.nih.gov/37128855/
Protein Cell
. 2023 Oct 25;14(10):713-725. doi: 10.1093/procel/pwad024.
The best practice for microbiome analysis using R
Tao Wen 1 2, Guoqing Niu 2, Tong Chen 3, Qirong Shen 2, Jun Yuan 2, Yong-Xin Liu 1
https://academic.oup.com/proteincell/article/14/10/713/7147618?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10599642/
Figure 1.
(C) Commonly used R packages for data-cleaning/manipulation and visualization.
Visualization in R language

### 2021-01-18
https://pubmed.ncbi.nlm.nih.gov/31848574/
Review Brief Bioinform
. 2021 Jan 18;22(1):178-193. doi: 10.1093/bib/bbz155.
Current challenges and best-practice protocols for microbiome analysis
Richa Bharti 1, Dominik G Grimm 1
https://academic.oup.com/bib/article/22/1/178/5678919?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7820839/

### 2020-12-03
https://pubmed.ncbi.nlm.nih.gov/33363701/
Review Comput Struct Biotechnol J
. 2020 Dec 3:18:4048-4062. doi: 10.1016/j.csbj.2020.11.049. eCollection 2020.
Measuring the microbiome: Best practices for developing and benchmarking microbiomics methods
Nicholas A Bokulich 1, Michal Ziemski 1, Michael S Robeson 2nd 2, Benjamin D Kaehler 3
https://www.sciencedirect.com/science/article/pii/S200103702030516X
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7744638/

### 2019-12
https://pubmed.ncbi.nlm.nih.gov/31548653/
Review Nat Rev Microbiol
. 2019 Dec;17(12):725-741. doi: 10.1038/s41579-019-0255-9. Epub 2019 Sep 23.
Common principles and best practices for engineering microbiomes
Christopher E Lawson 1, William R Harcombe 2, Roland Hatzenpichler 3 4 5, Stephen R Lindemann 6, Frank E LÃ¶ffler 7 8, Michelle A O'Malley 9 10, HÃ©ctor GarcÃ­a MartÃ­n 10 11 12 13, Brian F Pfleger 14, Lutgarde Raskin 15, Ophelia S Venturelli 14 16 17, David G Weissbrodt 18, Daniel R Noguera 19 20, Katherine D McMahon 21 22
https://www.nature.com/articles/s41579-019-0255-9
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8323346/

----------
## CoverM

https://github.com/wwood/CoverM
Aroney, S.T., Newell, R.J., Nissen, J.N., Camargo, A.P., Tyson, G.W. and Woodcroft, B.J., 2025. CoverM: Read alignment statistics for metagenomics. arXiv preprint arXiv:2501.11217. https://doi.org/10.48550/arXiv.2501.11217

https://kazumaxneo.hatenablog.com/entry/2021/08/01/225920
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ãƒªãƒ¼ãƒ‰ã‚«ãƒãƒ¬ãƒƒã‚¸ ã¨relative abundanceã®è¨ˆç®—ãƒ„ãƒ¼ãƒ« coverM - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

https://yokazaki.hatenablog.com/entry/2019/12/06/212324
ã‚³ãƒ³ãƒ†ã‚£ã‚°ãƒ»ã‚²ãƒãƒ ã¸ã®ãƒªãƒ¼ãƒ‰ãƒãƒƒãƒ”ãƒ³ã‚°ãƒ»ã‚«ãƒãƒ¬ãƒƒã‚¸ã®è¨ˆç®—æ–¹æ³• - yokaã®blog
ã€€åå‰ã‚’è¦‹ã¦ã‚ã‹ã‚‹ã‚ˆã†ã«ã€CheckMãªã©ã§æœ‰åãªHugenholtzã¨Tyson ã®ãƒ©ãƒœã®M-Toolsã®ä¸€ã¤ã¨ã—ã¦é–‹ç™ºã•ã‚Œã¦ã„ã‚‹ã€‚

----------
## bromberglab

https://services.bromberglab.org/
mi-faser
annotate molecular functionality directly from sequencing read data

### mi-faser
https://services.bromberglab.org/mifaser/
https://services.bromberglab.org/mifaser/submit
mi-faser, microbiome - functional annotation of sequencing reads
is a super-fast ( < 20min/10GB of reads ) and accurate ( > 90% precision ) method for annotation of
molecular functionality encoded in sequencing read data without the need for assembly or gene finding.

https://pubmed.ncbi.nlm.nih.gov/29194524/
Nucleic Acids Res
. 2018 Feb 28;46(4):e23. doi: 10.1093/nar/gkx1209.
Functional sequencing read annotation for high precision microbiome analysis
Chengsheng Zhu 1, Maximilian Miller 1 2 3, Srinayani Marpaka 1, Pavel Vaysberg 1, Malte C RÃ¼hlemann 4, Guojun Wu 5, Femke-Anouska Heinsen 4, Marie Tempel 6, Liping Zhao 1 5 7, Wolfgang Lieb 6, Andre Franke 4, Yana Bromberg 1 8 9
https://academic.oup.com/nar/article/46/4/e23/4670955?login=false
https://pmc.ncbi.nlm.nih.gov/articles/PMC5829635/
AVAILABILITY

### rebean
https://services.bromberglab.org/rebean/
Read Embedding Based Enzyme ANnotation
REBEAN is a DNA language Model for enzymatic annotation of sequencing reads. REBEAN is built for discovery of enzymatic function in both known and unknown sequence space mitigating drawbacks of alignment and translation-based approaches. Read more.
https://services.bromberglab.org/rebean/about

https://www.biorxiv.org/content/10.1101/2024.12.10.627786v1.full
Deciphering enzymatic potential in metagenomic reads through DNA language model | bioRxiv

Read functional annotations were generated by mi-faser (See Methods), an alignment-based method that attained âˆ¼90% precision and âˆ¼50% recall in testing (Zhu, Miller et al. 2018).

----------
## rRNA

https://kazumaxneo.hatenablog.com/archive/category/rRNA
rRNA ã‚«ãƒ†ã‚´ãƒªãƒ¼ã®è¨˜äº‹ä¸€è¦§ - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

https://kazumaxneo.hatenablog.com/entry/2022/06/24/011756
ãƒšã‚¢ã‚¨ãƒ³ãƒ‰ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚·ãƒ³ã‚°ãƒªãƒ¼ãƒ‰ã‚’ä½¿ã£ã¦ãƒ¡ã‚¿ã‚²ãƒãƒ ã‚¢ã‚»ãƒ³ãƒ–ãƒªã‚²ãƒãƒ ã¨16S rRNAãƒãƒ¼ã‚«ãƒ¼éºä¼å­ã‚’çµã³ä»˜ã‘ã‚‹ MarkerMAG - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

https://kazumaxneo.hatenablog.com/entry/2022/03/09/233539
ãƒ¡ã‚¿ã‚²ãƒãƒ ã€ãƒ¡ã‚¿ãƒˆãƒ©ãƒ³ã‚¹ã‚¹ã‚¯ãƒªãƒ—ãƒˆãƒ¼ãƒ ã€ncRNAã®ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚·ãƒ³ã‚°ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰rRNAé…åˆ—ã‚’æ­£ç¢ºã‹ã¤é«˜é€Ÿã«æ¤œå‡ºãƒ»é™¤å»ã™ã‚‹ RiboDetector - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

### matam
https://github.com/bonsai-team/matam

https://pubmed.ncbi.nlm.nih.gov/29040406/
Bioinformatics
. 2018 Feb 15;34(4):585-591. doi: 10.1093/bioinformatics/btx644.
MATAM: reconstruction of phylogenetic marker genes from short sequencing reads in metagenomes
Pierre Pericard 1 2, Yoann Dufresne 1 2, LoÃ¯c Couderc 1 3, Samuel Blanquart 1 2, HÃ©lÃ¨ne Touzet 1 2
https://academic.oup.com/bioinformatics/article/34/4/585/4457361

https://kazumaxneo.hatenablog.com/entry/2018/09/18/162843
ãƒ¡ã‚¿ã‚²ãƒãƒ ã‹ã‚‰16Sãªã©ã®ã‚¿ãƒ¼ã‚²ãƒƒãƒˆã‚¢ã‚»ãƒ³ãƒ–ãƒªã‚’è¡Œã† MATAM - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------
### phyloFlash
https://github.com/HRGV/phyloFlash
https://hrgv.github.io/phyloFlash/

https://pubmed.ncbi.nlm.nih.gov/33109753/
mSystems
. 2020 Oct 27;5(5):e00920-20. doi: 10.1128/mSystems.00920-20.
phyloFlash: Rapid Small-Subunit rRNA Profiling and Targeted Assembly from Metagenomes
Harald R Gruber-Vodicka # 1, Brandon K B Seah # 2, Elmar Pruesse 3
https://journals.asm.org/doi/10.1128/msystems.00920-20
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7593591/
https://colil.dbcls.jp/browse/papers/33109753/1/100/year

https://kazumaxneo.hatenablog.com/entry/2019/02/12/073000
ãƒ¡ã‚¿ã‚²ãƒãƒ ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰rRNAã‚’ã‚¿ãƒ¼ã‚²ãƒƒãƒˆã‚¢ã‚»ãƒ³ãƒ–ãƒªã—ã€ç³»çµ±ã‚¢ã‚µã‚¤ãƒ³ã€å®šé‡ã€æ¯”è¼ƒã™ã‚‹ phyloFlash - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

ã—ã‹ã—ã€è‡ªå‹•ãƒ“ãƒ‹ãƒ³ã‚°ã®é€²æ­©ã«ã‚‚ã‹ã‹ã‚ã‚‰ãšï¼ˆref.23ï¼‰ã€ã»ã¨ã‚“ã©ã®metagenome assembled genomes (MAGs) ã¯ã€SSU rRNAéºä¼å­ã®æ–­ç‰‡ã•ãˆå«ã‚“ã§ã„ãªã„ã€ã¾ãŸå…¨éºä¼å­ã¯è¨€ã†ã¾ã§ã‚‚ãªãå«ã‚“ã§ã„ãªã„ï¼ˆref.1ã€24ï¼‰ã€‚

ã€€ç†æƒ³çš„ã«ã¯ã€æ—¢å­˜ã®è†¨å¤§ãªSSU rRNAéºä¼å­ã®ãƒŠãƒ¬ãƒƒã‚¸ãƒ™ãƒ¼ã‚¹ã‚’ï¼ˆãƒ¡ã‚¿ï¼‰ã‚²ãƒãƒŸã‚¯ã‚¹ãƒ—ãƒ­ã‚¸ã‚§ã‚¯ãƒˆã§æ´»ç”¨ã—ã¦ã€ã„ãã¤ã‹ã®ç•°ãªã‚‹çµæœã‚’å¾—ãŸã„ã¨è€ƒãˆã¦ã„ã‚‹ï¼šã‚¢ã‚»ãƒ³ãƒ–ãƒªãªã—ã®åˆ†é¡å­¦çš„ãƒ—ãƒ­ãƒ•ã‚¡ã‚¤ãƒªãƒ³ã‚°ï¼ˆref.6ã€7ï¼‰ã€ç³»çµ±å­¦ã®ãŸã‚ã®å…¨é•·é…åˆ—ã®ã‚¿ãƒ¼ã‚²ãƒƒãƒˆã‚¢ã‚»ãƒ³ãƒ–ãƒªã¨ã€ãã‚Œã‚ˆã‚‹ãƒ—ãƒ­ãƒ¼ãƒ–è¨­è¨ˆï¼ˆref.15ã€16ã€25ã€26ï¼‰ã€ãŠã‚ˆã³SSU rRNAé…åˆ—ã‚’å®Œå…¨ãªã‚²ãƒãƒ ã¸ãƒªãƒ³ã‚¯ã•ã›ã‚‹ï¼ˆref.27ï¼‰ã€‚ã“ã‚Œã‚‰ã®å„ç›®çš„ã®ãŸã‚ã«ã€ãã‚Œãã‚Œç‹¬è‡ªã®é•·æ‰€ã¨çŸ­æ‰€ã‚’æŒã¤åˆ¥ã€…ã®ã‚½ãƒ•ãƒˆã‚¦ã‚§ã‚¢ãƒ„ãƒ¼ãƒ«ãŒã™ã§ã«é–‹ç™ºã•ã‚Œã¦ã„ã‚‹ï¼ˆref.16ï¼‰ï¼ˆMATAMç´¹ä»‹ï¼‰ã€‚ãŸã ã—ã€ã“ã‚Œã‚‰ã¯åŒã˜å•é¡Œã®æœ€çµ‚çš„ã«ã¯ç•°ãªã‚‹å´é¢ã§ã‚ã‚‹ãŸã‚ã€ä¸€ç·’ã«æ¤œè¨ã™ã‚‹å¿…è¦ãŒã‚ã‚‹ã€‚


https://pubmed.ncbi.nlm.nih.gov/39693444/
Comparative Study Sci Adv
. 2024 Dec 20;10(51):eadq0645. doi: 10.1126/sciadv.adq0645. Epub 2024 Dec 18.
A global comparison of surface and subsurface microbiomes reveals large-scale biodiversity gradients, and a marine-terrestrial divide
https://www.science.org/doi/full/10.1126/sciadv.adq0645

The metagenomes were used for taxonomic analyses based on two different taxonomic markers, 16S rRNA gene sequences retrieved by phyloFlash (pF16S; before read assembly) and ribosomal protein S3 (rpS3) genes (after read assembly). 

Metagenomic short reads were mapped to the SILVA SSU reference database (144) to assign nearest taxonomic units, and full-length 16S/18S rRNA gene sequences were reconstructed from metagenomes using phyloFlash v3.4 (155). 

----------
### PhyloSift

https://pubmed.ncbi.nlm.nih.gov/35749534/
PLoS One
. 2022 Jun 24;17(6):e0270372. doi: 10.1371/journal.pone.0270372. eCollection 2022.
Phylogenetic diversity analysis of shotgun metagenomic reads describes gut microbiome development and treatment effects in the post-weaned pig
Daniela Gaio 1, Matthew Z DeMaere 1, Kay Anantanawat 1, Graeme J Eamens 2, Linda Falconer 2, Toni A Chapman 2, Steven Djordjevic 1, Aaron E Darling 1
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0270372
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9232140/

----------
### EzBioCloud

https://www.ezbiocloud.net/

https://pubmed.ncbi.nlm.nih.gov/38888585/
Int J Syst Evol Microbiol
. 2024 Jun;74(6):006421. doi: 10.1099/ijsem.0.006421.
EzBioCloud: a genome-driven database and platform for microbiome identification and discovery
Mauricio Chalita 1, Yeong Ouk Kim 1 2, Sein Park 1 2, Hyun-Seok Oh 1, Jae Hyoung Cho 1, Jeongsup Moon 1, Nuga Baek 1, Changsik Moon 1, Kihyun Lee 1, Junwon Yang 1 2, Gi Gyun Nam 1, Yeonjae Jung 1 2, Seong-In Na 1, Michael James Bailey 1, Jongsik Chun 1


https://pubmed.ncbi.nlm.nih.gov/28005526/
Int J Syst Evol Microbiol
. 2017 May;67(5):1613-1617. doi: 10.1099/ijsem.0.001755. Epub 2017 May 30.
Introducing EzBioCloud: a taxonomically united database of 16S rRNA gene sequences and whole-genome assemblies
Seok-Hwan Yoon 1, Sung-Min Ha 1, Soonjae Kwon 1, Jeongmin Lim 1, Yeseul Kim 1, Hyungseok Seo 1, Jongsik Chun 1
https://pmc.ncbi.nlm.nih.gov/articles/PMC5563544/

https://kazumaxneo.hatenablog.com/entry/2019/01/30/073000
ãƒã‚¯ãƒ†ãƒªã‚¢ã¨ã‚¢ãƒ¼ã‚­ã‚¢ã®ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ EzBioCloud - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------

## updates

### 2022

### 2022-11-28

https://www.nature.com/articles/s43705-022-00204-6
Unexpected absence of ribosomal protein genes from metagenome-assembled genomes | ISME Communications


6:25 AM Â· Dec 8, 2022
https://twitter.com/luispedrocoelho/status/1600602523206791168
Luis Pedro CoelhoğŸŒ» on Twitter: ""Unexpected absence of ribosomal protein genes from metagenome-assembled genomes" by Mise &amp; Iwasaki (Quoted text is from paper [slightly edited for space], unquoted from me) https://t.co/Xlw9v5V3c6" / Twitter


6:40 AM Â· Nov 30, 2022
https://twitter.com/iwasakiw/status/1597707175836934146
è«–æ–‡ã€Œãƒ¡ã‚¿ã‚²ãƒãƒ ã‚¢ã‚»ãƒ³ãƒ–ãƒ«ã‚²ãƒãƒ ã‹ã‚‰ã®äºˆæƒ³å¤–ã®ãƒªãƒœã‚½ãƒ¼ãƒ ã‚¿ãƒ³ãƒ‘ã‚¯è³ªéºä¼å­ã®æ¬ å¤±ã€ã‚’å‡ºç‰ˆã—ã¾ã—ãŸï¼
Kazumori Mise and Wataru Iwasaki.
Unexpected absence of ribosomal protein genes from metagenome-assembled genomes.
ISME Communications, 2, 118. (2022)
ã‚·ãƒ§ãƒƒãƒˆã‚¬ãƒ³ã‚·ãƒ¼ã‚±ãƒ³ã‚¹ã‹ã‚‰ç’°å¢ƒå¾®ç”Ÿç‰©ã®ã‚²ãƒãƒ ã‚’å¾©å…ƒã™ã‚‹MAGï¼ˆãƒ¡ã‚¿ã‚²ãƒãƒ ã‚¢ã‚»ãƒ³ãƒ–ãƒ«ã‚²ãƒãƒ ï¼‰è§£æãŒä¸€èˆ¬çš„ã«ãªã‚Šã€æœªåŸ¹é¤Šå¾®ç”Ÿç‰©ã®é€²åŒ–ã‚„ç”Ÿç†ãƒ»ç”Ÿæ…‹ã«é–¢ã™ã‚‹ç ”ç©¶ãŒå¤§ããé€²ã‚“ã§ã„ã¾ã™ã€‚ä¸€æ–¹ã§MAGã«ã¯ã¾ã å•é¡Œç‚¹ã‚‚å¤šãã€ä¾‹ãˆã°ã€åå¾©é…åˆ—ã‚„å¤–æ¥é…åˆ—ã¯MAGã‹ã‚‰æ¬ è½ã—ã‚„ã™ã„ã“ã¨ãŒè‰¯ãçŸ¥ã‚‰ã‚Œã¦ã„ã¾ã™ã€‚
æœ¬ç ”ç©¶ã§ã¯ã€åå¾©é…åˆ—ã§ã‚‚å¤–æ¥é…åˆ—ã§ã‚‚ãªã„ã€Œãƒªãƒœã‚½ãƒ¼ãƒ ã‚¿ãƒ³ãƒ‘ã‚¯è³ªéºä¼å­ã€ã‚‚ã¾ãŸã€äºˆæƒ³å¤–ã«MAGã‹ã‚‰æ¬ è½ã—ã‚„ã™ã„ã“ã¨ã‚’æ˜ã‚‰ã‹ã«ã—ã¾ã—ãŸã€‚ãã—ã¦ã€ã‚³ãƒ‰ãƒ³ä½¿ç”¨é »åº¦ã®åã‚Šã«ã‚ˆã‚‹ãƒ“ãƒ‹ãƒ³ã‚°ã®èª¤ã‚ŠãŒãã®åŸå› ã§ã‚ã‚Šã€ã“ã‚ŒãŒå¾®ç”Ÿç‰©ã®ç”Ÿå­˜æˆ¦ç•¥ã«ã‚‚ãƒªãƒ³ã‚¯ã—ã¦ã„ã‚‹ã“ã¨ã‚’æ˜ã‚‰ã‹ã«ã—ã¾ã—ãŸã€‚

### 2021-01-07

https://pubmed.ncbi.nlm.nih.gov/33462601/
Genome Biol Evol
. 2021 Jan 7;13(1):evaa238. doi: 10.1093/gbe/evaa238.
Anomalous Phylogenetic Behavior of Ribosomal Proteins in Metagenome-Assembled Asgard Archaea
Sriram G Garg 1, Nils Kapust 1, Weili Lin 2, Michael Knopp 1, Fernando D K Tria 1, Shijulal Nelson-Sathi 3, Sven B Gould 1, Lu Fan 4 5 6, Ruixin Zhu 2, Chuanlun Zhang 4 5 7, William F Martin 1
https://academic.oup.com/gbe/article/13/1/evaa238/5988511
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7813640/

### 

8:50 AM Â· Apr 29, 2022
https://twitter.com/RobynJWright/status/1519826469610041344
Robyn Wright on Twitter: "Q: Which is better for taxonomic classification of #metagenomics samples - Kraken2 or MetaPhlAn 3? A: It really depends! Read the very short story in this thread, or the full story in my preprint w/ @BetaScience and AndrÃ© Comeau: https://t.co/2U3e7odn8Y 1/" / Twitter

### 2022-02-05

1:34 AM Â· Feb 6, 2022
https://twitter.com/arroboso/status/1490000882980769792
á´€ÊŸá´‡á´Šá´€É´á´…Ê€á´ êœ±á´€É´á´„Êœá´‡á´¢-êœ°ÊŸá´Ê€á´‡êœ± ğŸ§¬ğŸ‘¨ğŸ½â€ğŸ’» on Twitter: "@bielleogy Have you tried MetaPhlAn? I think that using single copy marker genes will get you a better classification than using k-mer signatures like Kraken2 or others. At least that is what we tested in a publication some time ago. https://t.co/NijLnyVA7v https://t.co/nj0hE7UTad" / Twitter
https://www.nature.com/articles/s41598-018-30515-5
Analysis of sequencing strategies and tools for taxonomic annotation: Defining standards for progressive metagenomics | Scientific Reports


11:13 PM Â· Feb 5, 2022
https://twitter.com/AMohamed85/status/1489965514902446083
Amin Mohamed on Twitter: "@bielleogy For MAGs I recently used GTDB (bacteria and archaea only). For read-based analyses, I used kraken2 and metaphlan after removing host reads. Might be better to try on contigs (metaSpades) rather than reads!" / Twitter


6:10 AM Â· Feb 5, 2022
https://twitter.com/bielleogy/status/1489708006283440129
Dr. Elinne Becket on Twitter: "Any marine (or related) microbiologists have recs for metagenome taxonomic classifiers for those sample types? Or has anyone in those fields had success getting a decent % classified from Kraken2? I'm only getting like 10-15% classified despite every option I've tried." / Twitter


### 2021


### 2020


https://twitter.com/gggtta/status/1215107137476513792
æ¾æœ¬ on Twitter: "ã‚µãƒ³ãƒ—ãƒ«ä¸­ã«å«ã¾ã‚Œã‚‹OTUã‚„ãƒã‚¯ãƒ†ãƒªã‚¢ã®ç›¸å¯¾å­˜åœ¨é‡ã®ãƒ—ãƒ­ãƒ•ã‚¡ã‚¤ãƒ«ãŒã¾ã¨ã¾ã£ãŸã€ã‚µãƒ³ãƒ—ãƒ«æ•°Ã—OTUæ•°ã®è¡Œåˆ—ã®ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã£ã¦ã©ã“ã‹ã«ãªã„ã‹ãªï¼Ÿ" / Twitter
12:04 PM Â· Jan 9, 2020



### 2019

https://qiita.com/advent-calendar/2019/metagenome
ãƒ¡ã‚¿ã‚²ãƒãƒ  Advent Calendar 2019

https://qiita.com/hi-mori/items/a1d3e0923f2cc85edb84
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®æƒ…å ±è§£æãŒå­¦ã¹ã‚‹Webæ•™æï¼ˆã‚¹ãƒ©ã‚¤ãƒ‰ã‚„å‹•ç”»ï¼‰

https://qiita.com/Yohei__K/items/dffc8767581bfbaad812
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ã‚¤ã‚±ã¦ã‚‹ãƒãƒƒãƒˆãƒ¯ãƒ¼ã‚¯è§£ææ‰‹æ³•ã€WGCNAè§£èª¬
R packageã«é–¢ã—ã¦

https://www.ncbi.nlm.nih.gov/pubmed/31848574/
Brief Bioinform. 2019 Dec 18. pii: bbz155. doi: 10.1093/bib/bbz155. [Epub ahead of print]
Current challenges and best-practice protocols for microbiome analysis.
Bharti R, Grimm DG.
https://github.com/grimmlab/MicrobiomeBestPracticeReview
Both the workflows are controlled by bash scripts


https://twitter.com/tjsharpton/status/1192820360556793856
Thomas Sharpton on Twitter: "Published! Happy to share our review on statistical methods for integrating multi'omic microbiome data. This results from a wonderful NIGMS/DMS sponsored collaboration with my brilliant statistics colleagues. I hope you find it helpful! cc @CourtneyRArmour https://t.co/lrXI2PXrZK https://t.co/b2mIbFcKSU" / Twitter
12:05 AM Â· Nov 9, 2019
![](https://pbs.twimg.com/media/EI2_yXcX0AM31w3?format=jpg&name=small)



https://twitter.com/WvSchaik/status/1148595145790689281
Willem van Schaik on Twitter: "Really interesting approach to link ALL microbiome studies and allow for comparative analyses: https://t.co/PVM3HCi7hT by JoÃ£o Rodriguez #FEMS2019"
10:09 AM - 9 Jul 2019

Posted May 21, 2019.
https://www.biorxiv.org/content/10.1101/644740v1
Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment
https://twitter.com/NiranjanTW/status/1131022012556533760
Our hospital microbiome work as part of @metasub! All Illumina and Nanopore data should be available from ENA: Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment https://www.biorxiv.org/content/10.1101/644740v1 â€¦
@kalisvar @oontek1 @mason_lab
1000s of @nanopore assembled genomes and circular plasmid sequences too!


https://www.ncbi.nlm.nih.gov/pubmed/30418481
Bioinformatics. 2019 Jun 1;35(12):2150-2152. doi: 10.1093/bioinformatics/bty905.
MAGpy: a reproducible pipeline for the downstream analysis of metagenome-assembled genomes (MAGs).
Stewart RD1, Auffret MD2, Snelling TJ3, Roehe R2, Watson M1.


https://twitter.com/ASMicrobiology/status/1126925145069817856
We need an ecological understanding of the microbiome as a community of interacting individuals  and an evolutionary understanding of the microbiome as a trait of the host: an @mSystemsJ perspective from @sarahmhird http://ow.ly/UAkL50u2gEp 
3:01 PM - 10 May 2019
https://twitter.com/sarahmhird
Sarah Hird


https://twitter.com/Tyu_Shi/status/1110601849386618880
Tyu_Shi on Twitter: "ãƒã‚¦ã‚¹gut microbiomeã®è§£æã§ãƒ™ãƒ³ãƒ€ãƒ¼ãŒé•ã†ãƒã‚¦ã‚¹ã¯åˆ¥ç‰©ã¨ã—ã¦æ‰±ã†ã®ã¯ã»ã¼å¸¸è­˜ãªã®ã§é©šãã¯ç„¡ã„ãŒã€ã“ã†ã„ã†ã®ã‚’è«–æ–‡ã«ã—ã¦ãã‚Œã‚‹ã®ã¯å¼•ç”¨ã§æ¸ˆã¾ã›ã‚‰ã‚Œã‚‹ã®ã§ã‚ã‚ŠãŒãŸã„ã€‚ Mouse vendor influence on the bacterial and viral gut composition exceeds the effect of diet https://t.co/mwS8HwmvMf"
1:58 PM - 26 Mar 2019

https://twitter.com/Tyu_Shi/status/1110583615526596609
reference basedã¨de novo basedã®hybridå‹ã¨ã„ã†ã“ã¨ã§ä¸€æ™‚æœŸæŒã¦å›ƒã•ã‚Œã¦æµè¡Œã‚Šãã†ã«ãªã£ãŸopen or closed reference OTU clusteringæ³•ãŒã€ã‚ã‚Šã¨ã—ã£ã‹ã‚Šã¨æ‰¹åˆ¤ã•ã‚Œã¦ã„ã‚‹ã€‚
12:45 PM - 26 Mar 2019

https://twitter.com/PatSchloss/status/1110509388664619009
Pat Schloss on Twitter: "Please stop using the open (and closed) reference clustering methods based on USEARCH/VSEARCH. Instead, use de novo clustering algorithms to form OTUs 1/"
7:50 AM - 26 Mar 2019

https://www.ncbi.nlm.nih.gov/pubmed/30902113
Microbiome. 2019 Mar 22;7(1):46. doi: 10.1186/s40168-019-0658-x.
Sunbeam: an extensible pipeline for analyzing metagenomic sequencing experiments.
Clarke EL1, Taylor LJ1, Zhao C2, Connell A1, Lee JJ2, Fett B2, Bushman FD1, Bittinger K3.


### 2018

https://twitter.com/pathogenomenick/status/1053620015579172864
Nick Loman on Twitter: "An extremely informative thread explaining why metagenomic assemblies of related species and strains results in subsequent difficulties during binning:â€¦ https://t.co/CzUcpRiYdJ"
8:12 AM - 20 Oct 2018

https://www.ncbi.nlm.nih.gov/pubmed/29950328
J Clin Microbiol. 2018 Aug 27;56(9). pii: e00813-18. doi: 10.1128/JCM.00813-18. Print 2018 Sep.
Lowering the Barriers to Routine Whole-Genome Sequencing of Bacteria in the Clinical Microbiology Laboratory.
Rhoads DD1,2.


https://www.ncbi.nlm.nih.gov/pubmed/29460198
J Plant Res. 2018 Jul;131(4):709-717. doi: 10.1007/s10265-018-1017-x. Epub 2018 Feb 19.
Collaborative environmental DNA sampling from petal surfaces of flowering cherry Cerasusâ€‰Ã—â€‰yedoensis 'Somei-yoshino' across the Japanese archipelago.
Ohta T1, Kawashima T2, Shinozaki NO3, Dobashi A4, Hiraoka S5, Hoshino T6, Kanno K7, Kataoka T8, Kawashima S9, Matsui M10, Nemoto W11, Nishijima S12,13,14, Suganuma N15, Suzuki H16, Taguchi YH17, Takenaka Y18, Tanigawa Y19, Tsuneyoshi M20, Yoshitake K21, Sato Y22, Yamashita R22, Arakawa K23, Iwasaki W10.

Fig.2: "Deblur" -> "Deblur or DADA2" ã«ä¿®æ­£ã—
https://twitter.com/merenbey/status/999416078446071809
A. Murat Eren (Meren) on Twitter: "This is unfair to DADA2
6:25 PM - 23 May 2018


### 2017

https://www.ncbi.nlm.nih.gov/pubmed/28321234
Front Genet. 2017 Mar 6;8:23. doi: 10.3389/fgene.2017.00023. eCollection 2017.
A Review of Bioinformatics Tools for Bio-Prospecting from Metagenomic Sequence Data.
Roumpeka DD1, Wallace RJ2, Escalettes F3, Fotheringham I3, Watson M1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5337752/
Data Sharing and Online Portals
MG-RAST (Glass et al., 2010) 
EBI MetaGenomics (Hunter et al., 2014) 
IMG/M (Markowitz et al., 2014) 

https://www.ncbi.nlm.nih.gov/pubmed/25983555
Bioinform Biol Insights. 2015 May 5;9:75-88. doi: 10.4137/BBI.S12462. eCollection 2015.
Metagenomics: tools and insights for analyzing next-generation sequencing data derived from biodiversity studies.
Oulas A1, Pavloudi C2, Polymenakou P1, Pavlopoulos GA3, Papanikolaou N3, Kotoulas G1, Arvanitidis C1, Iliopoulos I3.


### 2017

https://www.microbe.net/2017/11/15/nice-title-nice-paper-microbiome-datasets-are-compositional-and-this-is-not-optional/
Nice title, nice paper: Microbiome Datasets Are Compositional: And This Is Not OptionalÂ  â€“ microBEnet: the microbiology of the Built Environment network

1:39 PM - 18 Dec 2017
https://twitter.com/Tyu_Shi/status/942826707022520320
Tyu_Shi on Twitter: "ã¾ã è¨˜äº‹ã¯å°‘ãªã„ãŒã€ãƒ’ãƒˆè…¸å†…ãƒ¡ã‚¿é–¢é€£ã®Dalhousie Universityã§ã®journal clubã®blogã€‚Points of Interestã¨Points of ConfusionãŒç®‡æ¡æ›¸ãã«ã•ã‚Œã¦ã„ã‚‹ã®ã§ã€å…ƒè«–æ–‡è‡ªåˆ†ã§èª­ã‚“ã å¾Œã«blogèª­ã‚“ã§äººã¨ã®è§£é‡ˆã®å·®ç•°ã‚’çŸ¥ã‚‹â€¦ https://t.co/HdO5AftF7a"

https://www.ncbi.nlm.nih.gov/pubmed/30197908
Genes Dis. 2017 Sep;4(3):138-148. doi: 10.1016/j.gendis.2017.06.001. Epub 2017 Jun 23.
Hypothesis Testing and Statistical Analysis of Microbiome.
Xia Y1,2, Sun J2.

2017å¹´11æœˆ23æ—¥
http://www.natureasia.com/ja-jp/nature/551/7681/nature24460/
ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹ï¼šå®šé‡çš„ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ãƒ—ãƒ­ãƒ•ã‚¡ã‚¤ãƒªãƒ³ã‚°ã§æ˜ã‚‰ã‹ã«ãªã£ãŸè…¸å†…ç´°èŒç¾¤é›†ã®å¤‰å‹•ã¨å¾®ç”Ÿç‰©é‡ã®é–¢é€£

3:09 PM - 7 Nov 2017
beneficial microbes æœ‰ç›Šãªå¾®ç”Ÿç‰© https://twitter.com/phylogenomics/status/927991509017866240


### 2011

https://wg.biosciencedbc.jp/doku.php/ã‚²ãƒãƒ ãƒ»ãƒ¡ã‚¿ã‚²ãƒãƒ æƒ…å ±ã‚’åŸºç›¤ã¨ã—ãŸå¾®ç”Ÿç‰©dbã®çµ±åˆ
ã‚²ãƒãƒ ãƒ»ãƒ¡ã‚¿ã‚²ãƒãƒ æƒ…å ±ã‚’åŸºç›¤ã¨ã—ãŸå¾®ç”Ÿç‰©dbã®çµ±åˆ.txt Â· æœ€çµ‚æ›´æ–°: 2011/12/27 09:54 +0900 (å¤–éƒ¨ç·¨é›†)


----------


----------
### ASaiM
https://www.biorxiv.org/content/early/2017/09/04/183970
ASaiM: a Galaxy-based framework to analyze raw shotgun data from microbiota | bioRxiv

Published on Jul 9, 2015
https://www.slideshare.net/bebatut/asaim-an-environment
JOBIM 2015 - ASaiM: an environment to analyze intestinal microbiota - Demo with analysis of gut metatranscriptomic sequences

https://twitter.com/galaxyproject/status/794453380202725376
Galaxy Project on Twitter: "Computational workflow for fine-grained analysis of metagenomic samples, PÃ©rez-Wohlfeil et al. https://t.co/MAJKM97Pxb #usegalaxy @InfoUMA https://t.co/FAtN47hRES"


----------
## HGT

ãƒ¡ã‚¿ã‚²ãƒãƒ ã«ãŠã‘ã‚‹éºä¼å­æ°´å¹³ä¼æ’­ã‚¤ãƒ™ãƒ³ãƒˆåŒå®šæ‰‹æ³•ã«é–¢ã™ã‚‹ç·èª¬
review
https://www.ncbi.nlm.nih.gov/pubmed/31504488
Genome Biol Evol. 2019 Oct 1;11(10):2750-2766. doi: 10.1093/gbe/evz184.
Current and Promising Approaches to Identify Horizontal Gene Transfer Events in Metagenomes.
Douglas GM1, Langille MGI1.


### DaisyGPS
https://www.ncbi.nlm.nih.gov/pubmed/31335917
PLoS Comput Biol. 2019 Jul 23;15(7):e1007208. doi: 10.1371/journal.pcbi.1007208. eCollection 2019 Jul.
Where did you come from, where did you go: Refining metagenomic analysis tools for horizontal gene transfer characterisation.
Seiler E1,2, Trappe K1, Renard BY1.

### MetaCHIP

https://www.ncbi.nlm.nih.gov/pubmed/30832740
Microbiome. 2019 Mar 4;7(1):36. doi: 10.1186/s40168-019-0649-y.
MetaCHIP: community-level horizontal gene transfer identification through the combination of best-match and phylogenetic approaches.
Song W1,2, Wemheuer B1,3, Zhang S1,2, Steensen K1,4, Thomas T5,6.

http://kazumaxneo.hatenablog.com/entry/2019/03/12/224600
ãƒ¡ã‚¿ã‚²ãƒãƒ binsã‹ã‚‰HGTã‚’æ¤œå‡ºã™ã‚‹ MetaCHIP - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------
## MGnify

https://www.ebi.ac.uk/metagenomics/
MGnify

https://www.ebi.ac.uk/metagenomics/sequence-search/search/phmmer
phmmer search | EBI metagenomics


https://www.ncbi.nlm.nih.gov/pubmed/31696235
Nucleic Acids Res. 2019 Nov 7. pii: gkz1035. doi: 10.1093/nar/gkz1035. [Epub ahead of print]
MGnify: the microbiome analysis resource in 2020.
Mitchell AL1, Almeida A1,2, Beracochea M1, Boland M1, Burgin J1, Cochrane G1, Crusoe MR3, Kale V1, Potter SC1, Richardson LJ1, Sakharova E1, Scheremetjew M1, Korobeynikov A4, Shlemov A4, Kunyavskaya O4, Lapidus A4, Finn RD1.
https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz1035/5614179

https://integbio.jp/dbcatalog/record/nbdc01916
EBI Metagenomics ã¯ å¯¾è±¡ç¯„å›²å¤‰æ›´ã«ã‚ã‚ã›ã¦åç§°ã‚’Mgnifyã«å¤‰æ›´ã—ã¾ã—ãŸã€‚ã“ã¡ã‚‰ã¯ã™ã¹ã¦ã®ãƒ¡ã‚¿ã‚²ãƒãƒŸã‚¯ã‚¹ç ”ç©¶è€…ã‚’æ”¯æ´ã™ã‚‹ã“ã¨ã‚’ç›®çš„ã¨ã—ãŸã€è‡ªç”±ã«åˆ©ç”¨å¯èƒ½ãªãƒªã‚½ãƒ¼ã‚¹ã§ã™ã€‚ãƒ¡ã‚¿ã‚²ãƒãƒ ãƒ‡ãƒ¼ã‚¿ã‚’è‡ªå‹•çš„ã«è§£æï½¥ä¿å­˜ã™ã‚‹ãƒ‘ã‚¤ãƒ—ãƒ©ã‚¤ãƒ³ã¨ã—ã¦æ©Ÿèƒ½ã—ã¾ã™ã€‚ç³»çµ±ç™ºç”Ÿã®å¤šæ§˜æ€§ã®ä»–ã€ã‚µãƒ³ãƒ—ãƒ«ã®æ©Ÿèƒ½çš„ãŠã‚ˆã³ä»£è¬çš„ãƒãƒ†ãƒ³ã‚·ãƒ£ãƒ«ã«ã¤ã„ã¦ã®çŸ¥è¦‹ã‚’æä¾›ã™ã‚‹ã“ã¨ã‚’ç›®çš„ã¨ã—ã¦ã„ã¾ã™ã€‚ãƒªãƒã‚¸ãƒˆãƒªå†…ã®ã™ã¹ã¦ã®å…¬é–‹ãƒ‡ãƒ¼ã‚¿ã¯ã€è‡ªç”±ã«å‚ç…§ã™ã‚‹ã“ã¨ãŒã§ãã¾ã™ã€‚





----------

### Metagenomic Assembly

https://omictools.com/metagenomic-assembly-category
Assembly software tools | Metagenomic sequencing analysis - OMICtools

https://www.ncbi.nlm.nih.gov/pubmed/28693474
BMC Genomics. 2017 Jul 10;18(1):521. 
Assembling metagenomes, one community at a time.

https://twitter.com/StevenJRobbins/status/849484107797204997
Three very recent papers comparing metagenome assembly tools:

https://www.ncbi.nlm.nih.gov/pubmed/28099457
PLoS One. 2017 Jan 18;12(1):e0169662.
Comparing and Evaluating Metagenome Assembly Tools from a Microbiologist's Perspective - Not Only Size Matters!

https://www.ncbi.nlm.nih.gov/pubmed/27698619
Yale J Biol Med. 2016 Sep 30;89(3):353-362. eCollection 2016.
Metagenomic Assembly: Overview, Challenges and Applications.
Ghurye JS1, Cepeda-Espinoza V1, Pop M1.

[MetAMOS](https://github.com/treangen/MetAMOS)
https://www.ncbi.nlm.nih.gov/pubmed/23320958
Genome Biol. 2013 Jan 15;14(1):R2. doi: 10.1186/gb-2013-14-1-r2.
MetAMOS: a modular and open source metagenomic assembly and analysis pipeline.

[MOCAT](http://www.bork.embl.de/mocat/)
https://www.ncbi.nlm.nih.gov/pubmed/23082188
PLoS One. 2012;7(10):e47656.
MOCAT: a metagenomics assembly and gene prediction toolkit.

----------
## book

### 2018-11-19
ã‚ˆãã‚ã‹ã‚‹ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹å…¥é–€/è—¤åšå¹¸/è¬›è«‡ç¤¾
https://www.kspub.co.jp/book/detail/5138212.html
http://bookclub.kodansha.co.jp/product?item=0000275926
ï¼™ç« ã€€ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ

### 2016-12-01
https://www.yodosha.co.jp/yodobook/book/9784758101974/
å®Ÿé¨“åŒ»å­¦åˆ¥å†Šï¼šNGSã‚¢ãƒ—ãƒªã‚±ãƒ¼ã‚·ãƒ§ãƒ³ ä»Šã™ãå§‹ã‚ã‚‹ï¼ã€€ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã€€å®Ÿé¨“ãƒ—ãƒ­ãƒˆã‚³ãƒ¼ãƒ«ã€œãƒ’ãƒˆå¸¸åœ¨ç´°èŒå¢ã‹ã‚‰ç’°å¢ƒãƒ¡ã‚¿ã‚²ãƒãƒ ã¾ã§ã‚µãƒ³ãƒ—ãƒ«èª¿è£½ã¨è§£æã®ã‚³ãƒ„ - ç¾ŠåœŸç¤¾
ç¬¬1ç« ã€€ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã®åŸºæœ¬çŸ¥è­˜
2ï¼ãƒ¡ã‚¿ã‚²ãƒãƒ ãƒ‡ãƒ¼ã‚¿ã®æƒ…å ±è§£æã®ã‚ªãƒ¼ãƒãƒ¼ãƒ“ãƒ¥ãƒ¼ã€æ±ã€€å…‰ä¸€ï¼Œæ£®ã€€å®™å²ï¼Œé»’å·ã€€é¡•ã€‘â€©
### 2016-09-12
http://www.kyoto-up.or.jp/book.php?id=2128
å¾®ç”Ÿç‰©ç”Ÿæ…‹å­¦ã€€ã‚²ãƒãƒ è§£æã‹ã‚‰ã‚¨ã‚³ã‚·ã‚¹ãƒ†ãƒ ã¾ã§
Chapter 10ã€€å¾®ç”Ÿç‰©ã¨ã‚¦ã‚£ãƒ«ã‚¹ã®ã‚²ãƒãƒ ãŠã‚ˆã³ãƒ¡ã‚¿ã‚²ãƒãƒ 

----------



----------
## NA
not available
ã“ã®ãƒ„ãƒ¼ãƒ«ã¯ä½¿ç”¨ã§ããªããªã‚Šã¾ã—ãŸã€‚

### cognizer
https://omictools.com/cognizer-tool
This tool is not available anymore.
https://www.ncbi.nlm.nih.gov/pubmed/26561344
PLoS One. 2015 Nov 11;10(11):e0142102. doi: 10.1371/journal.pone.0142102. eCollection 2015.
COGNIZER: A Framework for Functional Annotation of Metagenomic Datasets.
Bose T1, Haque MM1, Reddy C1, Mande SS1.
The COGNIZER framework includes a cross-mapping database that enables end-users to simultaneously derive/infer KEGG, Pfam, GO, and SEED subsystem information from the COG annotations.




----------
## CAMI
Critical Assessment of Metagenome Interpretation

https://twitter.com/CAMI_challenge

https://msystems.asm.org/content/3/4/e00103-18
Critical Assessment of Metagenome Interpretation Enters the Second Round | mSystems

Currently, CAMI is preparing for a second round of challenges, tentatively planned to open later this year. CAMI will provide data sets representing different environments and again offer assembly, taxonomic and genomic binning, as well as taxonomic profiling challenges (Fig. 1). Two multisample â€œtoyâ€ data sets representing microbial communities from different human body sites and from mouse gut are already provided to allow participants to prepare for the challenges (https://data.cami-challenge.org/participate).

https://www.ncbi.nlm.nih.gov/pubmed/28967888
Nat Methods. 2017 Nov;14(11):1063-1071. doi: 10.1038/nmeth.4458. Epub 2017 Oct 2.
Critical Assessment of Metagenome Interpretation-a benchmark of metagenomics software.

----------
## batch effect

### 2024

2024 è‘—è€… Fan, Howard James
https://open.bu.edu/handle/2144/47924
Improving batch effect correction of metagenomic data: applications in the black womenâ€™s health study
First, I evaluated ComBat-Seq, along with existing microbiome-specific tools, in removing batch effects from both simulated 16S rRNA and real-world shotgun metagenomic sequencing data while preserving effects belonging to biological factors of interest. Second, I applied ComBat-Seq in an epidemiological study 

### ComBat-seq
https://github.com/zhangyuqing/ComBat-seq
https://pubmed.ncbi.nlm.nih.gov/33015620/
NAR Genom Bioinform
. 2020 Sep;2(3):lqaa078. doi: 10.1093/nargab/lqaa078. Epub 2020 Sep 21.
ComBat-seq: batch effect adjustment for RNA-seq count data
Yuqing Zhang 1, Giovanni Parmigiani 2, W Evan Johnson 3
https://academic.oup.com/nargab/article/2/3/lqaa078/5909519
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7518324/

### 2023 Oct 4
https://pubmed.ncbi.nlm.nih.gov/37841330/
Review Comput Struct Biotechnol J
. 2023 Oct 4:21:4804-4815. doi: 10.1016/j.csbj.2023.10.001. eCollection 2023.
Data pre-processing for analyzing microbiome data - A mini review
Ruwen Zhou 1, Siu Kin Ng 1, Joseph Jao Yiu Sung 1 2, Wilson Wen Bin Goh 1 3 4, Sunny Hei Wong 1 2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10569954/
https://www.csbj.org/article/S2001-0370(23)00357-4/fulltext
2. Pre-processing methods
1) Filtering
2) Batch Effect Correction 
3) Imputation 
4) Normalization

Table 3
Common methods for batch effect correction. 

### MBECS
https://bioconductor.org/packages/MBECS/
https://bioconductor.org/packages/release/bioc/vignettes/MBECS/inst/doc/mbecs_vignette.html
https://github.com/rmolbrich/MBECS
https://pubmed.ncbi.nlm.nih.gov/37138207/
BMC Bioinformatics
. 2023 May 3;24(1):182. doi: 10.1186/s12859-023-05252-w.
MBECS: Microbiome Batch Effects Correction Suite
Michael Olbrich 1 2 3, Axel KÃ¼nstner # 4 5, Hauke Busch # 6
https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05252-w
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10155362/

### PLSDAbatch
https://pubmed.ncbi.nlm.nih.gov/36653900/
Brief Bioinform
. 2023 Mar 19;24(2):bbac622. doi: 10.1093/bib/bbac622.
PLSDA-batch: a multivariate framework to correct for batch effects in microbiome data
Yiwen Wang 1 2, Kim-Anh LÃª Cao 2
https://academic.oup.com/bib/article/24/2/bbac622/6991121
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10025448/
Data availability
The R package â€˜PLSDAbatchâ€™ along with the case study datasets, simulations and all analyses are fully reproducible and available on GitHub: https://github.com/EvaYiwenWang/PLSDAbatch.

review
https://pubmed.ncbi.nlm.nih.gov/31776547/
Brief Bioinform
. 2020 Dec 1;21(6):1954-1970. doi: 10.1093/bib/bbz105.
Managing batch effects in microbiome data
Yiwen Wang 1, Kim-Anh LÃªCao 1
https://academic.oup.com/bib/article/21/6/1954/5643537
Data Availability Statement
All analyses are fully reproducible and available as a bookdown document on the gitHub link, https://evayiwenwang.github.io/Managing_batch_effects/.

https://evayiwenwang.github.io/Managing_batch_effects/
Managing Batch Effects in Microbiome Data
Yiwen Wang, Kim-Anh LÃª Cao
2019-10-22

https://colil.dbcls.jp/browse/papers/31776547/1/100/year
Managing batch effects in microbiome data.
Wang Y, LeCao KA Brief Bioinform. 2020 Dec 1;21(6):1954-1970. PMID:31776547
    is cited by the following papers.

### ConQuR
### 2022-09-15
3 years ago
https://github.com/wdl2459/ConQuR
https://wdl2459.github.io/ConQuR/ConQuR.Vignette.html
https://pubmed.ncbi.nlm.nih.gov/36109499/
Nat Commun
. 2022 Sep 15;13(1):5418. doi: 10.1038/s41467-022-33071-9.
Batch effects removal for microbiome data via conditional quantile regression
Wodan Ling 1, Jiuyao Lu 2, Ni Zhao 3, Anju Lulla 4, Anna M Plantinga 5, Weijia Fu 6, Angela Zhang 1 6, Hongjiao Liu 1 6, Hoseung Song 1, Zhigang Li 7, Jun Chen 8, Timothy W Randolph 1, Wei Li A Koay 9 10, James R White 11, Lenore J Launer 12, Anthony A Fodor 13, Katie A Meyer 4, Michael C Wu 14 15
https://www.nature.com/articles/s41467-022-33071-9
https://pmc.ncbi.nlm.nih.gov/articles/PMC9477887/
Code availability
The R package ConQuR47 is available at https://github.com/wdl2459/ConQuR in formats appropriate for Macintosh, Windows, or Linux systems. A vignette demonstrating use of the package (a full analysis pipeline, including the standard fitting strategy, penalized fitting strategy, the fine-tuned result, and investigations on the original and batch-removed taxa read count tables) is included (https://wdl2459.github.io/ConQuR/ConQuR.Vignette.html).

11:10 PM Â· Apr 22, 2023
https://x.com/AhmedMoustafa/status/1649777593330159619
Ahmed Moustafa on X: "Pretty good job reducing batch effect in #microbiome data by ConQuR in a single line of R code ğŸ‘ŒğŸ» #RStats #Bioinformatics https://t.co/gyk8lSHUYt https://t.co/AJqyYDd1j4" / X


### 2021-05-18
https://pubmed.ncbi.nlm.nih.gov/34006335/
Review Microbiome
. 2021 May 18;9(1):113. doi: 10.1186/s40168-021-01059-0.
Identifying biases and their potential solutions in human microbiome studies
Jacob T Nearing 1, AndrÃ© M Comeau 2, Morgan G I Langille 3 4
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01059-0
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8132403/

1:15 AM Â· May 23, 2021
https://twitter.com/phylogenomics/status/1396137556345974788
Jonathan Eisen on X: "This is really a phenomenally useful paper that I think everyone interested in #microbiome studies should read: "Identifying biases and their potential solutions in human microbiome studies" by @JTNearing et al. https://t.co/dhWrS1sj1E https://t.co/Icn6HWhkrN" / X


### 

https://www.ncbi.nlm.nih.gov/pubmed/30388197
Bioinformatics. 2019 Jul 1;35(13):2348. doi: 10.1093/bioinformatics/bty874.
Batch effects correction for microbiome data with Dirichlet-multinomial regression.
Dai Z, Wong SH, Yu J, Wei Y.

8xè‡¨åºŠç ”ç©¶ã®ç´„2,000xç³ä¾¿ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ãƒ¡ã‚¿ã‚¢ãƒŠãƒªã‚·ã‚¹ã«ã‚ˆã‚Šã€è¤‡æ•°ã®ç–¾æ‚£ï¼ç‰¹å®šã®ç—…æ°—ã«é–¢é€£ã™ã‚‹ãƒ’ãƒˆè…¸å†…ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ã®æ©Ÿèƒ½(KEGG annotated gene families)ã‚’åŒå®šã€‚ãƒãƒƒãƒåŠ¹æœ(batch effects)ã«è¨€åŠ
https://www.ncbi.nlm.nih.gov/pubmed/31120034
mSystems. 2019 May 14;4(4). pii: e00332-18. doi: 10.1128/mSystems.00332-18.
A Metagenomic Meta-analysis Reveals Functional Signatures of Health and Disease in the Human Gut Microbiome.
Armour CR1,2, Nayfach S3,4, Pollard KS4,5,6, Sharpton TJ7,8.
https://msystems.asm.org/content/4/4/e00332-18
"certain study effects (i.e., batch effects) are unlikely to be the source of variance between samples."

----------
## meta-analysis
https://ja.wikipedia.org/wiki/ãƒ¡ã‚¿ã‚¢ãƒŠãƒªã‚·ã‚¹

https://github.com/haruosuz/statistics#meta-analysis

https://twitter.com/search?f=tweets&q=microbiome%20meta-analysis

----------
### metamicrobiomeR

https://github.com/haruosuz/r4bioinfo/blob/master/R_microb/README.md#metamicrobiomer

----------

https://twitter.com/EricTopol/status/1112741724302893056
Eric Topol on Twitter: "It's not enough to characterize the gut #microbiome by bacterial content. How #metagenomics tracks with colon #cancer, specific signatures https://t.co/yIH1TeYeVg https://t.co/jqRIcffhGZ @NatureMedicineâ€¦ https://t.co/VfDiUXjqvO"
11:41 AM - 1 Apr 2019

https://www.nature.com/articles/s41591-019-0406-6
Meta-analysis of fecal metagenomes reveals global microbial signatures that are specific for colorectal cancer | Nature Medicine
Here, a meta-analysis of eight geographically and technically diverse fecal shotgun metagenomic studies of colorectal cancer (CRC, nâ€‰=â€‰768), which was controlled for several confounders, identified a core set of 29 species significantly enriched in CRC metagenomes (false discovery rate (FDR)â€‰<â€‰1â€‰Ã—â€‰10âˆ’5). 

https://www.nature.com/articles/s41591-019-0405-7
Metagenomic analysis of colorectal cancer datasets identifies cross-cohort microbial diagnostic signatures and a link with choline degradation | Nature Medicine
We performed a meta-analysis of five publicly available datasets and two new cohorts and validated the findings on two additional cohorts, considering in total 969â€‰fecal metagenomes. 

### Qiita
### 2018-10

https://github.com/qiita-spots/qiita

https://pubmed.ncbi.nlm.nih.gov/30275573/
Meta-Analysis Nat Methods
. 2018 Oct;15(10):796-798. doi: 10.1038/s41592-018-0141-9. Epub 2018 Oct 1.
Qiita: rapid, web-enabled microbiome meta-analysis
Antonio Gonzalez 1, Jose A Navas-Molina 1 2 3, Tomasz Kosciolek 1, Daniel McDonald 1, Yoshiki VÃ¡zquez-Baeza 1, Gail Ackermann 1, Jeff DeReus 1, Stefan Janssen 1, Austin D Swafford 4, Stephanie B Orchanian 4, Jon G Sanders 1, Joshua Shorenstein 1 5, Hannes Holste 1 2, Semar Petrus 6, Adam Robbins-Pianka 7, Colin J Brislawn 8, Mingxun Wang 9, Jai Ram Rideout 10, Evan Bolyen 10, Matthew Dillon 10, J Gregory Caporaso 10 11, Pieter C Dorrestein 1 4 9, Rob Knight 12 13 14
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6235622/
Despite these challenges, meta-analyses of microbiomes have a rich history of success, identifying the major global drivers of diversity in microbial communities4, characterizing the evolution of the vertebrate gut microbiome5, and surveying specialized fields such as the built environment6. Meta-analyses also enable scientists to identify important biases such as DNA extraction, primers, or analytical pipelines7,8, which need to be controlled to generate biological discoveries.

10:30 PM Â· Oct 4, 2018
https://twitter.com/strnr/status/1047841444214558720
Stephen Turner on Twitter: "Qiita (pronounced chee-tah): rapid, web-enabled microbiome meta-analysis Paper: https://t.co/aXTv0cTCND Web: https://t.co/XfwpO4bP0I Docs: https://t.co/CKEEffUSCW Source: https://t.co/HWOT2qYHeZ https://t.co/d13hW5GJ3n" / Twitter

6:44 AM Â· Oct 2, 2018
https://twitter.com/gibbological/status/1046878477981343744
Sean Gibbons ğŸ¦ ğŸ’© @gibbological@mstdn.science on Twitter: "Qiita: rapid, web-enabled microbiome meta-analysis https://t.co/4Osl11rR8X from @KnightLabNews @gregcaporaso and @Pdorrestein1 labs. Useful tool for all us 'data parasites' :)" / Twitter
https://twitter.com/inschool4life/status/1047107731327467521
Did they comment on batch effects?

### almlab
### 2018-04-23

https://be.mit.edu/directory/eric-alm
Eric Alm
http://almlab.mit.edu/

2:55 AM Â· Apr 24, 2018
https://twitter.com/gibbological/status/988476547614105601
Sean Gibbons ğŸ¦ ğŸ’© @gibbological@mstdn.science on Twitter: "Befuddled by batch effects? Check out our simple, non-parametric solution for 16S case-control studies in @PLOSCompBiol https://t.co/grMDm7SIQl -- method available as a @qiime2 plugin thanks to @cduvallet" / Twitter

https://pubmed.ncbi.nlm.nih.gov/29684016/
Comparative Study PLoS Comput Biol
. 2018 Apr 23;14(4):e1006102. doi: 10.1371/journal.pcbi.1006102. eCollection 2018 Apr.
Correcting for batch effects in case-control microbiome studies
Sean M Gibbons 1 2 3, Claire Duvallet 1 2, Eric J Alm 1 2 3

April 17, 2018
https://cduvallet.github.io/posts/2018/04/microbiome-data
An empirical analysis of microbiome data availability - Claire Duvallet

https://www.ncbi.nlm.nih.gov/pubmed/29209090
Nat Commun. 2017 Dec 5;8(1):1784. doi: 10.1038/s41467-017-01973-8.
Meta-analysis of gut microbiome studies identifies disease-specific and shared responses.
Duvallet C1,2, Gibbons SM1,2,3, Gurry T1,2,3, Irizarry RA4,5, Alm EJ6,7,8.
We perform a cross-disease meta-analysis of these studies using standardized methods. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5716994/
Code availability
The code to reproduce all of the analyses in this paper is available at https://github.com/cduvallet/microbiomeHD. 

3:05 AM Â· Dec 7, 2017
https://twitter.com/Kazu_Kasahara/status/938469390739234816
Kazuyuki Kasahara on Twitter: "æµè¡Œã—ã¦ã„ã‚‹ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ç ”ç©¶ã«è­¦é˜ã‚’é³´ã‚‰ã™è«–æ–‡ã€‚28å ±ã®ç ”ç©¶ã€10ç–¾æ‚£ã®ãƒ¡ã‚¿è§£æã‹ã‚‰ã€ç—…æ°—ã§èªã‚ã‚‹"dysbiosis"ã®å¤šãã¯ç–¾æ‚£ç‰¹ç•°çš„ã§ã¯ãªãã€ãƒã‚¤ã‚ªãƒãƒ¼ã‚«ãƒ¼ã«ãªã‚Šé›£ã„ã€‚ https://t.co/AVJHhcCCcI" / Twitter

5:45 AM Â· Dec 6, 2017
https://twitter.com/cduvallet/status/938147253512015873
Claire Duvallet on Twitter: "The meta-analysis is out in @NatureComms! We found that diseases are characterized by consistent types of microbial community shifts, and that lots of bacteria respond non-specifically to multiple diseases. @gibbological @ThomasGurry @ejalm @rafalab https://t.co/XJxmxApgXv" / Twitter

### 2018-06
https://twitter.com/ayaKT/status/1005378142146457600
ayaKT on Twitter: "~19000äººè¦æ¨¡ã®microbiome-GWAS ã«ç€æ‰‹ã—ã¾ã—ãŸè«–æ–‡ã€‚ã©ã‚“ãªè§£æçµæœãŒå‡ºã¦ãã‚‹ã®ã‹æ¥½ã—ã¿ã€‚ ã„ã‚ã‚“ãª16Sã‚·ãƒ¼ã‚±ãƒ³ã‚¹æ‰‹æ³•ã‚’æ¯”è¼ƒå¯èƒ½ã«ã™ã‚‹è€ƒãˆã‹ãŸã‚‚ã€å‹‰å¼·ã«ãªã‚‹ã€‚fmfm https://t.co/hNMXyZHzWj"
5:16 AM - 9 Jun 2018

https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0479-3
Meta-analysis of human genome-microbiome association studies: the MiBioGen consortium initiative | Microbiome
We are now in the process of benchmarking the association tests and performing meta-analyses of genome-wide associations. All pipeline and summary statistics results will be shared using public data repositories.

### 2017-12-22
https://www.ncbi.nlm.nih.gov/pubmed/29273717
Nat Commun. 2017 Dec 22;8(1):2260. doi: 10.1038/s41467-017-02209-5.
Strain profiling and epidemiology of bacterial species from metagenomic sequencing.
Albanese D1, Donati C2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5741664/
- Here we present StrainEst, a novel, reference-based method that uses the single-nucleotide variants (SNV) profiles of the available genomes of selected species to determine the number and identity of coexisting strains and their relative abundances in mixed metagenomic samples.
- meta-analyses

### 2017-11-20
https://twitter.com/NatureMicrobiol/status/932704694912389120
Nature Microbiology on Twitter: ".@drkellysierra & co: Integration and meta-analysis of global #soil datasets uncover patterns of #microbial structure,â€¦ https://t.co/QB51kFBtK5"
3:18 PM - 20 Nov 2017

https://pubmed.ncbi.nlm.nih.gov/29158606/
Nat Microbiol
. 2018 Feb;3(2):189-196. doi: 10.1038/s41564-017-0062-x. Epub 2017 Nov 20.
Detecting Macroecological Patterns in Bacterial Communities Across Independent Studies of Global Soils
- Whereas previous meta-analysis efforts have focused on bacterial diversity measures or abundances of major taxa, we show that disparate amplicon sequence data can be combined at the taxonomy-based level to assess bacterial community structure.

### 2016
https://www.ncbi.nlm.nih.gov/pubmed/27400279
PLoS Comput Biol. 2016 Jul 11;12(7):e1004977. doi: 10.1371/journal.pcbi.1004977. eCollection 2016 Jul.
Machine Learning Meta-analysis of Large Metagenomic Datasets: Tools and Biological Insights.
Pasolli E1, Truong DT1, Malik F2, Waldron L2, Segata N1.
http://segatalab.cibio.unitn.it/tools/metaml
MetAML - Metagenomic prediction Analysis based on Machine Learning
æ©Ÿæ¢°å­¦ç¿’

### 2015
https://www.ncbi.nlm.nih.gov/pubmed/26459172
Microbiome. 2015 Oct 13;3:49. doi: 10.1186/s40168-015-0108-3.
Microbiota of the indoor environment: a meta-analysis.
Adams RI1, Bateman AC2, Bik HM3,4, Meadow JF5.
å®¤å†…ç’°å¢ƒã®å¾®ç”Ÿç‰©ç¾¤é›†ï¼šãƒ¡ã‚¿ã‚¢ãƒŠãƒªã‚·ã‚¹

----------
## mothur-qiime
https://mothur.org

https://www.pediatricsurgery.site/entry/2017/11/19/200241
mothuré–‹ç™ºè€…ã«ã‚ˆã‚‹mothurã¨QIIMEã®æ¯”è¼ƒ - Note of Pediatric Surgery

https://biosciencedbc.jp/gadget/human/170831_mori_170830.pdf
ãƒ¡ã‚¿16Sè§£æåˆ©ç‚¹
ãƒ»ãƒã‚·ãƒ³ãƒã‚šãƒ¯ãƒ¼ã¯å°‘ãªãã¦æ¸ˆã¿ã€è§£æãƒ„ãƒ¼ãƒ«ã‚‚æ™®åŠ(QIIMEãƒ»mothurç­‰)

https://www.jstage.jst.go.jp/article/jim/29/3/29_135/_article/-char/ja/
ç·ã€€ã€€èª¬ ï¼œç‰¹é›†ï¼šè…¸å†…ç´°èŒå¢Microbiotaã®åˆ†å­ç”Ÿç‰©å­¦çš„è§£ææ³•ã®æˆæœã¨æœªæ¥ï¼
é€²åŒ–ã™ã‚‹æ¬¡ä¸–ä»£ã‚·ãƒ¼ã‚±ãƒ³ã‚µãƒ¼ã«ã‚ˆã‚‹è…¸å†…ç´°èŒå¢ã®è§£æ
æ± ä½³é‚£å­, ä¸­å±± äºŒéƒ
2015 å¹´ 29 å·» 3 å· p. 135-144
https://www.jstage.jst.go.jp/article/jim/29/3/29_135/_pdf
ãƒ†ã‚™ãƒ¼ã‚¿ã‚»ãƒƒãƒˆã¯,16S rRNA é…åˆ—ã‹ã‚‰èŒå¢ãƒ†ã‚™ãƒ¼ã‚¿ã‚’ å¾—ã‚‹ãŸã‚ã®ã‚ªãƒ¼ãƒ•ã‚šãƒ³ã‚½ãƒ¼ã‚¹ã‚½ãƒ•ãƒˆã‚¦ã‚§ã‚¢ã‚’é–‹ç™ºã—ã¦ã„ ã‚‹ Mothur ãƒ•ã‚šãƒ­ã‚·ã‚™ã‚§ã‚¯ãƒˆ(4)ã‹ã‚™,Silva ã® 16S rRNA ã®å…¨é•·é…åˆ—ãƒ†ã‚™ãƒ¼ã‚¿ãƒ˜ã‚™ãƒ¼ã‚¹ã‹ã‚‰ 14,956 æœ¬ã‚’ç¶²ç¾…çš„ã«æŠ½å‡º ã—ä½œæˆã—ãŸãƒ†ã‚™ãƒ¼ã‚¿ã‚»ãƒƒãƒˆ silva.bacteria.fasta ã‚’ç”¨ã„ãŸ(http://www.mothur.org/wiki/Silva_reference_files).

ã“ã“ã¦ã‚™ ã¯,Mothur åŒæ§˜,èŒå¢è§£æã®ã‚ªãƒ¼ãƒ•ã‚šãƒ³ã‚½ãƒ¼ã‚¹ã‚½ãƒ•ãƒˆã‚¦ã‚§ ã‚¢ã¦ã‚™ã‚ã‚‹ QIIME(Quantitative Insights into Microbial Ecology, http://qiime.org/)(5) ã® assign_taxonomy.py ã®ãƒ•ã‚šãƒ­ã‚¯ã‚™ãƒ©ãƒ ã® uclust ã®ã‚¢ãƒ«ã‚³ã‚™ãƒªã‚¹ã‚™ãƒ ã‚’ç”¨ã„ãŸ.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3775626/
PeerJ. 2013; 1: e157.
Abundance-weighted phylogenetic diversity measures distinguish microbial community states and are robust to sampling depth

In microbial ecology studies, the most commonly used ways of investigating alpha (within-sample) diversity are either to apply non-phylogenetic measures such as Simpsonâ€™s index to Operational Taxonomic Unit (OTU) groupings, or to use classical phylogenetic diversity (PD), which is not abundance-weighted. 

As of the publication of this paper, no abundance-weighted phylogenetic alpha diversity measures are implemented in either mothur (Schloss et al., 2009) or QIIME (Caporaso et al., 2010), two of the most popular tools for analysis of microbial ecology data.

----------
## 16S

Posted October 13, 2018.
https://www.biorxiv.org/content/10.1101/441576v1
Nomenclature Errors in Public 16S rRNA Gene Reference Databases

https://pubmed.ncbi.nlm.nih.gov/29688343/
FEMS Microbiol Lett
. 2018 May 1;365(10):fny104. doi: 10.1093/femsle/fny104.
A town on fire! Integrating 16S rRNA gene amplicon analyses into an undergraduate microbiology lecture class
Tammy C Tobin 1, Ashley Shade 2
https://academic.oup.com/femsle/article/365/10/fny104/4980908?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5954299/
In this module, microbiology students used Quantitative Insights into Microbial Ecology (QIIME) to perform taxonomic, phylogenetic and statistical analyses on bacterial communities from three hot mine fire-impacted surface soils using 16S rRNA gene amplicon sequences. 

https://twitter.com/BioMickWatson

https://www.pediatricsurgery.site/entry/2017/11/19/200241
mothuré–‹ç™ºè€…ã«ã‚ˆã‚‹mothurã¨QIIMEã®æ¯”è¼ƒ - Note of Pediatric Surgery

mothurã§ã¯RDPã€SILVAã€greengenesã‚’ä½¿ç”¨ã™ã‚‹ã“ã¨ãŒã§ãã‚‹

https://www.ncbi.nlm.nih.gov/pubmed/28705636
J Biotechnol. 2017 Nov 10;261:2-9. doi: 10.1016/j.jbiotec.2017.07.010. Epub 2017 Jul 10.
A review of bioinformatics platforms for comparative genomics. Recent developments of the EDGAR 2.0 platform and its utility for taxonomic and phylogenetic studies.
Yu J1, Blom J2, Glaeser SP3, Jaenicke S4, Juhre T4, Rupp O4, Schwengers O4, SpÃ¤nig S4, Goesmann A4.
https://www.sciencedirect.com/science/article/pii/S0168165617315225
But, because it is well known that the phylogenetic resolution of the primary phylogenetic marker, the 16S rRNA gene, is limited at the lower taxonomic levels, the integration of a genome sequence based phylogeny into the polyphasic approach of the prokaryotic taxonomy and systematics is the clear consequence in the genome area (Chun and Rainey, 2014, Ramasamy et al., 2014, Oren and Garrity, 2014, Vandamme and Peeters, 2014, Sangal et al., 2016).

https://www.ncbi.nlm.nih.gov/pubmed/28662636
BMC Genomics. 2017 Jun 29;18(1):499. doi: 10.1186/s12864-017-3888-y.
Relating genomic characteristics to environmental preferences and ubiquity in different microbial taxa.
Cobo-SimÃ³n M1, Tamames J2.

https://www.researchgate.net/post/Is_Greengenes_or_SILVA_better_for_bacterial_microbiome_studies
Is Greengenes or SILVA better for bacterial microbiome studies?

https://www.ncbi.nlm.nih.gov/pubmed/28361695
BMC Genomics. 2017 Mar 14;18(Suppl 2):114. doi: 10.1186/s12864-017-3501-4.
SILVA, RDP, Greengenes, NCBI and OTT - how do these taxonomies compare?
BalvoÄiÅ«tÄ— M1, Huson DH2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5374703/
For the comparisons we used a taxonomy associated with the Greengenes database as released on May 2013. Although Greengenes is still included in some metagenomic analyses packages, for example QIIME [22], it has not been updated for the last three years.

Mar 2017
https://forum.qiime2.org/t/classification-using-greengenes-v-silva/409
Classification using GreenGenes v. SILVA - User support - QIIME 2 Forum

https://www.ncbi.nlm.nih.gov/pubmed/27166378
Nucleic Acids Res. 2016 Jun 20;44(11):5022-33. doi: 10.1093/nar/gkw396. Epub 2016 May 10.
Phylogeny-aware identification and correction of taxonomically mislabeled sequences.
Kozlov AM1, Zhang J2, Yilmaz P3, GlÃ¶ckner FO4, Stamatakis A5.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4914121/
First, we evaluate taxonomic annotations of type strains only, using the same sequence set and alignment (LTP v123) for all four databases (data sets GG13_T, LTP123_T, RDP11_T and SLV123_T in Table â€‹Table1).1). 

https://mothur.org/blog/2015/No-greengenes-hasnt-improved/
No, greengenes' reference alignment hasn't improved

- SILVA (good for alignment and classification)
- greengenes (good for alignment and classification)
- RDP (good for classification)

https://www.ncbi.nlm.nih.gov/pubmed/27603265
Genome. 2016 Oct;59(10):783-791. Epub 2016 May 11.
bcgTree: automatized phylogenetic tree building from bacterial core genomes.
Ankenbrand MJ1,1, Keller A1,1.
The need for multi-gene analyses in scientific fields such as phylogenetics and DNA barcoding has increased in recent years. In particular, these approaches are increasingly important for differentiating bacterial species, where reliance on the standard 16S rDNA marker can result in poor resolution. 

https://www.ncbi.nlm.nih.gov/pubmed/23460914
PLoS One. 2013;8(2):e57923. doi: 10.1371/journal.pone.0057923. Epub 2013 Feb 27.
The variability of the 16S rRNA gene in bacterial genomes and its consequences for bacterial community analyses.
VÄ›trovskÃ½ T1, Baldrian P.


2017/08/30
https://www.aist.go.jp/aist_j/press_release/pr2017/pr20170830/pr20170830.html
ç”£ç·ç ”ï¼šé€²åŒ–ç³»çµ±åˆ†é¡ã®æŒ‡æ¨™ã¨ãªã‚‹16S rRNAéºä¼å­ã®é€²åŒ–çš„ãªä¸­ç«‹æ€§ã‚’å®Ÿé¨“çš„ã«è¨¼æ˜

https://www.nite.go.jp/nbrc/safety/risk_assessment.html
å¾®ç”Ÿç‰©åˆ©ç”¨ã«ãŠã‘ã‚‹ãƒªã‚¹ã‚¯è©•ä¾¡ã«é–¢ã™ã‚‹æŠ€è¡“æƒ…å ±ã®æä¾› | ãƒã‚¤ã‚ªãƒ†ã‚¯ãƒãƒ­ã‚¸ãƒ¼ | è£½å“è©•ä¾¡æŠ€è¡“åŸºç›¤æ©Ÿæ§‹
DDHã«ãŠã‘ã‚‹èŒç¨®åˆ¤å®šã®é–¾å€¤ã§ã‚ã‚‹é¡ä¼¼åº¦70%ã¯ã€ANIã«ãŠã„ã¦95-96%ã«ç›¸å½“ã™ã‚‹ã¨å ±å‘Šã•ã‚Œã¦ã„ã¾ã™ã€‚ã“ã®ã“ã¨ã‹ã‚‰ã€è¿‘å¹´ã§ã¯ã€DDHã®ä»£æ›¿ã¨ã—ã¦ANIã«åŸºã¥ãæ–°ç¨®ã®ææ¡ˆãŒè¡Œã‚ã‚Œã¦ãã¦ã„ã¾ã™(Baek et al. 2015, IJSEM 65: 504; Shahraki et al. 2015, IJSEM 65: 504)ã€‚
OTUã¨ã¯ã€ã‚ã‚‹ä¸€å®šä»¥ä¸Šã®é¡ä¼¼æ€§ï¼ˆä¸€èˆ¬çš„ã«ã¯96-97%ï¼‰ã‚’æŒã¤é…åˆ—åŒå£«ã‚’ä¸€ã¤ã®èŒç¨®ã®ã‚ˆã†ã«æ‰±ã†ãŸã‚ã®æ“ä½œä¸Šã®åˆ†é¡å˜ä½ã§ã™ã€‚

ãƒã‚¤ã‚ªãƒ¬ãƒ¡ãƒ‡ã‚£ã‚¨ãƒ¼ã‚·ãƒ§ãƒ³ã«ãŠã‘ã‚‹ç”Ÿæ…‹ç³»å½±éŸ¿è©•ä¾¡æ‰‹æ³•	Illuminaç¤¾MiSeqã‚·ãƒ¼ã‚±ãƒ³ã‚µãƒ¼ã‚’ç”¨ã„ãŸã€ç’°å¢ƒä¸­ã®å¾®ç”Ÿç‰©ç”Ÿæ…‹ç³»è§£æã®å®Ÿé¨“æ‰‹æ³•ãŠã‚ˆã³ãƒ‘ã‚¤ãƒ—ãƒ©ã‚¤ãƒ³ãƒ„ãƒ¼ãƒ«ã€ŒQIIMEã€ã‚’ç”¨ã„ãŸãƒ‡ãƒ¼ã‚¿è§£ææ‰‹æ³•ã®æ‰‹é †æ›¸ã§ã™ã€‚ 2016å¹´3æœˆ25æ—¥ https://www.nite.go.jp/data/000081956.pdf

https://seikagaku.jbsoc.or.jp/10.14952/SEIKAGAKU.2015.870475/data/index.html
Journal of Japanese Biochemical Society 87(4): 475-477 (2015)
16Sãƒªãƒœã‚½ãƒ¼ãƒ RNAã®æ°´å¹³ä¼æ’­å®Ÿé¨“ã‹ã‚‰ã¿ãˆã¦ãã‚‹ãƒªãƒœã‚½ãƒ¼ãƒ ã®å¯å¡‘æ€§
ä½ƒ ç¾é›ª1,2ï¼Œå®®å´ å¥å¤ªéƒ1,3

https://www.jstage.jst.go.jp/article/kagakutoseibutsu/52/2/52_70/_pdf
åŒ–å­¦ã¨ç”Ÿç‰© Vol. 52, No. 2, 2014
16S rRNAéºä¼å­ã®ã€Œæ°´å¹³ä¼æ’­ã€
ç•°ç¨®16S rRNAã«ã‚ˆã‚‹éºä¼çš„ç›¸è£œ

2014
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3903645/
MetaMetaDB: A Database and Analytic System for Investigating Microbial Habitability

Therefore, MetaMetaDB chooses the hits exhibiting above 97%, 95%, 90%, 85%, and 80% identity from the search result list, which correspond approximately to the taxonomic levels of species, genus, family, order, and class [1], respectively, 

02/02/13
http://www.jarmam.gr.jp/situmon/16sr_rna.html
16s rRNA ã®ç›¸åŒæ€§ã¨åŒå®šç²¾åº¦ã®é–¢ä¿‚ã¯???
ä¸€èˆ¬çš„ã«ã¯, 97%ä»¥ä¸Šã®ç›¸åŒæ€§ãŒã‚ã‚Œã°é¡ç¸é–¢ä¿‚ãŒã‚ã‚Šã€99%ä»¥ä¸Šã§ã‚ã‚Œã°åŒç¨®ã§ã‚ã‚‹å¯èƒ½æ€§ãŒé«˜ã„ã¨ã—ã¦ã„ã¾ã™ã€‚
æ¥µç«¯ãªä¾‹ã§ã¯, 100%ã®ç›¸åŒæ€§ãŒèªã‚ã‚‰ã‚Œã¦ã‚‚, å±ãƒ¬ãƒ™ãƒ«ã§ç•°ãªã‚‹ä¾‹ã•ãˆç¤ºã•ã‚Œã¦ã„ã‚‹ã®ã§ã™ã€‚

2012 å¹´
https://www.jstage.jst.go.jp/article/jslab/23/1/23_24/_pdf
ç·èª¬
16S rRNA éºä¼å­ã®å¤§é‡ã‚·ãƒ¼ã‚¯ã‚§ãƒ³ã‚·ãƒ³ã‚°ã«ã‚ˆã‚‹ èŒå¢è§£æã®ç¾çŠ¶ã¨å•é¡Œç‚¹
æ˜¯å‰‡ æœ‰å¸Œ, Jiahui Jiang, ä¸­å±± äºŒéƒ
å±ãƒ¬ãƒ˜ã‚™ãƒ«ã¾ã¦ã‚™ã®èŒå¢ãƒ•ã‚šãƒ­ãƒ•ã‚¡ã‚¤ãƒ«ã¦ã‚™ã‚ ã‚Œã¯ã‚™ã€Ribosomal Database Project I(I ä»¥ä¸‹ã€RDP II)ã®ã‚¦ã‚§ ãƒ•ã‚™ã‚µã‚¤ãƒˆã«ã‚ã‚‹ RDP classifier ã‚’ç”¨ã„ã‚‹ã“ã¨ã¦ã‚™å®¹æ˜“ã«èŒå¢ ãƒ•ã‚šãƒ­ãƒ•ã‚¡ã‚¤ãƒ«ã«å¤‰æ›ã™ã‚‹ã“ã¨ã‹ã‚™ã¦ã‚™ãã‚‹ã€‚
ä¸€æ–¹ã€ç¨®ãƒ¬ãƒ˜ã‚™ãƒ«ã¦ã‚™ã®èŒå¢ãƒ†ã‚™ãƒ¼ã‚¿ã‚’å¾—ã‚‹ã“ã¨ã¯éå¸¸ã«å›°é›£ ã¦ã‚™ã‚ã‚‹ã€‚å®Ÿéš›ã«ã€ç†è«–ä¸Š 16S rRNA éºä¼å­ã®éƒ¨åˆ†é…åˆ—ã‹ ã‚‰ç¨®ã‚’ç‰¹å®šã™ã‚‹ã“ã¨ã¯ä¸å¯èƒ½ã¦ã‚™ã‚ã‚Šã€

http://www.aist.go.jp/aist_j/press_release/pr2012/pr20121030/pr20121030.html
ç”£ç·ç ”ï¼šç”Ÿç‰©ç¨®ã‚’è¶ŠãˆãŸ16S rRNAéºä¼å­ã®æ©Ÿèƒ½ç›¸è£œæ€§ã‚’ç¢ºèª

2011
https://www.sbj.or.jp/wp-content/uploads/file/sbj/8912/8912_yomoyama_2.pdf
ç”Ÿç‰©å·¥å­¦åŸºç¤è¬›åº§ ãƒã‚¤ã‚ªã‚ˆã‚‚ã‚„ã¾è©± ä½•ã‹ã‚‰å§‹ã‚ã‚ˆã† å¾®ç”Ÿç‰©ã®åŒå®š ç´°èŒãƒ»ã‚¢ãƒ¼ã‚­ã‚¢ç·¨
æµœç”° ç››ä¹‹,éˆ´æœ¨ å¥ä¸€æœ—
ç‰¹ã« 16S rRNA éºä¼å­é…åˆ—ã®ç›¸åŒå€¤ã‹ã‚™ 97% ã«æº€ãŸãªã„å ´åˆ ãªã¨ã‚™ã¯æ–°å±ã«ãªã‚‹å¯èƒ½æ€§ã‚‚ã‚ã‚‹ãŸã‚,ç³»çµ±æ¨¹ã‚’ä½œæˆã—ã¦ æ—¢çŸ¥ã®å±ã®ã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼ã«å«ã¾ã‚Œã‚‹ã®ã‹ã¨ã‚™ã†ã‹ã‚’æ…é‡ã«ç¢º èªã—ãŸã»ã†ã‹ã‚™ã‚ˆã„.

https://www.mitsui-norin.co.jp/mmid/knowledge/yokota/index3.html
MMIDï½œå¾®ç”Ÿç‰©ã‚’çŸ¥ã‚ã†ï½œå¾®ç”Ÿç‰©åˆ†é¡åŒå®šè¬›åº§
16S ï½’RNAéºä¼å­å¡©åŸºé…åˆ—ã®ç›¸åŒæ€§ãŒ98ï¼…ä»¥ä¸Šã®å ´åˆã€å¿…ãšã—ã‚‚åŒç¨®ã§ã‚ã‚‹ã¨ã¯é™ã‚‰ãšã€å±ã«ã‚ˆã£ã¦ã¯99ï¼…ä»¥ä¸Šã§ã‚ã£ã¦ã‚‚åˆ¥ç¨®ã®å ´åˆãŒã‚ã‚Šã¾ã™ã€‚

https://ja.wikipedia.org/wiki/16S_rRNAç³»çµ±è§£æ

http://www.jarmam.gr.jp/situmon3/16sr-rna.html
16S rRNAã§ã®å¾®ç”Ÿç‰©åŒå®š

----------
## amplicon-shotgun

targeted amplicon sequencing (16S, 18S, ITS) vs. shotgun sequencing (metagenomics)

https://pubmed.ncbi.nlm.nih.gov/28337071/
Review J Biomol Tech
. 2017 Apr;28(1):8-18. doi: 10.7171/jbt.17-2801-006. Epub 2017 Mar 17.
International Standards for Genomes, Transcriptomes, and Metagenomes
Christopher E Mason 1, Ebrahim Afshinnekoo 2, Scott Tighe 3, Shixiu Wu 4, Shawn Levy 5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5359768/

TABLE 3
Comparison of metagenomic assay capabilities and limitations

Data type	16S	18S	ITS	Shotgun


https://www.ncbi.nlm.nih.gov/pubmed/28761145
Sci Rep. 2017 Jul 31;7(1):6589. doi: 10.1038/s41598-017-06665-3.
Large-scale differences in microbial biodiversity discovery between 16S amplicon and shotgun sequencing.
Tessler M1,2, Neumann JS3, Afshinnekoo E4,5,6, Pineda M4,5, Hersch R7, Velho LFM8,9, Segovia BT8, Lansac-Toha FA8, Lemke M10, DeSalle R7, Mason CE11,12,13, Brugler MR14,15.

https://www.ncbi.nlm.nih.gov/pubmed/21121044
Pac Symp Biocomput. 2011:165-76.
Comparing bacterial communities inferred from 16S rRNA gene sequencing and shotgun metagenomics.
Shah N1, Tang H, Doak TG, Ye Y.

ãƒœã‚¹ãƒˆãƒ³äº¤é€šã‚·ã‚¹ãƒ†ãƒ ã®ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ 
https://github.com/haruosuz/metasub/tree/master/references#boston-subway-microbiome
https://microbe.net/2016/06/30/microbes-on-the-boston-t/
The 16S and shotgun results were mostly complementary. Both showed high abundances of Staphylococcus, Corynebacterium, and Streptococcus. The main difference was that the V4 primers could not detect Propionibacterium acnes: these primers are not ideal for skin microbial communities â€“ which is what the subway most resembled (based on Microbial SourceTracker and correlations in abundance with skin taxa). 

https://twitter.com/Tyu_Shi/status/982940944495403009
Tyu_Shi on Twitter: "ã—ãŸãŒã£ã¦ã€16S rRNAéºä¼å­ã®ã¿ã®è§£æã‚’ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã¨å‘¼ã¶ã®ã¯ã€ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ æ¥­ç•Œã§å¸¸è­˜ã§ã¯ã‚ã‚Šã¾ã›ã‚“ã€‚æ¦‚å¿µã¨ã—ã¦ã¯èª¤ç”¨ã ã¨å€‹äººçš„ã«ã¯è€ƒãˆã¦ãŠã‚Šã¾ã™ã—ã€å¤ãã‹ã‚‰ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã«é–¢ã‚ã£ã¦ããŸUC Davisã®J. Eisenç­‰ã®ç ”ç©¶è€…ã‚‚ã€ãŸã³ãŸã³ãã®èª¤ç”¨ã‚’æ‰¹åˆ¤ã—ã¦ãŠã‚Šã¾ã™ã€‚â€¦ https://t.co/SpajjZUOIW"
7:19 AM - 8 Apr 2018

https://twitter.com/Tyu_Shi/status/248334905816276993
Tyu_Shi on Twitter: "Many INSDC entry of 16S rRNA amplicon study use terms "metagenome". troublesome! #GSC14"
4:17 AM - 19 Sep 2012

https://twitter.com/phylogenomics/status/966887584608567297
Jonathan Eisen on Twitter: "Not sure where you are going with this but I don't think there can be any debate here - the term "metagenome" and its variants like "metagenomics" clearly refer to genome-scale studies and thus there is simply no way a 16S study should be called "metagenomics"â€¦ https://t.co/ScdBSzIzlG"
11:08 PM - 22 Feb 2018

https://phylogenomics.blogspot.com/2012/08/referring-to-16s-surveys-as.html
The Tree of Life: Referring to 16S surveys as "metagenomics" is misleading and annoying #badomics #OmicMimicry

----------

### 2018-06-05

https://pubmed.ncbi.nlm.nih.gov/29871915/
mBio
. 2018 Jun 5;9(3):e00525-18. doi: 10.1128/mBio.00525-18.
Identifying and Overcoming Threats to Reproducibility, Replicability, Robustness, and Generalizability in Microbiome Research
Patrick D Schloss 1
https://journals.asm.org/doi/10.1128/mBio.00525-18
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5989067/

 replicability
 confounding variable
 samples are not randomized across sequencing runs.
 batch effects

----------

### 2018-03-19
https://pubmed.ncbi.nlm.nih.gov/29427429/
Review Appl Environ Microbiol
. 2018 Mar 19;84(7):e02627-17. doi: 10.1128/AEM.02627-17. Print 2018 Apr 1.
The Madness of Microbiome: Attempting To Find Consensus "Best Practice" for 16S Microbiome Studies
Jolinda Pollock # 1 2, Laura Glendinning # 2, Trong Wisedchanwet 2, Mick Watson 2
https://journals.asm.org/doi/full/10.1128/AEM.02627-17
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861821/
Of the three most commonly used alignments which are guided by secondary structure (i.e., Greengenes [73], RDP [74], and SILVA [75]), the Greengenes alignment was observed to be of poor quality, leading to significantly greater richness and diversity estimates.

https://www.microbe.net/2018/03/03/the-attempt-to-find-best-practice-for-16s-studies/
The attempt to find â€œbest practiceâ€ for 16S studies â€“ microBEnet: the microbiology of the Built Environment network

12:34 AM Â· Feb 27, 2018
https://twitter.com/BioMickWatson/status/968147242401058817
Mick W@tson ğŸ„ on Twitter: "If you're working in 16S microbiome research, please read our paper which tries to suggest best practices! https://t.co/XWVRxUKFRW" / Twitter


----------
## Hi-C

https://phasegenomics.com/hi-c-solves-the-problem-of-linking-plasmids-to-host-cells-in-a-mixed-sample/
Hi-C solves the problem of linking plasmids to hosts in microbiome samples - Phase Genomics

https://filgen.jp/Product/Bioscience5-seq/index50.html
æ¬¡ä¸–ä»£ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚·ãƒ³ã‚°(æ¬¡ä¸–ä»£ã‚·ãƒ¼ã‚±ãƒ³ã‚·ãƒ³ã‚°/NGS)ã‚µãƒ¼ãƒ“ã‚¹ / Proximo(TM)Hi-Cã‚²ãƒãƒ ã‚¹ã‚­ãƒ£ãƒ•ã‚©ãƒ¼ãƒ«ãƒ‡ã‚£ãƒ³ã‚°å—è¨—è§£æã€€ï¼šï¼šãƒ•ã‚£ãƒ«ã‚¸ã‚§ãƒ³æ ªå¼ä¼šç¤¾ï¼šï¼š

https://filgen.jp/Product/Bioscience4/Phase_Genomics/index.html
Phase Genomicsç¤¾è£½ Hi-Cè§£æç”¨ã‚µãƒ³ãƒ—ãƒ«èª¿æ•´ã‚­ãƒƒãƒˆã€€ï¼šï¼šãƒ•ã‚£ãƒ«ã‚¸ã‚§ãƒ³æ ªå¼ä¼šç¤¾ï¼šï¼š

ã€ã©ã®ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ãŒã©ã®å®¿ä¸»ã‚²ãƒãƒ ã«é–¢é€£ã™ã‚‹ã‹ï¼ˆä¾‹ï¼šèŒæ ªä¸­ã®æŠ—èŒæŠµæŠ—æ€§ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®åŒå®šï¼‰ãªã©ã®ãƒ‡ãƒ¼ã‚¿ãŒå¾—ã‚‰ã‚Œã¾ã™ã€‚

ãƒ»ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ - å®¿ä¸»ã«ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã‚’å‰²ã‚Šå½“ã¦ã‚‹ã“ã¨ãŒã§ãã‚‹ã€‚

Posted November 30, 2023.
https://www.biorxiv.org/content/10.1101/2023.11.30.569469v1
Detection of rare plasmid hosts using a targeted Hi-C approach
https://www.biorxiv.org/content/10.1101/2023.11.30.569469v1.full
https://www.biorxiv.org/content/10.1101/2023.11.30.569469v1.full.pdf
Salvador Castaneda-Barba1,2,3, Benjamin J. Ridenhour1,3,4, Eva M. Top1,2,3,4 8 *â€ , Thibault
Stalder1,2,3,4 9 *â€ ,

2020
https://www.jseb.jp/ç’°å¢ƒãƒã‚¤ã‚ªãƒ†ã‚¯ãƒãƒ­ã‚¸ãƒ¼å­¦ä¼šèªŒ%E3%80%8020å·»1å·/
ç’°å¢ƒä¸­ã®å¾®ç”Ÿç‰©ç¾¤é›†å†…ã«ãŠã‘ã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ä¼æ’­ã®å®Ÿæ…‹ã®è§£æ˜ã«å‘ã‘ã¦
Behaviors of Plasmids in Various Environmental Microcosms
æ–°è°· æ”¿å·±ï¼Œé‡‘åŸ å’Œç§€
https://www.jseb.jp/wordpress/wp-content/uploads/IL0069_12_ST_shintani.indd_.pdf

4.ã€€ç’°å¢ƒç´°èŒç¾¤é›†å†…ã«ãŠã‘ã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»åŸŸ

å›³ 3ï¼Hi-C ãƒ†ã‚¯ãƒãƒ­ã‚¸ãƒ¼ã‚’åˆ©ç”¨ã—ãŸãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã‚„å¯¾è±¡éºä¼å­ã®å¸°å±æ–¹æ³•

Hi-C ãƒ†ã‚¯ãƒãƒ­ã‚¸ãƒ¼ã‚’åˆ©ç”¨ã—ãŸæ–¹æ³•ã§ã‚ã‚‹ã€‚ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰
ã¨æŸ“è‰²ä½“ DNA ã‚’ï¼Œç´°èƒå†…ã§ãƒ›ãƒ«ãƒ ã‚¢ãƒ«ãƒ‡ãƒ’ãƒ‰ã‚’ç”¨ã„ã¦
å…±æœ‰çµåˆã•ã›ãŸå¾Œï¼Œåˆ¶é™é…µç´ å‡¦ç†ã—ãŸæ–­ç‰‡ã‚’ç’°çŠ¶åŒ–ã—ï¼Œ
ãƒšã‚¢ã‚¨ãƒ³ãƒ‰ãƒªãƒ¼ãƒ‰ã‚’ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚¹ã™ã‚‹ã€‚ã“ã‚Œã‚’ï¼Œãƒ¡ã‚¿ã‚²
ãƒãƒ é…åˆ—ã¨ã™ã‚Šåˆã‚ã›ã‚‹ã“ã¨ã§ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»ã‚’åŒå®š
ã™ã‚‹ï¼ˆå›³ 3ï¼‰ 38)ã€‚ã„ãšã‚Œã‚‚ï¼Œãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã‚„è–¬å‰¤è€æ€§éºä¼
å­ã‚’ã‚‚ã¤å¾®ç”Ÿç‰©ã®ç¨®é¡ã‚’ä¸€å®šæ•°åŒå®šã™ã‚‹ã“ã¨ãŒã§ãï¼Œã“
ã‚Œã‚‰ã®æ‰‹æ³•ã®æœ‰ç”¨æ€§ã¯ç¤ºã•ã‚Œã¦ã„ã‚‹ 37,38)ã€‚

37) Hultman, J., M. Tamminen, K. PÃ¤rnÃ¤nen, J. Cairns, A.
Karkman, and M. Virta. 2018. Host range of antibiotic resistance genes in wastewater treatment plant influent and effluent. FEMS Microbiol. Ecol. 94: (4). doi: 10.1093/femsec/
fiy038.
38) Stalder, T., M.O. Press, S. Sullivan, I. Liachko, and E.M. Top.
2019. Linking the resistome and plasmidome to the microbiome. ISME J. 13: 2437â€“2446.


https://www.jst.go.jp/report/2017/170925.html
JSTãƒˆãƒ”ãƒƒã‚¯ã‚¹ï¼š2017NGSãƒãƒ³ã‚ºã‚ªãƒ³è¬›ç¿’ä¼š é–‹å‚¬å ±å‘Š

2017NGSãƒãƒ³ã‚ºã‚ªãƒ³è¬›ç¿’ä¼š é–‹å‚¬å ±å‘Š
2017å¹´08æœˆ28æ—¥ï½09æœˆ01æ—¥ã€€æ±äº¬å¤§å­¦

ã€ŒHi-Cè§£æã€ï¼ˆå›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ æ± å…‰ä¸€ç ”ç©¶å“¡ï¼‰

https://biosciencedbc.jp/gadget/human/170901_higashi_170831.pdf
å¹³æˆ29å¹´åº¦
NGSãƒãƒ³ã‚ºã‚ªãƒ³è¬›ç¿’ä¼š
Hi-Cè§£æ
2017å¹´9æœˆ1æ—¥
å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ æ±å…‰ä¸€

https://togotv.dbcls.jp/ajacs2017017.html
NGSè§£æ(åˆã€œä¸­ç´š) Hi-Cè§£æ | TogoTV

https://biosciencedbc.jp/event/ngs/2017.html
NGSãƒãƒ³ã‚ºã‚ªãƒ³è¬›ç¿’ä¼š2017 - NBDC

Hi-Cè§£æ
æ± å…‰ä¸€
Chromosome Conformation Capture å®Ÿé¨“ã§ç”Ÿæˆã•ã‚ŒãŸãƒ‡ãƒ¼ã‚¿ã®æƒ…å ±è§£æã«ã¤ã„ã¦è¬›ç¿’ã‚’è¡Œã†äºˆå®šã§ã™ã€‚ãƒ’ãƒˆã‚²ãƒãƒ ã‚’å¯¾è±¡ã¨ã—ãŸHi-Cãƒ‡ãƒ¼ã‚¿ã‚’ç”¨ã„ã¦ã€ãƒãƒƒãƒ”ãƒ³ã‚°ã€ãƒ•ã‚£ãƒ«ã‚¿ãƒªãƒ³ã‚°ã€ã‚³ãƒ³ã‚¿ã‚¯ãƒˆãƒãƒƒãƒ—ç”Ÿæˆã€æ­£è¦åŒ–å‡¦ç†ã€æŸ“è‰²ä½“ä¸‰æ¬¡å…ƒæ§‹é€ å†æ§‹æˆãªã©ã€å„ç¨®ãƒ„ãƒ¼ãƒ«ã‚„Pythonã‚¹ã‚¯ãƒªãƒ—ãƒˆãªã©ã‚’ç”¨ã„ã¦è¡Œã†äºˆå®šã§ã™ã€‚

è¬›ç¾©å‹•ç”» ã‚¹ãƒ©ã‚¤ãƒ‰(11.6MB)
é–¢é€£æƒ…å ± (2024å¹´11æœˆä»˜è¨˜)

ãƒ‡ãƒ¼ã‚¿è§£æè¬›ç¿’ä¼šï¼šAJACSã€ŒHi-Cè§£æã‚’çŸ¥ã£ã¦ãƒ»å­¦ã‚“ã§ãƒ»ä½¿ã†ã€
æ—¥ã«ã¡ï¼š2025å¹´1æœˆ16æ—¥ 13:30-15:50
è¬›å¸«ï¼šå›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ æ± å…‰ä¸€æ°

https://biosciencedbc.jp/event/ajacs/ajacs2025-01-16-Hi-C.html
AJACSã€ŒHi-Cè§£æã‚’çŸ¥ã£ã¦ãƒ»å­¦ã‚“ã§ãƒ»ä½¿ã†ã€ - NBDC

æŸ“è‰²ä½“ã®ç«‹ä½“æ§‹é€ ã®è§£ææ‰‹æ³•ã§ã‚ã‚‹ Hi-C (High-throughput chromosome conformation capture) è§£æã®åŸç†ã€å®Ÿéš›ã®è§£æã®æµã‚Œã‚’å­¦ã³ã¾ã™ã€‚

13:40ï½14:40

Hi-Cè§£æã‚’çŸ¥ã‚‹

æ± å…‰ä¸€æ° (å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€)

Hi-Cè§£æã®åŸç†ã¨æŸ“è‰²ä½“é«˜æ¬¡æ§‹é€ ã®åŸºç¤ã‚’è§£èª¬ã™ã‚‹ã€‚æŸ“è‰²ä½“ç›¸äº’ä½œç”¨ã®ç ”ç©¶ä¾‹ã‚„å®Ÿé¨“æ‰‹æ³•ã®æ¦‚è¦ã‚’ç¤ºã—ã€Hi-Cè§£æã®å¯èƒ½æ€§ã¨èª²é¡Œã‚’ç†è§£ã™ã‚‹ãŸã‚ã®åŸºç¤çŸ¥è­˜ã‚’æä¾›ã™ã‚‹ã€‚

14:50ï½15:50

Hi-Cãƒ‡ãƒ¼ã‚¿ã‚’ä½¿ã†

æ± å…‰ä¸€æ° (å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€)

Hi-Cãƒ‡ãƒ¼ã‚¿è§£æã®æµã‚Œã‚’ç´¹ä»‹ã—ã€å¯è¦–åŒ–æ‰‹æ³•ã‚„ä»–ã®ã‚¨ãƒ”ã‚¸ã‚§ãƒãƒ†ã‚£ã‚¯ã‚¹ãƒ‡ãƒ¼ã‚¿ã¨ã®æ¯”è¼ƒä¾‹ã‚’ç¤ºã™ã€‚å…¬é–‹ãƒ‡ãƒ¼ã‚¿ã‚’ç”¨ã„ãŸãƒ‡ãƒ¢ã‚’é€šã˜ã¦ã€å®Ÿè·µçš„ãªè§£ææ–¹æ³•ã‚’å­¦ã¶æ©Ÿä¼šã‚’æä¾›ã™ã‚‹ã€‚

æ—¥æ™‚	2025å¹´1æœˆ16æ—¥ (æœ¨) 13:30-15:50

----------
## diversity index
å¤šæ§˜æ€§æŒ‡æ•°

https://www.nies.go.jp/kanko/news/20/20-1/20-1-06.html
ã€Œç”Ÿç‰©å¤šæ§˜æ€§ã®æ¸›å°‘æ©Ÿæ§‹ã¨ä¿å…¨ãƒ—ãƒ­ã‚¸ã‚§ã‚¯ãƒˆã€ãŒç›®æŒ‡ã™ã‚‚ã®ï¼ˆ2001å¹´åº¦ 20å·»1å·ï¼‰ï½œ
å¤šæ§˜æ€§ã‚’ï¼Œãã‚Œãã‚Œç¨®é¡æ•°ï¼Œå‡ç­‰æ€§ï¼Œç³»çµ±çš„è·é›¢ã§è¨ˆã£ã¦ã„ã‚‹ã“ã¨ã«ãªã‚‹ã€‚

![https://www.nies.go.jp/kanko/news/20/20-1/20-1-06.html](https://www.nies.go.jp/kanko/news/20/20-1/gh5r5k000001g6rd-img/20-1-06-f1.jpg)

https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/188bto4/alma9926310659504034
ç”Ÿç‰©å¤šæ§˜æ€§ã®å¤šæ§˜æ€§ / æ£®ç« è‘— ; ç”²å±±éš†å¸ã‚³ãƒ¼ãƒ‡ã‚£ãƒãƒ¼ã‚¿ãƒ¼.
æ±äº¬: å…±ç«‹å‡ºç‰ˆ, 2018.1.
2018.1

Maruzen eBook Library Purchased Titles
è³¼å…¥æ¸ˆ/Purchased

https://www.kyoritsu-pub.co.jp/bookdetail/9784320009226
ç”Ÿç‰©å¤šæ§˜æ€§ã®å¤šæ§˜æ€§ / æ£® ç« ã€€è‘— ç”²å±± éš†å¸ã€€ã‚³ãƒ¼ãƒ‡ã‚£ãƒãƒ¼ã‚¿ãƒ¼ | å…±ç«‹å‡ºç‰ˆ
2.5 ç”Ÿç‰©å¤šæ§˜æ€§ã®å®šé‡åŒ–
2.5.1 ç¨®ã¨ã„ã†å°ºåº¦ã«åŸºã¥ã 49
2.5.2 ç”Ÿãç‰©ã®ç‰¹æ€§ã«åŸºã¥ã 62

ç³»çµ±çš„å¤šæ§˜æ€§

Box 2.6
ç³»çµ±æ¨¹ã«åŸºã¥ãå¤šæ§˜æ€§ã®è©•ä¾¡

73
 ç•°ãªã‚‹æŒ‡æ¨™ã‹ã‚‰è¦‹ãˆã¦ãã‚‹ã“ã¨
æ–¹æ³•è«–ã«é‡ãã‚’ç½®ã„ã¦ã—ã¾ã£ãŸã‹ã‚™ï¼Œã“ã“ã¾ã¦ã‚™ã¦ã‚™ä¼ãˆãŸã„ã“ã¨ã¯ï¼Œ ã€Œç•°ãªã‚‹æŒ‡æ¨™ã‚’ç”¨ã„ã‚‹ã¨ï¼Œç”Ÿç‰©å¤šæ§˜æ€§ã®è©•ä¾¡ã®çµæœã‹ã‚™å¤‰ã‚ã‚‹ã€ã¨ã„
ã†ã“ã¨ã¦ã‚™ã‚ã‚‹.ãƒ˜ã‚™ãƒ¼ã‚¿å¤šæ§˜æ€§ã‹ã‚™ä¸Šã‹ã‚™ã‚‹ä¸€æ–¹ã¦ã‚™ã‚¢ãƒ«ãƒ•ã‚¡å¤šæ§˜æ€§ã‹ã‚™ä¸‹ã‹ã‚™
ã‚‹ã“ã¨ã‹ã‚™ã‚ã‚‹ã“ã¨ï¼Œç¨®æ•°ã‹ã‚™åŒã—ã‚™ã¦ã‚™ã‚‚ç³»çµ±çš„å¤šæ§˜æ€§ã‹ã‚™å¤§ããç•°ãªã‚Šå¾—
ã‚‹ã“ã¨ï¼Œè©•ä¾¡å¯¾è±¡ã®æ©Ÿèƒ½å½¢è³ªã‚’å¤‰ãˆã‚‹ã¨æ©Ÿèƒ½çš„å¤šæ§˜æ€§ã®è¨ˆç®—çµæœã‹ã‚™
é€†è»¢ã—å¾—ã‚‹ã“ã¨ãªã¨ã‚™ã‚’è§£èª¬ã—ãŸ.


https://github.com/haruosuz/evolve/blob/master/README.md#fifthdimension
https://www.fifthdimension.jp/wiki.cgi?page=FrontPage&file=20100522BiometricsJapanPreprint%2Epdf&action=ATTACH
ç³»çµ±æ¨¹ã®ç¬¬ 3 ã®ç”¨é€”ã¯ã€å¤šæ§˜æ€§ã®å®šé‡åŒ–ã¦ã‚™ã‚ã‚‹ã€‚ç”Ÿç‰©å¤šæ§˜æ€§ã®é‡è¦æ€§ã‹ã‚™å«ã¯ã‚™ã‚Œã¦ä¹…ã—ã„ ã‹ã‚™ã€ã„ã¾ãŸã‚™ã«å¤šæ§˜æ€§ã¯ç¨®ãƒ»å±ãƒ»ç§‘ãªã¨ã‚™ã®åˆ†é¡ç¾¤ã®æ•°ã¦ã‚™å®šé‡åŒ–ã•ã‚Œã¦ã„ã‚‹ã€‚ã—ã‹ã—ã“ã‚Œã‚‰ã® åˆ†é¡ç¾¤ã¯äººé–“ã‹ã‚™ä¾¿å®œçš„ã«è¨­å®šã—ãŸã‚‚ã®ã¦ã‚™ã‚ã‚Šã€ãã®æ•°ã¦ã‚™ã¯ååˆ†ã«è‡ªç„¶ç•Œã®å¤šæ§˜æ€§ã‚’èªè­˜ ã¦ã‚™ãã¦ã„ã‚‹ã¨ã¯è¨€ã„é›£ã„ã¨è€ƒãˆã‚‰ã‚Œã‚‹ã€‚ã“ã‚Œã‚’ã€ç³»çµ±æ¨¹ä¸Šã¦ã‚™ã®ç”Ÿç‰©é–“ã®ãƒã‚šã‚¹ã®é•·ã•ã®å’Œ= ç³»çµ±çš„å¤šæ§˜æ€§ã¦ã‚™ç½®ãæ›ãˆã‚‹ã“ã¨ã¦ã‚™è§£æ±ºã—ã‚ˆã†ã¨ã„ã†ç ”ç©¶ã‹ã‚™å¾ã€…ã«å¢—ãˆã¦ãã¦ã„ã‚‹ (Faith, 1992; Forest et al., 2007)ã€‚

https://github.com/haruosuz/evolve/blob/master/README.md#leeswijzer
2007.3
http://www.naro.affrc.go.jp/archive/niaes/magazine/083/mgzn08304.html
è«–æ–‡ã®ç´¹ä»‹ï¼š ç”Ÿç‰©å¤šæ§˜æ€§ã‚’é€²åŒ–ç³»çµ±å­¦çš„ãªå°ºåº¦ã§æ¸¬ã‚‹ (æƒ…å ±ï¼šè¾²æ¥­ã¨ç’°å¢ƒ No.83 2007.3)
ã€Œç³»çµ±å­¦çš„å¤šæ§˜æ€§ï¼ˆPD: phylogenetic diversityï¼‰ã€
æ—§æ¥ã®ç¨®æ•°ãƒ™ãƒ¼ã‚¹ã®ç”Ÿç‰©å¤šæ§˜åº¦ã«ä»£ã‚ã‚‹å°ºåº¦ã¨ã—ã¦æå”±ã•ã‚ŒãŸã®ãŒç³»çµ±å­¦çš„å¤šæ§˜åº¦ã§ã‚ã‚‹ã€‚ã—ã‹ã—ã€ç³»çµ±å­¦çš„å¤šæ§˜åº¦ãã‚Œè‡ªä½“ã«é–¢ã—ã¦ã€ã„ã¾ãªãŠç†è«–çš„ãƒ»æ¦‚å¿µçš„ãªè«–äº‰ãŒç¶šã„ã¦ã„ã‚‹ ï¼ˆ3ï¼‰ã€‚ç³»çµ±å­¦çš„å¤šæ§˜æ€§ã®å°ºåº¦ã¯ã€ç³»çµ±æ¨¹ã®æ¨¹å½¢ã‚’è€ƒæ…®ã—ã¤ã¤ç”Ÿç‰©å¤šæ§˜æ€§ã®é‡ã¿ã¥ã‘ã‚’ã—ã¦ã„ã‚‹ã€‚æå”±è€…ã§ã‚ã‚‹ Daniel P. Faith ã®å®šç¾©ã§ã¯ ï¼ˆ1ï¼‰ã€ã‚ã‚‹ç”Ÿç‰©ç¾¤ã®ç³»çµ±å­¦çš„å¤šæ§˜åº¦ã¨ã¯ã€ãã®ç”Ÿç‰©ç¾¤ã‚’ç³»çµ±æ¨¹ä¸Šã§çµã¶æã®é•·ã•ã®ç·å’Œã¨ã—ã¦æ±‚ã‚ã‚‰ã‚Œã‚‹ã€‚ã“ã®ã¨ãã€æã®é•·ã•ã‚’ã©ã®ã‚ˆã†ã«ã—ã¦æ¨å®šã™ã‚‹ã®ã‹ã¨ã„ã†ç¾å®Ÿçš„å•é¡ŒãŒã‚ã‚‹ï¼ˆåˆ†å­ç³»çµ±æ¨¹ã¨å½¢æ…‹ç³»çµ±æ¨¹ã§ã¯ã€Œæã€ã®æ„å‘³ã¯ç•°ãªã£ã¦ã„ã‚‹ã ã‚ã†ï¼‰ã€‚

2015
https://www.jstage.jst.go.jp/article/taxa/38/0/38_KJ00009847305/_article/-char/ja/
ã‚¿ã‚¯ã‚µï¼šæ—¥æœ¬å‹•ç‰©åˆ†é¡å­¦ä¼šèªŒ/38 å·» (2015)/æ›¸èªŒ
ç”Ÿç‰©å¤šæ§˜æ€§ã¨å›ºæœ‰ç¨®ã®é–¢ä¿‚ã‚’ã‚ãã‚‹è‹¥å¹²ã®è€ƒå¯Ÿ
é–¢å£ ç§€å¤«
https://www.jstage.jst.go.jp/article/taxa/38/0/38_KJ00009847305/_pdf
ç”Ÿç‰©ç›¸ã®ã‚‚ã¤å¤šæ§˜æ€§ã¯ï¼Œåœ°åŸŸç’°å¢ƒã®ã‚‚ã¨ã§é€²åŒ–çš„
ãªã‚‰ã³ã«ç”Ÿç‰©åœ°ç†çš„ãªæ­´å²ã‚’èƒŒè² ã£ã¦æˆç«‹ã—ã¦ãŠ
ã‚Šï¼Œå„ç”Ÿç‰©ã¯é€²åŒ–çš„ã«çµã³ã¤ã„ã¦ã„ã‚‹ï¼ˆFaith, 1992;
Mooers, 2007ï¼‰ï¼ã“ã®åŸºæœ¬èªè­˜ã®ã‚‚ã¨ã§ã¯ï¼Œåœ°åŸŸã®
ç”Ÿç‰©å¤šæ§˜æ€§ã®è¡¨ç¾ã¯å˜ã«ã€Œç¨®æ•°ã€ã‚’ã‚«ã‚¦ãƒ³ãƒˆã™ã‚‹ã 
ã‘ã§ã¯ä¸ååˆ†ã§ã‚ã‚Šï¼Œç”Ÿç‰©å¤šæ§˜æ€§ã¨ã—ã¦ä¿å…¨ã™ã¹ã
ã‚‚ã®ã¯ï¼Œã‚€ã—ã‚é€²åŒ–çš„ãªé–¢ä¿‚ã‚’ç¤ºã™åˆ†å²ç³»çµ±æ¨¹ã«åŸº
ã¥ãç³»çµ±å­¦çš„å¤šæ§˜æ€§phylogenetic diversity (PD)ã§ã‚
ã‚ã†ï¼ç³»çµ±å­¦çš„å¤šæ§˜åº¦ã¯å¾“æ¥ã®ç¨®æ•°ã‚’ã‚‚ã¨ã«ã—ãŸç”Ÿ
ç‰©å¤šæ§˜åº¦ã«ä»£ã‚ã‚‹å°ºåº¦ã¨ã—ã¦æå”±ã•ã‚ŒãŸï¼

### Koyano_2012

2012
https://www.ism.ac.jp/editsec/toukei/pdf/60-2-263.pdf
çµ±è¨ˆæ•°ç† 60(2), 263-278, 2012-12
ç‰¹é›†ã€Œå¤šæ§˜æ€§ã¨é€²åŒ–ã®çµ±è¨ˆè§£æã€Î± å¤šæ§˜æ€§ã®æ¸¬å®šã¨ç¢ºç‡æ–‡å­—åˆ—ã®ç†è«–
å°è°·é‡ ä»â€ 
ï¼ˆå—ä»˜ 2012 å¹´ 1 æœˆ 4 æ—¥ï¼›æ”¹è¨‚ 5 æœˆ 29 æ—¥ï¼›æ¡æŠ 5 æœˆ 31 æ—¥ï¼‰
ä»£æ›¿çš„ãªæ–¹æ³•ã®ç ”ç©¶ã¨ã—ã¦ï¼ŒWatve and Gangalï¼ˆ1996ï¼‰, Hughes et al.ï¼ˆ2001ï¼‰, Hong et al.ï¼ˆ2006ï¼‰ãªã©ãŒã‚ã‚‹ãŒï¼Œç†è«–çš„ãªåŸºç¤ã‚’æŒã¤ä½“ç³»çš„ãªæ–¹æ³•ã¨ã¯è¨€ãˆãªã‹ã£ãŸï¼
https://pubmed.ncbi.nlm.nih.gov/20866445/
Phys Rev E Stat Nonlin Soft Matter Phys
. 2010 Jun;81(6 Pt 1):061912. doi: 10.1103/PhysRevE.81.061912. Epub 2010 Jun 7.
Quantifying biodiversity and asymptotics for a sequence of random strings
Hitoshi Koyano 1, Hirohisa Kishino
https://journals.aps.org/pre/abstract/10.1103/PhysRevE.81.061912

### McCoy_2013

https://pubmed.ncbi.nlm.nih.gov/24058885/
PeerJ
. 2013 Sep 12;1:e157. doi: 10.7717/peerj.157. eCollection 2013.
Abundance-weighted phylogenetic diversity measures distinguish microbial community states and are robust to sampling depth
Connor O McCoy 1, Frederick A Matsen 4th
https://peerj.com/articles/157/
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3775626/
Based on these results we encourage the use of abundance-weighted phylogenetic diversity measures, especially for cases such as microbial ecology where species delimitation is difficult.

### Kembel_2012

https://pubmed.ncbi.nlm.nih.gov/22278670/
ISME J
. 2012 Aug;6(8):1469-79. doi: 10.1038/ismej.2011.211. Epub 2012 Jan 26.
Architectural design influences the diversity and structure of the built environment microbiome
Steven W Kembel 1, Evan Jones, Jeff Kline, Dale Northcutt, Jason Stenson, Ann M Womack, Brendan Jm Bohannan, G Z Brown, Jessica L Green
https://www.nature.com/articles/ismej2011211
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3400407/
The phylogenetic diversity of airborne bacterial communities was lower indoors than outdoors, and mechanically ventilated rooms contained less diverse microbial communities than did window-ventilated rooms.

Phylogenetic diversity (PD) was calculated as Faith's PD (Faith, 1992), the total phylogenetic branch length separating OTUs in each rarefied sample.

The PD of airborne bacterial communities differed significantly among environments (ANOVA; F2,6=15.5, P=0.005), with highest diversity in outdoor air and lowest in indoor air from rooms that were mechanically ventilated (Figure 2). 

Figure 2
PD (total phylogenetic branch length; Faith's PD per 700 sequences) in different environments at a health-care facility: outdoors and indoors in patient rooms exposed to different ventilation sources (mechanical or window ventilation). 

### Kembel_2011

https://pubmed.ncbi.nlm.nih.gov/21912589/
PLoS One
. 2011;6(8):e23214. doi: 10.1371/journal.pone.0023214. Epub 2011 Aug 31.
The phylogenetic diversity of metagenomes
Steven W Kembel 1, Jonathan A Eisen, Katherine S Pollard, Jessica L Green
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0023214
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3166145/
Bacterial phylogenetic diversity was highest at intermediate depths beneath the ocean surface, whereas taxonomic diversity (diversity measured by binning sequences into taxonomically similar groups) showed no relationship with depth.

Figure 3
Taxonomic diversity and standardized phylogenetic diversity versus depth in environmental samples along an oceanic depth gradient at the HOT ALOHA site.

Jul 2011
http://www.ted.com/talks/jessica_green_are_we_filtering_the_wrong_microbes?language=ja
ã‚¸ã‚§ã‚·ã‚«ãƒ»ã‚°ãƒªãƒ¼ãƒ³ã€Œå¾®ç”Ÿç‰©ã‚’æ­£ã—ãå–ã‚Šé™¤ããŸã‚ã«ã€

### Dmean

2019
https://github.com/haruosuz/microbe/blob/master/references/README.MoBE.md
16Sãƒªãƒœã‚½ãƒ¼ãƒ RNAéºä¼å­é…åˆ—ã«åŸºã¥ãç´°èŒã®å¤šæ§˜æ€§ãƒ¬ãƒ™ãƒ«ã¯å±é–“ã§å¤§ããç•°ãªã£ãŸã€‚
https://www.ncbi.nlm.nih.gov/pubmed/30691394
BMC Genomics. 2019 Jan 28;20(1):92. doi: 10.1186/s12864-018-5389-z.
Comparative genomics of Bacteria commonly identified in the built environment.
Merino N, Zhang S, Tomita M, Suzuki H.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6350394/
The Dmean was also selected over the phylogenetic diversity index (PD) [64, 65] because it is less affected by the number of taxa (N) available in the LTP database, 

2018å¹´9æœˆ8æ—¥
https://sites.google.com/site/colloqfoundbio/conference/conference-12
ç¬¬ï¼‘ï¼’å›ç”Ÿç‰©å­¦åŸºç¤è«–ç ”ç©¶ä¼š - ç”Ÿç‰©å­¦åŸºç¤è«–ç ”ç©¶ä¼š
éˆ´æœ¨æ²»å¤«ï¼ˆæ…¶æ‡‰ç¾©å¡¾å¤§å­¦ç’°å¢ƒæƒ…å ±å­¦éƒ¨ï¼‰ã€Œãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»ç´°èŒã®éå»äºˆæ¸¬ã¨å¤šæ§˜æ€§æ¸¬å®šã€
ã€€ç´°èŒé–“ã‚’ç§»å‹•ã™ã‚‹æŸ“è‰²ä½“å¤–DNAã€ã™ãªã‚ã¡ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¯ã€ç´°èŒã®æ©Ÿèƒ½ï¼ˆè–¬å‰¤è€æ€§ã€ç—…åŸæ€§ã€ç‰©è³ªä»£è¬èƒ½ãªã©ï¼‰ã®ç²å¾—ã«é–¢ä¸ã™ã‚‹ã“ã¨ãŒçŸ¥ã‚‰ã‚Œã¦ã„ã‚‹ã€‚éºä¼å­æ°´å¹³ä¼æ’­ã®ç†è§£ã«ã¯ã€ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»åŸŸã®äºˆæ¸¬ãŒä¸å¯æ¬ ã§ã‚ã‚‹ã€‚æ§˜ã€…ãªç´°èŒã§æŸ“è‰²ä½“ã¨ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®é€£ç¶šå¡©åŸºçµ„æˆï¼ˆã‚ªãƒªã‚´ãƒŒã‚¯ãƒ¬ã‚ªãƒãƒ‰å‡ºç¾é »åº¦ï¼‰ãŒé¡ä¼¼ã—ã¦ã„ã‚‹ã“ã¨ã¯ã€ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ãŒå®¿ä¸»ã®å¡©åŸºçµ„æˆã‚’ç²å¾—ã—ãŸã“ã¨ã‚’ç¤ºå”†ã™ã‚‹ã€‚ãã“ã§ã€ç´°èŒæŸ“è‰²ä½“ã¨ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®é€£ç¶šå¡©åŸºçµ„æˆã«é–¢ã™ã‚‹é¡ä¼¼åº¦ã«åŸºã¥ã„ã¦ã€å„ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»åŸŸã‚’éå»äºˆæ¸¬ã—ãŸã€‚ã™ãªã‚ã¡ã€é€£ç¶šå¡©åŸºçµ„æˆã«é–¢ã™ã‚‹éé¡ä¼¼åº¦ï¼ˆãƒãƒãƒ©ãƒãƒ“ã‚¹è·é›¢ï¼‰ã®å°ã•ã„ç´°èŒã‚’ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»å€™è£œã¨ã—ã€ç‹­å®¿ä¸»åŸŸãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¨åºƒå®¿ä¸»åŸŸãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¨ã®é–“ã§éå»ã«æ»åœ¨ã—ãŸå®¿ä¸»ç´°èŒã®å¤šæ§˜æ€§ã«é•ã„ãŒèªã‚ã‚‰ã‚Œã‚‹ã‹ã©ã†ã‹ã‚’æ¤œè¨¼ã—ãŸã€‚
ã€€ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰é–“ã®å®¿ä¸»åŸŸã‚’æ¯”è¼ƒã™ã‚‹ãŸã‚ã«ã€ã€Œç´°èŒã®ç¨®é¡ã€ã¨ã€Œç´°èŒé–“ã®è·é›¢ã€ã«åŸºã¥ã„ã¦å®¿ä¸»å€™è£œã®å¤šæ§˜æ€§ã‚’æ¸¬å®šã—ãŸã€‚å…ˆãšã€ã€Œç´°èŒã®ç¨®é¡ã€ï¼ˆé–€ãƒ»ç¶±ãƒ»ç›®ãƒ»ç§‘ãƒ»å±ãƒ»ç¨®ãƒ»æ ªãªã©ã®åˆ†é¡ç¾¤ã®æ•°ï¼‰ã«åŸºã¥ã„ã¦å„ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»å¤šæ§˜æ€§ã‚’æ¸¬å®šã—ãŸã€‚åˆ†é¡ç¾¤ã®æ•°ã¯ã€ç´°èŒé–“ã®è·é›¢ã‚’è€ƒæ…®ã—ã¦ãŠã‚‰ãšã€å®¿ä¸»å€™è£œã¨ã—ã¦æ¤œå‡ºã•ã‚ŒãŸèŒæ ªã®æ•°ï¼ˆåˆ©ç”¨å¯èƒ½ãªã‚²ãƒãƒ é…åˆ—ã®åã‚Šï¼‰ã«ã‚ˆã£ã¦ã‚‚å½±éŸ¿ã•ã‚Œå¾—ã‚‹ã¨ã„ã†æ¬ ç‚¹ã‚’æœ‰ã™ã‚‹ã€‚ä¾‹ãˆã°ã€ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰é–“ã®å®¿ä¸»å¤šæ§˜æ€§ã®é«˜ä½ãŒåˆ†é¡éšç´šã«ã‚ˆã‚Šé€†è»¢ã™ã‚‹ã“ã¨ãŒã‚ã‚‹ã€‚ã•ã‚‰ã«æ–°ç¨®ãƒ»æ–°å±ã«å†åˆ†é¡ã•ã‚Œã¦ã„ã‚‹èŒæ ªã‚‚ã‚ã‚‹ã€‚æ¬¡ã«ã€ç¨®ãªã©ã®åˆ†é¡å­¦çš„æƒ…å ±ã‚’ç”¨ã„ã‚‹ã“ã¨ãªãã€16S rRNAéºä¼å­é…åˆ—ã«åŸºã¥ãã€Œç´°èŒé–“ã®è·é›¢ã€ã«åŸºã¥ã„ã¦ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»å¤šæ§˜æ€§ã‚’æ¸¬å®šã—ãŸã€‚ç‹­å®¿ä¸»åŸŸãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¯ã€åºƒå®¿ä¸»åŸŸãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¨æ¯”è¼ƒã—ã¦ã€å®¿ä¸»å€™è£œã®å¤šæ§˜æ€§ãŒä½ã‹ã£ãŸã€‚ã“ã®çµæœã¯ã€DNAå¡©åŸºçµ„æˆã®è§£æãŒãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»åŸŸã®äºˆæ¸¬ã«æœ‰åŠ¹ã§ã‚ã‚‹ã“ã¨ã‚’ç¤ºå”†ã™ã‚‹ã€‚

2010
https://www.ncbi.nlm.nih.gov/pubmed/20851899
J Bacteriol. 2010 Nov;192(22):6045-55. doi: 10.1128/JB.00277-10. Epub 2010 Sep 17.
Predicting plasmid promiscuity based on genomic signature.
Suzuki H1, Yano H, Brown CJ, Top EM.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2976448/
Second, we inferred plasmid host range based on genetic distance between putative evolutionary hosts without using taxonomic information. â€œTaxonomic richnessâ€ has drawbacks, because it does not take into account distances between bacterial strains and also can be influenced by the number of strains detected as candidate evolutionary hosts, which in turn is biased by the genome sequences available. For example, as shown in Table â€‹Table1,1, the numbers of classes, orders, families, genera, species, and strains for IncP plasmid pQKH54 were 2, 3, 3, 3, 6, and 11, respectively, while those for IncN plasmid R46 were 1, 1, 1, 13, 17, and 55. Thus, while pQKH54 showed higher taxonomic richness values at the class, order, and family levels, R46 showed higher taxonomic richness values at the genus, species, and strain levels. Moreover, several bacterial strains are still being reclassified into new species and even new genera. Therefore, a method for inferring evolutionary host diversity that does not require a species or any other taxonomic information was conducted.

1996
https://www.ncbi.nlm.nih.gov/pubmed/16535456
Appl Environ Microbiol. 1996 Nov;62(11):4299-301.
Problems in measuring bacterial diversity and a possible solution.
Watve MG, Gangal RM.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1388994/pdf/hw4299.pdf
Simpsonâ€™s
index, the Shannon index, and evenness indices give enough
weighting to the relative abundances along with species richness but fail to take into account the taxonomic distances
between species, individuals, or any other appropriate unit. An
information-based index would treat a community of four different biotypes of coliforms identical to another community
consisting of one species of coliforms, one of actinomycetes,
one of myxobacteria, and one of archaebacteria, whereas we
feel that the latter should be treated as more diverse. 

A simple alternative is to use the mean taxonomic distance (Dmean) between all pairs of isolates as a diversity index (14): 

----------

### NGS_DAT
https://twitter.com/hashtag/NGS_DAT

2019å¹´12æœˆ12æ—¥
https://gakken-mesh.jp/book/detail/9784780909838.html
æ¬¡ä¸–ä»£ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚µãƒ¼DRYè§£ææ•™æœ¬ã€€æ”¹è¨‚ç¬¬2ç‰ˆ | å­¦ç ”ãƒ¡ãƒ‡ã‚£ã‚«ãƒ«ç§€æ½¤ç¤¾
0 ã‹ã‚‰å§‹ã‚ã‚‹ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ å¿—æ³¢ å„ª https://github.com/youyuh48/NGSDRY2/
p.216
faith-pd-group-significance.qzv | ç³»çµ±å­¦çš„å¤šæ§˜æ€§ã¨ãã®çµ±è¨ˆè§£æ

https://docs.qiime2.org/2019.10/tutorials/moving-pictures/
Generate a tree for phylogenetic diversity analyses
QIIME supports several phylogenetic diversity metrics, including Faithâ€™s Phylogenetic Diversity and weighted and unweighted UniFrac. 

https://github.com/youyuh48/NGSDRY2/
æ¬¡ä¸–ä»£ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚µãƒ¼DRYè§£ææ•™æœ¬ æ”¹è¨‚ç¬¬2ç‰ˆ ã‚µãƒãƒ¼ãƒˆã‚µã‚¤ãƒˆ

QIIME2-01
Level2ï¼šå®Ÿè·µç·¨5ã€Œï¼ã‹ã‚‰å§‹ã‚ã‚‹ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã€æœ¬æ–‡éƒ¨åˆ†ã®ã‚³ãƒãƒ³ãƒ‰ã¨ãƒ‡ãƒ¼ã‚¿
QIIME2-02
Level2ï¼šå®Ÿè·µç·¨5ã€Œï¼ã‹ã‚‰å§‹ã‚ã‚‹ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã€è§£ææ‰‹é †æ›¸ã®ã‚³ãƒãƒ³ãƒ‰ã¨ãƒ‡ãƒ¼ã‚¿
Yamamoto-etal-2018-FrontMicrobiol
Level3ï¼šå¿œç”¨ç·¨14ã€Œãƒ¡ã‚¿16Sã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚¹ã®å„ã‚µãƒ³ãƒ—ãƒ«ã‹ã‚‰å¾—ã‚‰ã‚ŒãŸç´°èŒå¢çµ„æˆã®å·®ã‚’ä¸»åº§æ¨™åˆ†æãƒ»ã‚¯ãƒ©ã‚¹ã‚¿ãƒ¼åˆ†æã«ã‚ˆã‚Šå¯è¦–åŒ–ã™ã‚‹

----------

https://kazumaxneo.hatenablog.com/entry/2020/04/28/073000
DEICODE - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

Î²å¤šæ§˜æ€§ã¨ã¯ã€ç”Ÿç‰©ç¾¤é›†é–“ã®åˆ†é¡å­¦çš„ã¾ãŸã¯ç³»çµ±çš„æ§‹æˆã®é•ã„ã‚’è¡¨ã™ç”Ÿæ…‹å­¦çš„ãªæ¦‚å¿µã§ã‚ã‚‹ã€‚
UniFrac (ref.8)ã§ç¤ºã•ã‚ŒãŸã‚ˆã†ã«ã€ç³»çµ±æƒ…å ±ã‚’å–ã‚Šå…¥ã‚Œã‚‹ã“ã¨ã§ã€å¾®ç”Ÿç‰©ã®Î²å¤šæ§˜æ€§ã®æ¨å®šã¯å¤§å¹…ã«æ”¹å–„ã•ã‚ŒãŸã€‚ã—ã‹ã—ã€ presence/absence methodã§ã¯ã€ã—ã°ã—ã°ã‚³ãƒŸãƒ¥ãƒ‹ãƒ†ã‚£é–“ã®å·®ãŒé¡•è‘—ã«ç¾ã‚Œã€ãã‚Œã¯è±Šå¯Œã•ã«åŸºã¥ã„ãŸæ–¹æ³•ã§ã¯ä¸æ˜ç­ã§ã‚ã‚‹ã€‚ã“ã‚Œã¯é€†èª¬çš„ã«è¦‹ãˆã‚‹ã‹ã‚‚ã—ã‚Œãªã„ãŒã€è±Šå¯Œã•ã«åŸºã¥ã„ãŸæ‰‹æ³•ã¯ã‚³ãƒŸãƒ¥ãƒ‹ãƒ†ã‚£ã«é–¢ã™ã‚‹ã‚ˆã‚Šå¤šãã®æƒ…å ±ã‚’çµ±åˆã—ã¦ã„ã‚‹ã‹ã‚‰ã§ã‚ã‚‹ã€‚ã—ã‹ã—ã€é‡è¦ãªæ‹…ã„æ‰‹ãŒè±Šå¯Œãªç¨®ã§ã¯ãªãå¸Œå°‘ãªç¨®ã§ã‚ã£ãŸã‚Šã€è±Šå¯Œãªç¨®ãŒå¤§ããªãƒ©ãƒ³ãƒ€ãƒ å¤‰å‹•ã‚’ç¤ºã™å ´åˆã«ã¯ã€è±Šå¯Œã•ã®æƒ…å ±ã¯ç³»çµ±çš„ãªæŒ‡æ¨™ã‚’ç”¨ã„ã¦ã‚‚çµæœã‚’æ˜ç¢ºã«ã™ã‚‹ã©ã“ã‚ã‹ä¸æ˜ç­ã«ã—ã¦ã—ã¾ã†å¯èƒ½æ€§ãŒã‚ã‚‹(ref.9)ã€‚

----------

https://github.com/haruosuz/statistics/blob/master/README.md#distance

## UniFrac
UniFracã¯ã€è·é›¢è¨ˆé‡ distance metric ã®è¦ä»¶ã‚’æº€ãŸã™ã€‚

https://en.wikipedia.org/wiki/UniFrac
This definition satisfies the requirements of a distance metric, being non-negative, zero only when entities are identical, transitive, and conformant to the triangle inequality.

https://twitter.com/search?q=UniFrac%20lang%3Aja&src=typed_query&f=top
UniFrac lang:ja

### 2023-06-30
https://pubmed.ncbi.nlm.nih.gov/37387190/
Bioinformatics
. 2023 Jun 30;39(39 Suppl 1):i57-i65. doi: 10.1093/bioinformatics/btad238.
Finding phylogeny-aware and biologically meaningful averages of metagenomic samples: L2UniFrac
Wei Wei 1, Andrew Millward 2, David Koslicki 1 2 3
https://academic.oup.com/bioinformatics/article/39/Supplement_1/i57/7210517?login=false
https://pmc.ncbi.nlm.nih.gov/articles/PMC10311324/

2023/01/24
5:25
https://www.youtube.com/watch?v=93vXTm2AJ88
Applying UniFrac to Whole Genome Shotgun Data - Wei Wei - MICROBIOME - Poster - ISMB 2022

2022/05/10
https://www.youtube.com/watch?v=EXNOgmUyPfY&t=420s
Using the mantel test to compare ecological matrices using the vegan R package (CC211)
Riffomonas Project
7:00 Importing Unweighted and Weighted Unifrac distances

2021/04/28
17:25 / 36:00
https://www.youtube.com/watch?v=7H_LhmGafOc
Beta diversity metrics
QIIME 2
The slides in this video can be found here: https://bit.ly/3yTQ8xq

2016/01/30
https://www.youtube.com/watch?v=M8ylvsS0MHg
Microbiome Discovery 9: UniFrac

### 2022

1:52 AM Â· Feb 7, 2022
https://twitter.com/nishimoto_gut/status/1490367765445681153
ã«ã—ã‚‚ã¨@è…¸å†…ç´°èŒ on X: "# å¾®ç”Ÿç‰©å¢ã®è·é›¢æŒ‡æ¨™ã«é–¢ã™ã‚‹æ¯”è¼ƒ Bray-Curtis like ãªé‡ã¿ä»˜ã‘ UniFracãŒã„ã„ã‚“ã˜ã‚ƒãªã„ï¼Ÿã¨ã„ã†è«–æ–‡ã£ã½ã„ã€‚åˆ†é›¢èƒ½ãŒé«˜ã„ãï¼ã¨ä¸»å¼µã—ã¦ã„ã‚‹ã‚ˆã†ã ãŒã€æ™®æ®µè§£æã§ä½¿ã£ã¦ã„ã‚‹ç«‹å ´ã¨ã—ã¦ã¯ã€ç‰¹å¾´ã‚’èª¬æ˜ã—ã‚„ã™ã„æ‰‹æ³•ã®æ–¹ãŒã„ã„ã§ã™ã­ã€‚https://t.co/kEqDGRe6j9" / X

2022-02-05
https://pubmed.ncbi.nlm.nih.gov/35120564/
Microbiome
. 2022 Feb 5;10(1):25. doi: 10.1186/s40168-021-01199-3.
Performance determinants of unsupervised clustering methods for microbiome data
Yushu Shi 1, Liangliang Zhang 2, Christine B Peterson # 3, Kim-Anh Do # 3, Robert R Jenq # 4 5
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01199-3
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8817542/

2020-06-22
http://www.kaimeisha.com/index.php?å…±ç”Ÿå¾®ç”Ÿç‰©ã‹ã‚‰ã¿ãŸæ–°ã—ã„é€²åŒ–å­¦
https://www.hanmoto.com/bd/isbn/9784875253501
å…±ç”Ÿå¾®ç”Ÿç‰©ã‹ã‚‰ã¿ãŸæ–°ã—ã„é€²åŒ–å­¦
é•·è°·å·æ”¿ç¾(è‘—)
p.57: UniFrac è·é›¢

Dec 2019
https://forum.qiime2.org/t/does-it-make-sense-to-use-weighted-unifrac-distances-for-16s-data/12768
Does it make sense to use weighted unifrac distances for 16s data? - General Discussion - QIIME 2 Forum

https://www.ncbi.nlm.nih.gov/pubmed/30377368
Nat Methods. 2018 Nov;15(11):847-848. doi: 10.1038/s41592-018-0187-8.
Striped UniFrac: enabling microbiome analysis at unprecedented scale.
McDonald D1, VÃ¡zquez-Baeza Y1, Koslicki D2, McClelland J2, Reeve N1,3, Xu Z1, Gonzalez A1, Knight R4,5,6,7.


2018/10
https://www.kinokuniya.co.jp/f/dsg-02-9789811315336
ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ãƒ»ãƒ‡ãƒ¼ã‚¿ã®ï¼²çµ±è¨ˆè§£æ

        3.3.2 UniFrac Distance Metric Family       49 (1)

      9.5 Compare Microbiome Communities Using     320(7)
      the GUniFrac Package
        9.5.1 Introduction to UniFrac, Weighted    320(2)
        UniFrac and Generalized UniFrac
        Distance Metrics
        9.5.2 Breast Milk Data Set                 322(1)
        9.5.3 Comparing Microbiome Communities     323(4)
        Using the GUniFrac Package

https://link.springer.com/book/10.1007/978-981-13-1534-3
Statistical Analysis of Microbiome Data with R | SpringerLink

https://link.springer.com/chapter/10.1007/978-981-13-1534-3_3
Introductory Overview of Statistical Analysis of Microbiome Data
3.3.2 UniFrac Distance Metric Family

https://link.springer.com/chapter/10.1007/978-981-13-1534-3_9
Multivariate Community Analysis | SpringerLink
9.5 Compare Microbiome Communities Using the GUniFrac Package


2016-09-15
https://pubmed.ncbi.nlm.nih.gov/27632205/
PLoS One
. 2016 Sep 15;11(9):e0161196. doi: 10.1371/journal.pone.0161196. eCollection 2016.
Expanding the UniFrac Toolbox
Ruth G Wong 1, Jia R Wu 1, Gregory B Gloor 1
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0161196
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5025018/

2015-07-07
https://pubmed.ncbi.nlm.nih.gov/26150095/
BMC Bioinformatics
. 2015 Jul 7;16(1):211. doi: 10.1186/s12859-015-0640-y.
The UniFrac significance test is sensitive to tree topology
Catherine A Lozupone 1, Rob Knight 2
https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0640-y
https://pmc.ncbi.nlm.nih.gov/articles/PMC4492014/

2012-08-15
https://pubmed.ncbi.nlm.nih.gov/22711789/
Comparative Study Bioinformatics
. 2012 Aug 15;28(16):2106-13. doi: 10.1093/bioinformatics/bts342. Epub 2012 Jun 17.
Associating microbiome composition with environmental covariates using generalized UniFrac distances
Jun Chen 1, Kyle Bittinger, Emily S Charlson, Christian Hoffmann, James Lewis, Gary D Wu, Ronald G Collman, Frederic D Bushman, Hongzhe Li
https://academic.oup.com/bioinformatics/article/28/16/2106/324465?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413390/

2011-02
https://pubmed.ncbi.nlm.nih.gov/20827291/
ISME J
. 2011 Feb;5(2):169-72. doi: 10.1038/ismej.2010.133. Epub 2010 Sep 9.
UniFrac: an effective distance metric for microbial community comparison
Catherine Lozupone 1, Manuel E Lladser, Dan Knights, Jesse Stombaugh, Rob Knight
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105689
https://www.nature.com/articles/ismej2010133

2005-12
https://pubmed.ncbi.nlm.nih.gov/16332807/
Comparative Study Appl Environ Microbiol
. 2005 Dec;71(12):8228-35. doi: 10.1128/AEM.71.12.8228-8235.2005.
UniFrac: a new phylogenetic method for comparing microbial communities
Catherine Lozupone 1, Rob Knight
https://journals.asm.org/doi/10.1128/aem.71.12.8228-8235.2005
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1317376/



å¹³æˆ 28 å¹´ 3 æœˆç‰ˆï¼ˆVer.0.9ï¼‰
https://www.nite.go.jp/data/000081956.pdf
æ¬¡ä¸–ä»£ã‚·ãƒ¼ã‚±ãƒ³ã‚µãƒ¼ã‚’ç”¨ã„ãŸèŒå¢è§£æ
ï¼ˆ16S rRNA éºä¼å­ PCR ã‚µãƒ³ãƒ—ãƒ«ã®è§£æï¼‰
ç‹¬ç«‹è¡Œæ”¿æ³•äºº è£½å“è©•ä¾¡æŠ€è¡“åŸºç›¤æ©Ÿæ§‹
ãƒã‚¤ã‚ªãƒ†ã‚¯ãƒãƒ­ã‚¸ãƒ¼ã‚»ãƒ³ã‚¿ãƒ¼
2.5 UniFrac è§£æ
æ¯”è¼ƒã™ã‚‹ã‚µãƒ³ãƒ—ãƒ«ã® OTU ä»£è¡¨é…åˆ—ã‚’ç”¨ã„ã¦ç³»çµ±æ¨¹è§£æã‚’è¡Œã„ã€è©¦æ–™é–“ã§å…±æœ‰ã•ã‚Œã‚‹OTU ã®æé•·ã¨å„è©¦æ–™ã§å›ºæœ‰ãªæé•·ã®å‰²åˆã‹ã‚‰ã€èŒå¢æ§‹é€ ã®é•ã„ã‚’è·é›¢ UniFrac distance ã¨ã—ã¦è¨ˆç®—ã™ã‚‹è§£ææ–¹æ³•ã‚’ã„ã†ã€‚ç®—å‡ºã•ã‚ŒãŸ UniFrac distance ã‚’ç”¨ã„ã¦ã€ä¸»åº§æ¨™åˆ†æ(PCoA:Principal Coordinate Analysis)ã‚„ UPGMA æ³•ã«ã‚ˆã‚‹ã‚¯ãƒ©ã‚¹ã‚¿ãƒªãƒ³ã‚°è§£æã‚’è¡Œã†ã“ã¨ã§ã€è©¦æ–™é–“ã®ç›¸é•åº¦ã‚’è¦–è¦šåŒ–ã™ã‚‹ã“ã¨ãŒã§ãã‚‹ã€‚ã¾ãŸã€ãƒªãƒ¼ãƒ‰æ•°ã‚’è€ƒæ…®ã—ç´°èŒå¢ã®æ§‹æˆã‚’è¡¨ã™ Weighted UniFrac è§£æã¨ã€ãƒªãƒ¼ãƒ‰æ•°ã‚’è€ƒæ…®ã›ãšç´°èŒå¢ã®æ§‹æˆãƒ¡ãƒ³ãƒãƒ¼ã‚’è¡¨ã™ UnweightedUniFrac è§£æãŒã‚ã‚‹ã€‚

2014 å¹´
https://www.jstage.jst.go.jp/article/jsci/37/5/37_412/_article/-char/ja
ç‰¹é›†ï¼šè…¸å†…ç´°èŒã¨å…ç–«ç–¾æ‚£
ãƒ’ãƒˆè…¸å†…ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ è§£æã®ãŸã‚ã®æœ€æ–°æŠ€è¡“
æœéƒ¨ æ­£å¹³
https://www.jstage.jst.go.jp/article/jsci/37/5/37_412/_pdf/-char/ja
äºŒã¤ç›®ã¯æ¤œå‡ºã•ã‚ŒãŸ OTU é–“ã®é…åˆ—é¡ä¼¼åº¦ã‹ã‚‰å„ç´°èŒå¢ã®ç³»çµ±æ¨¹ã‚’ä½œæˆã—ï¼Œãã®ç³»çµ±æ¨¹ã®é¡ä¼¼æ€§ï¼ˆæ¯”è¼ƒã™ã‚‹ç´°èŒå¢ã¨ã®ç³»çµ±æ¨¹é–“ã§å…±æœ‰ã™ã‚‹æã®é•·ã•ã¨ãã‚Œãã‚Œã®ç´°èŒå¢ã«å›ºæœ‰ãªæã®å‰²åˆï¼‰ã‹ã‚‰ï¼Œç´°èŒå¢é–“ã®å…¨ä½“æ§‹é€ ã®ç›¸é•ã®ç¨‹åº¦ã‚’æ±‚ã‚ã‚‹ï¼ã“ã®è§£æã‚’ UniFrac è§£æã¨è¨€ã„ï¼Œç´°èŒå¢é–“ã®é¡ä¼¼æ€§ã‚’ 0ï¼ˆ100% é¡ä¼¼ã™ã‚‹ï¼‰ï½ 1ï¼ˆ100% é¡ä¼¼ã—ãªã„ï¼‰ã®è·é›¢ï¼ˆUniFrac è·é›¢ï¼‰ã§è¡¨ã™6ï¼‰ï¼UniFrac è§£æã«ã¯OTU ã«å«ã¾ã‚Œã‚‹ãƒªãƒ¼ãƒ‰æ•°ï¼ˆçµ„æˆæ¯”ï¼‰ã‚’è€ƒæ…®ã—ãªã„unweighted ã¨è€ƒæ…®ã—ãŸ weighted UniFrac è§£æãŒã‚ã‚‹ï¼å‰è€…ã¯èŒç¨®ã®æœ‰ç„¡ã ã‘ãŒï¼Œå¾Œè€…ã¯åŒä¸€èŒç¨®ã®çµ„æˆæ¯”ã®ç›¸é•ãŒä¸¡ç´°èŒå¢é–“ã®é¡ä¼¼æ€§ã®è·é›¢ã«åæ˜ ã•ã‚Œã‚‹ï¼




----------
2015-03-27

https://www.ncbi.nlm.nih.gov/pubmed/25815895
PLoS Comput Biol. 2015 Mar 27;11(3):e1004186. doi: 10.1371/journal.pcbi.1004186. eCollection 2015 Mar.
Explaining diversity in metagenomic datasets by phylogenetic-based feature weighting.
Albanese D1, De Filippo C2, Cavalieri D1, Donati C1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4376673/

Given that the sequences of marker genes are available, phylogenetic measures of diversity such as UniFrac [19,20] have proven to be able to identify subtle differences in the structures of microbial communities by weighting species abundances with the phylogenetic relationships amongst taxa.

Here we present PhyloRelief, a ranking strategy to identify the taxa significantly contributing to the differentiation of groups of amplicon metagenomic samples. 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4376673/figure/pcbi.1004186.g001/
Fig 1
Schema of the method.
B) Next, the matrix of the distances between the samples must be computed using a phylogenetic measure of Î²-diversity, such as weighted or unweighted UniFrac must be provided. C) The PhyloRelief strategy. 

We compared the performance of PhyloRelief coupled with the RF classifier to LEfSe [30], an algorithm that uses statistical tests for biomarker discovery, to MetaPhyl, a recent phylogeny-based method for the classification of microbial communities [31] and to Random Forest, used both as classifier and feature selection method. 

Software dependencies and availability
PhyloRelief is implemented in Python (http://www.python.org), and requires Python > = 2.7 with the NumPy/SciPy (http://www.scipy.org), Pandas (http://pandas.pydata.org/), DendroPy [43] and Statsmodels (http://statsmodels.sourceforge.net/) libraries. PhyloRelief software, scripts and data analyzed in this paper are available at http://compmetagen.github.io/phylorelief.

----------
## LEfSe
linear discriminant analysis (LDA) effect size (LEfSe) 
ç·šå½¢åˆ¤åˆ¥åˆ†æï¼ˆLDAï¼‰åŠ¹æœé‡ï¼ˆLEfSeï¼‰

### 2025-02-17
Commonly used compositional data analysis implementations are not advantageous in microbial differential abundance analyses benchmarked against biological ground truth
Samuel D. Gamboa-Tuz, Marcel Ramos, Eric Franzosa, Curtis Huttenhower, Nicola Segata, Sehyun Oh, Levi Waldron
bioRxiv 2025.02.13.638109; doi: https://doi.org/10.1101/2025.02.13.638109
This article is a preprint and has not been certified by peer review [what does this mean?].
https://www.biorxiv.org/content/10.1101/2025.02.13.638109v1.full
https://www.biorxiv.org/content/10.1101/2025.02.13.638109v1.full.pdf

### lefser
### 2024-11-28
https://github.com/waldronlab/lefser
https://pubmed.ncbi.nlm.nih.gov/39585730/
Bioinformatics
. 2024 Nov 28;40(12):btae707. doi: 10.1093/bioinformatics/btae707.
lefser: implementation of metagenomic biomarker discovery tool, LEfSe, in R
Asya Khleborodova 1 2, Samuel D Gamboa-Tuz 1 2, Marcel Ramos 1 2, Nicola Segata 3 4, Levi Waldron 1 2, Sehyun Oh 1 2
https://academic.oup.com/bioinformatics/article/40/12/btae707/7908399
https://pmc.ncbi.nlm.nih.gov/articles/PMC11665633/

### 2022

9:00 PM Â· May 19, 2022
https://x.com/Tyu_Shi/status/1527257881632210944
Hiroshi Mori on X: "LEfSeã¯éå¸¸ã«ã‚ˆãä½¿ã‚ã‚Œã¦ã„ã¾ã™ãŒã€LEfSeã®çµæœã«ã¤ã„ã¦å¤šé‡æ¤œå®šã®è£œæ­£ãŒã©ã®ã‚ˆã†ãªå ´åˆã«å¿…è¦ã‹ã¯LEfSeã®ã‚¢ãƒ«ã‚´ãƒªã‚ºãƒ ï¼ˆKruskal-Wallis testã€Mann-Whitney U test, ç·šå‹åˆ¤åˆ¥åˆ†æã€ã®3ã¤ã®çµ±è¨ˆæ‰‹æ³•ã‚’ã©ã®ã‚ˆã†ã«çµ„ã¿åˆã‚ã›ã‚‹ã®ã‹ï¼‰ã‚’ç†è§£ã—ãŸä¸Šã§æ…é‡ã«è€ƒãˆã‚‹å¿…è¦ãŒã‚ã‚Šã¾ã™ã€‚" / X

9:05 PM Â· May 19, 2022
https://x.com/Tyu_Shi/status/1527259100643749888
Hiroshi Mori on X: "LEfSeã‚‚å«ã‚ã¦ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ã®ç¾¤é–“æ¯”è¼ƒã®çµ±è¨ˆæ‰‹æ³•ã«ã¯æ§˜ã€…ãªã‚‚ã®ãŒã‚ã‚Šæœªã æ´»ç™ºã«ç ”ç©¶ãŒè¡Œã‚ã‚Œã¦ã„ã¾ã™ãŒã€ä¾‹ãˆã°ã€€https://t.co/Y5G276ySvtã€€ç­‰ã‚’èª­ã‚“ã§å„æ‰‹æ³•ã®ç‰¹æ€§ã‚„æ³¨æ„ç‚¹ç­‰ã«ã¤ã„ã¦ã‚¤ãƒ¡ãƒ¼ã‚¸ã‚’æŒã£ã¦ãŠãã¨è‰¯ã„ã§ã™ã€‚" / X

9:19 PM Â· May 19, 2022
https://x.com/Tyu_Shi/status/1527262591109988352
Hiroshi Mori on X: "LEfSeã¨å¤šé‡æ¤œå®šè£œæ­£ã¨ã®é–¢ä¿‚ã«ã¤ã„ã¦ã¯ã€LEfSeãŒä½•æ®µéšã‹ã«åˆ†ã‘ã¦æ¤œå®šã‚’è¡Œã„ã•ã‚‰ã«ãã®å¾Œç·šå‹åˆ¤åˆ¥åˆ†æã‚’è¡Œãªã£ã¦effect sizeã‚’å‡ºåŠ›ã™ã‚‹ã“ã¨ã‚‚ã‚ã£ã¦ã‚ã‹ã‚Šã«ããã€https://t.co/1BjIhZOLyiã€€ã®ã‚ˆã†ã«åº¦ã€…é–‹ç™ºè€…é”ã¨ãƒ¦ãƒ¼ã‚¶ãƒ¼ã¨ã®é–“ã§è­°è«–ã«ãªã£ã¦ã„ã¾ã™ã€‚" / X

2022-01-17
https://pubmed.ncbi.nlm.nih.gov/35039521/
Nat Commun
. 2022 Jan 17;13(1):342. doi: 10.1038/s41467-022-28034-z.
Microbiome differential abundance methods produce different results across 38 datasets
Jacob T Nearing # 1, Gavin M Douglas # 2, Molly G Hayes 3, Jocelyn MacDonald 4, Dhwani K Desai 5, Nicole Allward 6, Casey M A Jones 7, Robyn J Wright 7, Akhilesh S Dhanani 5, AndrÃ© M Comeau 5, Morgan G I Langille 5 7
https://www.nature.com/articles/s41467-022-28034-z
https://pmc.ncbi.nlm.nih.gov/articles/PMC8763921/

Mar 2020
https://forum.biobakery.org/t/multiple-comparison-correcting/258
Multiple comparison correcting - Downstream analysis and statistics / LEfSe - The bioBakery help forum

### 2011-06-24
https://www.ncbi.nlm.nih.gov/pubmed/21702898
Genome Biol. 2011 Jun 24;12(6):R60. doi: 10.1186/gb-2011-12-6-r60.
Metagenomic biomarker discovery and explanation.
Segata N1, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, Huttenhower C.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3218848/
LEfSe determines the features (organisms, clades, operational taxonomic units, genes, or functions) most likely to explain differences between classes
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3218848/figure/F6/
(a) Step 1, Kruskalâ€“Wallis
(b) Step 2, Wilcoxon (pairwise comparisons)
(c) Step 3, build a LDA model, effect size

LEfSeã®ã‚¹ãƒ†ãƒƒãƒ—

https://twbattaglia.gitbooks.io/introduction-to-qiime/content/lefse.html
LEfSe Â· An Introduction to QIIME 1.9.1

LEfSe (Linear discriminant analysis effect size) is a tool developed by the Huttenhower group to find biomarkers between 2 or more groups using relative abundances.

LEfSe uses a table of relative abundances which also includes sample identifiers and group meta data. 

https://picrust.github.io/picrust/tutorials/humann_tutorial.html#differential-abundance-analysis-with-lefse
Differential abundance analysis with LEfSe

https://doi.org/10.7875/togotv.2019.102
2019-04-12 LEfSeã‚’ä½¿ã£ã¦ç´°èŒå¢(Microbiome)ã®ç¾¤é–“æ¯”è¼ƒè§£æã¨ãƒã‚¤ã‚ªãƒãƒ¼ã‚«ãƒ¼æ¢ç´¢ã‚’è¡Œã†
Galaxy

2020/01/11
https://note.com/ytomy/n/n39b64afd3862
LEfSeã®ä½¿ç”¨æ³•
Galaxy

https://pediatricsurgery.hatenadiary.jp/archive/category/èŒå¢è§£æ
2020-09-21
Dockerã‚’ä½¿ã£ã¦LEfSeã‚’è¶…ç°¡å˜ã«å®Ÿè¡Œã™ã‚‹
2016-10-31
LEfSeã‚’å®Ÿéš›ã«ã‚„ã£ã¦ã¿ã‚‹
2016-10-31
LEfSeã‚’ã‚¤ãƒ³ã‚¹ãƒˆãƒ¼ãƒ«ã™ã‚‹

27 April 2017
https://pitagora-network.org/docs/galaxy-community-japan/Notes/Huttenhower_Lab_Workflows.html
LEfSe (Linear discriminant analysis Effect Size) ã¯ã€ã‚¯ãƒ©ã‚¹é–“ã®é•ã„ã‚’èª¬æ˜ã™ã‚‹å¯èƒ½æ€§ãŒæœ€ã‚‚é«˜ã„ ç‰¹å¾´ï¼ˆç”Ÿç‰©ã€ã‚¯ãƒ¬ãƒ¼ãƒ‰ã€OTUã€éºä¼å­ã€æ©Ÿèƒ½ï¼‰ã‚’æ±ºå®šã™ã‚‹ã€‚


----------
## 
kraken2 alpha beta diversity

----------

https://github.com/DerrickWood/kraken2/issues/80
Input assembled metagenoms Â· Issue #80 Â· DerrickWood/kraken2

https://github.com/DerrickWood/kraken2/issues/80#issuecomment-565222474
isardi commented on Dec 13, 2019
Hi @MichaelAngelou,
I have been trying to figure out how to calculate alpha and beta diversity using the Bracken output with no luck. What package do you use? Did you have to modify the output in anyway? I spent the whole day trying to figure this out so your input is very much appreciated. Thanks in advanced!

https://github.com/DerrickWood/kraken2/issues/80#issuecomment-565422169
ciemanek commented on Dec 13, 2019
Hi @isardi !
You can use R package vegan for calculating the diversity indices on Bracken output :)


----------

https://github.com/carpentries-lab/metagenomics-analysis
7 months ago

https://carpentries-lab.github.io/metagenomics-analysis/
Data Processing and Visualization for Metagenomics

----------

https://www.cell.com/cms/10.1016/j.isci.2020.100905/attachment/d3d5d291-4f11-4ecf-af92-558af5d1f474/mmc7
Data S1: Code Used for Analysis, Related to Figures 1-6
This file includes all of the analysis code used for QIIME2, Kraken2 and Bracken2, Phyloseq,
HUMANn2, and DeepARG.
All metagenomic processing was run using the Brown University OSCAR computing cluster

https://pubmed.ncbi.nlm.nih.gov/32106056/
iScience
. 2020 Mar 27;23(3):100905. doi: 10.1016/j.isci.2020.100905. Epub 2020 Feb 13.
Antimicrobial Resistance Gene Prevalence in a Population of Patients with Advanced Dementia Is Related to Specific Pathobionts
Aislinn D Rowan-Nash 1, Rafael Araos 2, Erika M C D'Agata 3, Peter Belenky 4
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7044522/

Associated Data
Supplementary Materials

Data S1. Code Used for Analysis, Related to Figures 1â€“6 This file includes all of the analysis code used for QIIME2, Kraken2 and Bracken2, Phyloseq, HUMANn2, and DeepARG.
All metagenomic processing was run using the Brown University OSCAR computing cluster.

mmc7.pdf (100K)

----------
## unclassified

http://clark.cs.ucr.edu
CLARK :: Fast and accurate classification method for metagenomic/genomic sequences

https://www.ncbi.nlm.nih.gov/pubmed/27540266
Bioinformatics. 2016 Dec 15;32(24):3823-3825. Epub 2016 Aug 18.
Higher classification sensitivity of short metagenomic reads with CLARK-S.
Ounit R1, Lonardi S1.

https://www.ncbi.nlm.nih.gov/pubmed/25879410
BMC Genomics. 2015 Mar 25;16:236. doi: 10.1186/s12864-015-1419-2.
CLARK: fast and accurate classification of metagenomic and genomic sequences using discriminative k-mers.
Ounit R1, Wanamaker S2, Close TJ3, Lonardi S4.

- [Kraken](http://ccb.jhu.edu/software/kraken/)
 - è«–æ–‡ [Wood DE, Salzberg SL. Genome Biol. 2014 "Kraken: ultrafast metagenomic sequence classification using exact alignments."]()



2019å¹´09æœˆ02æ—¥
https://qiita.com/xvtyzn/items/6a008b61d852dcbaed57
Qiime2ã‚’ä½¿ã£ãŸãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ‘ã‚¤ãƒ—ãƒ©ã‚¤ãƒ³

http://fox-bioinformatics.net/2019/01/05/qiime2-how-to-install/
QIIME2ã‚¤ãƒ³ã‚¹ãƒˆãƒ¼ãƒ«ã®æ–¹æ³•è§£èª¬ | Rã¨ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹


https://twitter.com/sjackman/status/1045005404227362817
Shaun Jackman on Twitter: "I have a 16 kbp query sequence, and I'd like to search all of SRA (including metagenomes) to see if any SRA read sets match this query sequence. Does this tool exist?"
1:41 PM - 26 Sep 2018

https://twitter.com/i/moments/1000738412117147648
All metagenomes searched at once?

https://twitter.com/tylerbarnumphd/status/1010247842772815872
Tyler Barnum on Twitter: "I wrote a tutorial on how to find new proteins in the ~8,000 Uncultivated Bacteria and Archaea (UBA) genomes from @donovan_parks. It's the largest collection of genomes that you can't search for on BLAST or IMG. ~1,500 metagenomes. https://t.co/PgavpbmsQg Feedback welcome!"
3:47 PM - 22 Jun 2018
https://tylerbarnum.wordpress.com/2018/06/22/searching-uncultivated-bacteria-and-archaea-uba-genomes-for-important-genes/
Searching Uncultivated Bacteria and Archaea (UBA) Genomes for Important Genes â€“ The Biolo-Gist
June 22, 2018


https://www.fifthdimension.jp/documents/metabarcodingtextbook/metabarcodingtextbook.ja.pdf
https://www.fifthdimension.jp/documents/metabarcodingtextbook/metabarcodingtextbook.ja.html
ç”Ÿæ…‹å­¦ã®ãŸã‚ã®ãƒ¡ã‚¿ãƒãƒ¼ã‚³ãƒ¼ãƒ‡ã‚£ãƒ³ã‚°ã¨DNAãƒãƒ¼ã‚³ãƒ¼ãƒ‡ã‚£ãƒ³ã‚°
ç”°è¾ºæ™¶å²
2018å¹´5æœˆ8æ—¥

----------
## 2022


----------

### MuDoGeR
æŸ»èª­å‰

https://github.com/mdsufz/MuDoGeR
mdsufz/MuDoGeR: MuDoGeR makes the recovery of genomes from prokaryotes, viruses, and eukaryotes from metagenomes easy.
Multi-Domain Genome Recovery v1.0 (MuDoGeR v1.0)

https://twitter.com/ulisses_rocha/status/1538814305965068290
Dr. Ulisses Rocha (he/him) on Twitter: "Are you interested in recovering genomes from Prokaryotes, viruses, and/or Eukaryotes? We just released the MuDoGeR: https://t.co/gZ7qL9KDk1 Let us know what you think about the tool. The manuscript will be uploaded to @biorxivpreprint in the next few days. #bioinformatics https://t.co/QbvZ9rYBz6" / Twitter

Posted June 24, 2022.
https://www.biorxiv.org/content/10.1101/2022.06.21.496983v2
MuDoGeR: Multi-Domain Genome Recovery from metagenomes made easy | bioRxiv

Only 8 of the 574 metagenomic libraries showed no recovery of Prokaryotic metagenome-assembled genomes (MAGs), putative viral contigs, or Eukaryotic metagenome-assembled bins (MABs) (Supplementary Table 1). In the following paragraphs, we detailed the data for recovering Prokaryotic MAGs, high-quality and complete uncultivated viral genomes (UViGs), and Eukaryotic MABs. Nevertheless

----------
### 2022-03-07
### MEDUSA

https://github.com/dalmolingroup/MEDUSA_supplements

8:05 PM Â· Mar 7, 2022
https://twitter.com/DalmolinRodrigo/status/1500789806795788288
Rodrigo Dalmolin on Twitter: "A new paper by our group published today. Medusa is a complete, freely available, modular pipeline for shotgun metagenomics with high performance, especially for identifying rare species. https://t.co/d3tpnWfGmr https://t.co/GuZMWNjlpk" / Twitter

https://pubmed.ncbi.nlm.nih.gov/35330728/
Front Genet
. 2022 Mar 7;13:814437. doi: 10.3389/fgene.2022.814437. eCollection 2022.
MEDUSA: A Pipeline for Sensitive Taxonomic Classification and Flexible Functional Annotation of Metagenomic Shotgun Sequences
Diego A A Morais 1, JoÃ£o V F Cavalcante 1, ShÃªnia S Monteiro 2, Matheus A B Pasquali 2 3, Rodrigo J S Dalmolin 1 4
https://www.frontiersin.org/articles/10.3389/fgene.2022.814437/full


----------
## 2021
----------
## 2021-10-26
## kaiju

https://anaconda.org/bioconda/kaiju
conda install -c bioconda kaiju

https://kaiju.binf.ku.dk/
Kaiju: Fast and sensitive taxonomic classification for metagenomics

Version 1.8.2
2021-10-26

https://github.com/bioinformatics-centre/kaiju

https://github.com/bioinformatics-centre/kaiju/blob/master/README.md#creating-the-reference-database-and-index

https://github.com/bioinformatics-centre/kaiju/blob/master/README.md#running-kaiju

For paired-end reads use -i firstfile.fastq and -j secondfile.fastq.

Kaiju can read input files in FASTQ and FASTA format, which may also be gzip-compressed.

https://pubmed.ncbi.nlm.nih.gov/27071849/
Nat Commun
. 2016 Apr 13;7:11257. doi: 10.1038/ncomms11257.
Fast and sensitive taxonomic classification for metagenomics with Kaiju
Peter Menzel 1, Kim Lee Ng 1, Anders Krogh 1
https://www.nature.com/articles/ncomms11257
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4833860/

In such samples using protein-level classification can increase accuracy, because protein sequences are more conserved than the underlying DNA, and microbial and viral genomes are typically densely packed with protein-coding genes13,14. 


Results
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4833860/#__sec1title

Protein-level sequence classification
Kaiju translates metagenomic sequencing reads into the six possible reading frames and searches for maximum exact matches (MEMs) of amino acid sequences in a given database of annotated proteins from microbial reference genomes. If matches to one or more database sequences are found for a read, Kaiju outputs the taxonomic identifier of the corresponding taxon, or it determines the LCA in the case of equally good matches to different taxa. Kaiju's underlying sequence comparison algorithm uses the Burrowsâ€“Wheeler transform (BWT) of the protein database, which enables exact string matching in time proportional to the length of the query, to achieve a high classification speed.



We compared the performance of Kaiju with the two k-mer-based programs Kraken and Clark, which performed best in speed and accuracy in a recent benchmark study18. 


https://kazumaxneo.hatenablog.com/entry/2018/05/23/121418
ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã‚’ä½¿ã£ã¦é«˜æ„Ÿåº¦ã«ãƒ¡ã‚¿ã‚²ãƒãƒ ã®taxonomy assignmentã‚’è¡Œã† kaiju - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹


----------
## 2020
----------

----------
### 2020-12-14
### metaphlan

https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-3.0
Francesco Beghini edited this page on Dec 14, 2020 Â· 13 revisions

https://kazumaxneo.hatenablog.com/entry/2022/02/21/220453
MetaPhlAn3 - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------
### 2020-11-02
### recentrifuge

https://sites.google.com/view/recentrifuge

https://github.com/khyox/recentrifuge/wiki
Jose Manuel MartÃ­ edited this page on Nov 2, 2020 Â· 23 revisions

https://pubmed.ncbi.nlm.nih.gov/30958827/
Comparative Study PLoS Comput Biol
. 2019 Apr 8;15(4):e1006967. doi: 10.1371/journal.pcbi.1006967. eCollection 2019 Apr.
Recentrifuge: Robust comparative analysis and contamination removal for metagenomics
Jose Manuel MartÃ­ 1
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006967
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6472834/

7:30 AM Â· Apr 20, 2019
https://twitter.com/kazumachack/status/1119367593821741056
Kazuma Uesaka on Twitter: "centrifugeã‚„krakenã€CLARKã€LMATã®çµæœã‚’å…¥åŠ›ã¨ã—ã¦ãƒã‚¬ã‚³ãƒ³ã®ãƒ¡ã‚¿ã‚²ãƒãƒ ã‚µãƒ³ãƒ—ãƒ«ã‚„ã‚³ãƒ³ã‚¿ãƒŸã‚’é™¤ãã€ã‚¨ãƒ³ãƒªãƒƒãƒã•ã‚Œã¦ã„ã‚‹ç”Ÿç‰©ã®åˆ†æã‚’åŠ©ã‘ã‚‹ãƒ„ãƒ¼ãƒ«ã§ã™ã€‚ Recentrifuge - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹ https://t.co/sBrRgFunfo" / Twitter

https://kazumaxneo.hatenablog.com/entry/2019/04/20/073000
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ã‚³ãƒ³ã‚¿ãƒŸé™¤å»ã‚„ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ã‚µãƒ³ãƒ—ãƒ«é–“æ¯”è¼ƒã‚’è¡Œã£ã¦çµæœã‚’è¦–è¦šåŒ–ã™ã‚‹ Recentrifuge - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹


----------
### 2020-03-29
### centrifuge

http://ccb.jhu.edu/software/centrifuge/index.shtml
Centrifuge
Classifier for metagenomic sequences

Due to the rapid spread of SARS-CoV-2 and its devastating effects, we provide additional Centrifuge indices in the hope that they will be useful for biomedical research related to the virus. The first two indexes include 106 complete SARS-CoV-2 genomes downloaded from GenBank as follows: (3/29/2020)

http://ccb.jhu.edu/software/centrifuge/manual.shtml

https://pubmed.ncbi.nlm.nih.gov/27852649/
Genome Res
. 2016 Dec;26(12):1721-1729. doi: 10.1101/gr.210641.116. Epub 2016 Oct 17.
Centrifuge: rapid and sensitive classification of metagenomic sequences
Daehwan Kim 1, Li Song 1 2, Florian P Breitwieser 1, Steven L Salzberg 1 2 3
https://genome.cshlp.org/content/26/12/1721.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5131823/
Data access
Centrifuge is available as free, open-source software from https://github.com/infphilo/centrifuge/archive/centrifuge-genome-research.zip and provided in Supplemental Data S1. The fruitshake sequencing data from this study have been submitted to the NCBI BioProject database (http://www.ncbi.nlm.nih.gov/bioproject/) under accession number PRJNA343503.


https://kazumaxneo.hatenablog.com/entry/2018/05/30/150433
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ãƒªãƒ¼ãƒ‰ã®ç³»çµ±ã‚¢ã‚µã‚¤ãƒ³ãƒ¡ãƒ³ãƒˆã‚’è¡Œã† Centrifuge - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹



----------
### 2020-12
### kraken2

https://ccb.jhu.edu/software/kraken2/
2020/12 UPDATE
https://ccb.jhu.edu/software/kraken2/index.shtml?t=downloads
2020/06 UPDATE


https://pubmed.ncbi.nlm.nih.gov/31779668/
Genome Biol
. 2019 Nov 28;20(1):257. doi: 10.1186/s13059-019-1891-0.
Improved metagenomic analysis with Kraken 2
Derrick E Wood 1 2, Jennifer Lu 2 3, Ben Langmead 4 5
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1891-0
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6883579/

https://pubmed.ncbi.nlm.nih.gov/24580807/
Comparative Study Genome Biol
. 2014 Mar 3;15(3):R46. doi: 10.1186/gb-2014-15-3-r46.
Kraken: ultrafast metagenomic sequence classification using exact alignments
Derrick E Wood, Steven L Salzberg
https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r46
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053813/


æ›´æ–°æ—¥ 2020å¹´11æœˆ25æ—¥
https://qiita.com/Adaachill/items/ec08e7701c9975616f36
metagenome ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚¹ã‹ã‚‰é«˜é€Ÿã«å„readã®ç³»çµ±ã‚’æ¨å®šã™ã‚‹Kraken2[ç’°å¢ƒæ§‹ç¯‰]

https://kazumaxneo.hatenablog.com/entry/2017/08/28/235248
ãƒ¡ã‚¿ã‚²ãƒãƒ ãƒ‡ãƒ¼ã‚¿ã®é«˜é€Ÿãªtaxonomy assignmentã‚’è¡Œã† kraken - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------
### 2021-10-09
### krakenuniq

https://github.com/fbreitwieser/krakenuniq

https://github.com/fbreitwieser/krakenuniq/blob/master/MANUAL.md
Pavian works great for visualizing and analyzing KrakenUniq results!

https://pubmed.ncbi.nlm.nih.gov/30445993/
Genome Biol
. 2018 Nov 16;19(1):198. doi: 10.1186/s13059-018-1568-0.
KrakenUniq: confident and fast metagenomics classification using unique k-mer counts
F P Breitwieser 1, D N Baker 2 3, S L Salzberg 4 5 6
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1568-0
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6238331/

https://kazumaxneo.hatenablog.com/entry/2019/01/16/073000
KrakenUniq - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

----------
### 2019-08-07
### bracken

http://ccb.jhu.edu/software/bracken/
08/07/2019 - v2.5.0 release

https://peerj.com/articles/cs-104/
Bracken: estimating species abundance in metagenomics data [PeerJ]

Software and data availability
Bracken is written in Perl and Python and is freely available for download at http://ccb.jhu.edu/software/bracken/. The reads from the skin microbiome experiment are freely available from NCBI under BioProject PRJNA316735.

https://kazumaxneo.hatenablog.com/entry/2019/02/01/073000
Bracken - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹


----------
## web-tool

### manta

https://mizuguchilab.org/ja
æ°´å£è³¢å¸ç ”ç©¶å®¤

https://mizuguchilab.org/manta/
MANTA
Microbiota And pheNotype correlaTion Analysis platform

An integrative database and analysis platform that relates microbiome and phenotypic data


https://pubmed.ncbi.nlm.nih.gov/33275647/
PLoS One
. 2020 Dec 4;15(12):e0243609. doi: 10.1371/journal.pone.0243609. eCollection 2020.
MANTA, an integrative database and analysis platform that relates microbiome and phenotypic data
Yi-An Chen 1, Jonguk Park 1, Yayoi Natsume-Kitatani 1, Hitoshi Kawashima 1, Attayeb Mohsen 1, Koji Hosomi 2, Kumpei Tanisawa 3, Harumi Ohno 3, Kana Konishi 3, Haruka Murakami 3, Motohiko Miyachi 3, Jun Kunisawa 2, Kenji Mizuguchi 1


https://kazumaxneo.hatenablog.com/entry/2021/06/09/073000
ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ã¨è¡¨ç¾å‹ã®ãƒ‡ãƒ¼ã‚¿ã‚’é–¢é€£ä»˜ã‘ã‚‹çµ±åˆãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹è§£æãƒ—ãƒ©ãƒƒãƒˆãƒ•ã‚©ãƒ¼ãƒ  MANTA - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

MicrobiomeAnalystã€Calypsoã€Shiny-phyloseqã€ãŠã‚ˆã³Mianã¯ã€ã“ã‚Œã‚‰ã®èª²é¡Œã«å¯¾å‡¦ã™ã‚‹ãŸã‚ã®ã‚¦ã‚§ãƒ–ãƒ™ãƒ¼ã‚¹ã®ã‚ªãƒ³ãƒ©ã‚¤ãƒ³ãƒ„ãƒ¼ãƒ«ã§ã‚ã‚‹ã€‚ã“ã‚Œã‚‰ã®ãƒ„ãƒ¼ãƒ«ã¯ã€Rï¼ˆphyloseqã€veganã€ade4ãªã©ï¼‰ã‚„Pythonãƒ‘ãƒƒã‚±ãƒ¼ã‚¸ã‚’ä»²ä»‹ã™ã‚‹ãŸã‚ã®å¯¾è©±çš„ãªã‚¦ã‚§ãƒ–ã‚¤ãƒ³ã‚¿ãƒ¼ãƒ•ã‚§ãƒ¼ã‚¹ã‚’æä¾›ã—ã¦ã„ã‚‹ã€‚





### pavian
https://github.com/fbreitwieser/pavian
Pavian is a interactive browser application for analyzing and visualization metagenomics classification results from classifiers such as Kraken, KrakenUniq, Kraken 2, Centrifuge and MetaPhlAn. Pavian also provides an alignment viewer for validation of matches to a particular genome.

https://fbreitwieser.shinyapps.io/pavian/

https://github.com/fbreitwieser/pavian/blob/master/README.md#supported-formats
Supported formats
pavian natively supports the Kraken and MetaPhlAn-style report formats. In extension, you can use Centrifuge results by running centrifuge-kreport on Centrifuge output files, and Kaiju results by running kraken-report on Kaiju output files (see issue #11)

### 2020-02-15
https://pubmed.ncbi.nlm.nih.gov/31553437/
Bioinformatics
. 2020 Feb 15;36(4):1303-1304. doi: 10.1093/bioinformatics/btz715.
Pavian: interactive analysis of metagenomics data for microbiome studies and pathogen identification
Florian P Breitwieser 1, Steven L Salzberg 2
https://academic.oup.com/bioinformatics/article/36/4/1303/5573755
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8215911/

https://kazumaxneo.hatenablog.com/entry/2018/05/30/210209
ãƒ¡ã‚¿ã‚²ãƒãƒ åˆ†æãƒ„ãƒ¼ãƒ« Pavian - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

åˆ†é¡ã¨è¦–è¦šåŒ–ã‚’çµ±åˆã—ãŸTaxonomerï¼ˆFlygare et alã€2016ï¼‰ï¼ˆç´¹ä»‹ï¼‰ã¨æ¯”è¼ƒã—ã¦ã€Pavianã¯ã„ãã¤ã‹ã®ã‚µãƒ³ãƒ—ãƒ«ã®è¦–è¦šåŒ–ã¨æ¯”è¼ƒã®ãŸã‚ã®è¿½åŠ ã®æ–¹æ³•ã‚’æä¾›ã™ã‚‹ã€‚

### 2016-05-26
### taxonomer

https://taxonomer.iobio.io/
Taxonomer is a kmer-based ultrafast metagenomics tool for assigning taxonomy to sequencing reads from both clinical and environmental samples.

https://pubmed.ncbi.nlm.nih.gov/27224977/
Genome Biol
. 2016 May 26;17(1):111. doi: 10.1186/s13059-016-0969-1.
Taxonomer: an interactive metagenomics analysis portal for universal pathogen detection and host mRNA expression profiling
Steven Flygare 1, Keith Simmon 2, Chase Miller 1, Yi Qiao 1, Brett Kennedy 1, Tonya Di Sera 1, Erin H Graf 3, Keith D Tardif 4, AurÃ©lie Kapusta 1, Shawn Rynearson 1, Chris Stockmann 5, Krista Queen 6, Suxiang Tong 6, Karl V Voelkerding 3 4, Anne Blaschke 5, Carrie L Byington 5, Seema Jain 6, Andrew Pavia 5, Krow Ampofo 5, Karen Eilbeck 2 7, Gabor Marth 1 7, Mark Yandell 8 9, Robert Schlaberg 10 11
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0969-1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4880956/

https://kazumaxneo.hatenablog.com/entry/2018/05/31/170700
ãƒ¡ã‚¿ã‚²ãƒãƒ ã‚’åˆ†é¡ã—ã€çµæœã‚’å¯è¦–åŒ–ã™ã‚‹ Taxonomer - macã§ã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹

ã€€Taxonomerã¯ã€è¶…é«˜é€Ÿã€ãƒ¦ãƒ¼ã‚¶ãƒ¼ãƒ•ãƒ¬ãƒ³ãƒ‰ãƒªãƒ¼ãªã‚¦ã‚§ãƒ–ãƒ™ãƒ¼ã‚¹ã®ãƒ¡ã‚¿ã‚²ãƒãƒ é…åˆ—è§£æãƒ„ãƒ¼ãƒ«ã§ã‚ã‚‹ã€‚

Taxonomer webã‚µãƒ¼ãƒ“ã‚¹ã«ã‚¢ã‚¯ã‚»ã‚¹ã™ã‚‹ã€‚ä½¿ç”¨ã™ã‚‹ã«ã¯Sign upãŒå¿…è¦ã«ãªã‚‹ã€‚Sign up for freeã‹ã‚‰ç™»éŒ²ã™ã‚‹ã€‚

----------
## benchmark

### 2022-03-24

å¤ä»£ã‚¦ã‚¤ãƒ«ã‚¹DNAã«é–¢ã™ã‚‹ãƒ¡ã‚¿ã‚²ãƒãƒ åˆ†é¡æ³•ã®ãƒ™ãƒ³ãƒãƒãƒ¼ã‚¯ã€‚4ã¤ã®åˆ†é¡æ³•ï¼ˆCentrifugeã€Kraken2ã€DIAMONDã€MetaPhlAn2ï¼‰ã«ã¤ã„ã¦ã€ãƒªãƒ¼ãƒ‰é…åˆ—ã®ç”±æ¥ã™ã‚‹ã‚¦ã‚¤ãƒ«ã‚¹ã‚’åŒå®šã™ã‚‹æ€§èƒ½ã‚’è©•ä¾¡ã—ãŸçµæœã€CentrifugeãŒæœ€é«˜ã®æ„Ÿåº¦ã¨ç²¾åº¦ã‚’é”æˆã—ãŸæ¨å¥¨ãƒ„ãƒ¼ãƒ«ã€‚
The authors compared Centrifuge, Kraken2, DIAMOND and MetaPhlAn2. Centrifuge is their recommended tool.
https://pubmed.ncbi.nlm.nih.gov/35356467/
PeerJ
. 2022 Mar 24;10:e12784. doi: 10.7717/peerj.12784. eCollection 2022.
Benchmarking metagenomics classifiers on ancient viral DNA: a simulation study
Yami Ommar Arizmendi CÃ¡rdenas 1 2, Samuel Neuenschwander 1 3, Anna-Sapfo Malaspinas 1 2
https://peerj.com/articles/12784/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8958974/
Across most simulations, more than 228 out of the 233 simulated viruses were recovered by Centrifuge, Kraken2 and DIAMOND, in contrast to MetaPhlAn2 which recovered only around one third. Overall, Centrifuge and Kraken2 had the best performance with the highest values of sensitivity and precision. We found that deamination damage had little impact on the performance of the classifiers, less than the sequencing error and the length of the reads. Since Centrifuge can handle short reads (in contrast to DIAMOND and Kraken2 with default settings) and since it achieve the highest sensitivity and precision at the species level across all the simulations performed, it is our recommended tool. 

### 2019-08-08
### Ye_2019

https://pubmed.ncbi.nlm.nih.gov/31398336/
Review Cell
. 2019 Aug 8;178(4):779-794. doi: 10.1016/j.cell.2019.07.010.
Benchmarking Metagenomics Tools for Taxonomic Classification
Simon H Ye 1, Katherine J Siddle 2, Daniel J Park 3, Pardis C Sabeti 4
https://www.sciencedirect.com/science/article/pii/S0092867419307755
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6716367/

Several groups have previously benchmarked metagenomic tools (Lindgreen et al., 2016, Mavromatis et al., 2007, McIntyre et al., 2017, Meyer et al., 2019, Sczyrba et al., 2017), but the continual introduction of newer tools requires ongoing evaluation to compare them against established tools. 

Evaluating the precision-recall of 20 classifiers
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6716367/#S4title
Here we benchmarked 20 metagenomic classifiers to compare performance in classification precision, recall, F1, speed, and other metrics using a uniform database to eliminate any confounding effects of differences in default databases. DNA to DNA classifiers evaluated here were Kraken (and its add-on for more accurate abundance quantification Bracken), Kraken2, KrakenUniq, k-SLAM, MegaBLAST, metaOthello, CLARK, CLARK-S, GOTTCHA, taxMaps, prophyle, PathSeq, Centrifuge, and Karp. DNA to Protein classifiers evaluated were DIAMOND, Kaiju, and MMseqs2. We

https://scholar.google.co.jp/scholar?cites=6688630901807222522&as_sdt=2005&sciodt=0,5&hl=ja

### 2017-09-21
https://pubmed.ncbi.nlm.nih.gov/28934964/
Genome Biol
. 2017 Sep 21;18(1):182. doi: 10.1186/s13059-017-1299-7.
Comprehensive benchmarking and ensemble approaches for metagenomic classifiers
Alexa B R McIntyre 1 2 3, Rachid Ounit 4, Ebrahim Afshinnekoo 2 3 5, Robert J Prill 6, Elizabeth HÃ©naff 2 3, Noah Alexander 2 3, Samuel S Minot 7, David Danko 1 2 3, Jonathan Foox 2 3, Sofia Ahsanuddin 2 3, Scott Tighe 8, Nur A Hasan 9 10, Poorani Subramanian 9, Kelly Moffat 9, Shawn Levy 11, Stefano Lonardi 4, Nick Greenfield 7, Rita R Colwell 9 12, Gail L Rosen 13, Christopher E Mason 14 15 16
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1299-7
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5609029/
Overall, pairing tools with different classification strategies (k-mer, alignment, marker) can combine their respective advantages.

https://scholar.google.co.jp/scholar?cites=8605030642958534701&as_sdt=2005&sciodt=0,5&hl=ja


----------
## people


### duvallet
Claire Duvallet

MITã®Eric Almãƒ©ãƒœã€‚å¾®ç”Ÿç‰©ç¾¤é›†ãƒ‡ãƒ¼ã‚¿ã®ãƒ¡ã‚¿ã‚¢ãƒŠãƒªã‚·ã‚¹ã€‚ãƒãƒƒãƒåŠ¹æœã€‚

https://twitter.com/cduvallet

7:42 PM - 23 Apr 2018
https://twitter.com/cduvallet/status/988563768765505536
Claire Duvallet on Twitter: "Meaning to make a meta-analysis but blocked or bothered by batch effects? Use your controls as nulls: normalize features in cases relative to their distribution in control samples. Learn more at @PLOSCompBiol https://t.co/PHyVZoZlej Congrats @gibbological!! @ejalm"

1:55 PM - 23 Apr 2018
https://twitter.com/gibbological/status/988476547614105601
Sean Gibbons on Twitter: "Befuddled by batch effects? Check out our simple, non-parametric solution for 16S case-control studies in @PLOSCompBiol https://t.co/grMDm7SIQl -- method available as a @qiime2 plugin thanks to @cduvallet"

April 17, 2018
https://cduvallet.github.io/posts/2018/04/microbiome-data
An empirical analysis of microbiome data availability - Claire Duvallet

19 January 2018
https://onlinelibrary.wiley.com/doi/full/10.1111/1751-7915.13047
Metaâ€analysis generates and prioritizes hypotheses for translational microbiome research - Duvallet - 2018 - Microbial Biotechnology - Wiley Online Library
Claire Duvallet

https://pubmed.ncbi.nlm.nih.gov/29209090/
2017 Dec 5;8(1):1784. doi: 10.1038/s41467-017-01973-8.
Meta-analysis of Gut Microbiome Studies Identifies Disease-Specific and Shared Responses
Claire Duvallet 1 2, Sean M Gibbons 1 2 3, Thomas Gurry 1 2 3, Rafael A Irizarry 4 5, Eric J Alm 6 7 8


05 December 2017
https://www.nature.com/articles/s41467-017-01973-8
Meta-analysis of gut microbiome studies identifies disease-specific and shared responses | Nature Communications
Claire Duvallet, Sean M. Gibbons, Thomas Gurry, Rafael A. Irizarry & Eric J. Alm
https://www.nature.com/articles/s41467-017-01973-8
We perform a cross-disease meta-analysis of these studies using standardized methods. 

----------
### Mori

https://researchmap.jp/himori/


11:35 AM Â· May 31, 2023
https://twitter.com/Tyu_Shi/status/1663736034977521664
Hiroshi Mori on X: "æ“¬ä¼¼çš„ãªãƒ’ãƒˆè…¸å†…ç´°èŒç¾¤é›†ã‚’ä½œè£½ã—æ§˜ã€…ãªè§£ææ‰‹æ³•ã‚’è©•ä¾¡ https://t.co/X9zWtvGZxQ è«–æ–‡ã¯ã“ã¡ã‚‰ https://t.co/PRyZREPTHg" / X
2023/05/31
https://www.nig.ac.jp/nig/ja/2023/05/research-highlights_ja/pr20230531.html
æ“¬ä¼¼çš„ãªãƒ’ãƒˆè…¸å†…ç´°èŒç¾¤é›†ã‚’ä½œè£½ã—æ§˜ã€…ãªè§£ææ‰‹æ³•ã‚’è©•ä¾¡
https://pubmed.ncbi.nlm.nih.gov/37253538/
DNA Res
. 2023 Jun 1;30(3):dsad010. doi: 10.1093/dnares/dsad010.
Assessment of metagenomic workflows using a newly constructed human gut microbiome mock community
Hiroshi Mori 1 2, Tamotsu Kato 3 4, Hiroaki Ozawa 3 4, Mitsuo Sakamoto 5, Takumi Murakami 1 2, Todd D Taylor 6, Atsushi Toyoda 1 7, Moriya Ohkuma 5, Ken Kurokawa 1 2, Hiroshi Ohno 3 4 6
https://academic.oup.com/dnaresearch/article/30/3/dsad010/7179629
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10229288/





2024.03.09
https://doi.org/10.7875/togotv.2024.011
ãƒ¡ã‚¿ã‚²ãƒãƒ ã®ã‚¢ãƒãƒ†ãƒ¼ã‚·ãƒ§ãƒ³ @ éºä¼ç ”ç ”ç©¶ä¼š Annotathon2023
11:54

2020.12.12
https://doi.org/10.7875/togotv.2020.090
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ (MicrobeDB.jp) @ AJACSã‚ªãƒ³ãƒ©ã‚¤ãƒ³3 | TogoTV
57:55

2020å¹´ï¼ˆIIBMP2020ï¼‰
https://www.jsbi.org/iibmp2020/program/sponsor.html
9/2ï¼ˆæ°´ï¼‰14:45ã€œ16:15	ãƒ‡ãƒ¼ã‚¿ã‚µã‚¤ã‚¨ãƒ³ãƒ†ã‚£ã‚¹ãƒˆé¤Šæˆã‚»ãƒƒã‚·ãƒ§ãƒ³ï¼šå®Ÿè·µçš„ã‚ªãƒŸãƒƒã‚¯ã‚¹è§£æ
æ£® å®™å²ï¼ˆå›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ï¼‰
ã€Œãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã€
https://www.youtube.com/watch?v=7mhsOuGYgYM&list=PL_dbAF_dbOEpcBhGhyv9Wvl2XU9MnmFUD&index=5
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æï¼ˆIIBMP2020ï¼‰ - YouTube
12:00 / 20:31
- 16S rRNA gene reference DB
  - SILVA
  - Greengenes
  - RDP

2020.12.12
https://doi.org/10.7875/togotv.2020.090
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ (MicrobeDB.jp) @ AJACSã‚ªãƒ³ãƒ©ã‚¤ãƒ³3
- https://youtu.be/pyoKka5K0Bg?t=2291
38:12 / 57:54
VITCOMIC2
- https://youtu.be/pyoKka5K0Bg?t=2559
42:40 / 57:54
LEA

2020.01.16
https://doi.org/10.7875/togotv.2020.016
å¾®ç”Ÿç‰©çµ±åˆãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹MicrobeDB.jpã®æ´»ç”¨æ³• @ AJACSä¸–ç”°è°· | TogoTV
æœ¬æ—¥ã®çµ±åˆTVã¯ã€2019å¹´11æœˆ29æ—¥ã«é–‹å‚¬ã•ã‚ŒãŸçµ±åˆãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹è¬›ç¿’ä¼š:AJACSä¸–ç”°è°·ã‹ã‚‰ã€å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ æ£® å®™å² æ° ã«ã‚ˆã‚‹ã€Œå¾®ç”Ÿç‰©çµ±åˆãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹MicrobeDB.jpã®æ´»ç”¨æ³•ã€ã‚’ãŠé€ã‚Šã—ã¾ã™ã€‚ç´„53åˆ†ã§ã™ã€‚
ç›®æ¨™:å…¬å…±ã®å¾®ç”Ÿç‰©ã®ã‚²ãƒãƒ ãƒ»ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ‡ãƒ¼ã‚¿ç­‰ã‚’MicrobeDB.jpã§æ¤œç´¢ã™ã‚‹æ–¹æ³•ã€ãŠã‚ˆã³ã€VITCOMIC2ã€LEAç­‰ã‚’ç”¨ã„ã¦ã€è‡ªåˆ†ã®æŒã¤ãƒã‚¤ã‚¯ãƒ­ãƒã‚¤ã‚ªãƒ¼ãƒ ã®ã‚¢ãƒ³ãƒ—ãƒªã‚³ãƒ³ã‚·ãƒ¼ã‚±ãƒ³ã‚¹ãƒ‡ãƒ¼ã‚¿ã‚„ãƒ¡ã‚¿ã‚²ãƒãƒ ã‚·ãƒ¼ã‚±ãƒ³ã‚¹ãƒ‡ãƒ¼ã‚¿ã‹ã‚‰ç³»çµ±çµ„æˆã‚’æ¨å®šã—ã€ç³»çµ±çµ„æˆãŒä¼¼ãŸMicrobeDB.jpä¸Šã®ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ‡ãƒ¼ã‚¿ã‚’æ¤œç´¢ã™ã‚‹æ–¹æ³•ãŒã‚ã‹ã‚‹ã€‚
https://youtu.be/SDus42JzFXc?t=2508
41:48 / 53:45

2019.12.16
https://doi.org/10.7875/ajacs.2019.033
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ„ãƒ¼ãƒ« | TogoTV
https://doi.org/10.7875/togotv.2019.203
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ„ãƒ¼ãƒ« @ AJACSåå‹2 | TogoTV
01:29:00

2019å¹´9æœˆ25æ—¥
https://github.com/AJACS-training/AJACS79
çµ±åˆãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹è¬›ç¿’ä¼šï¼šAJACS åå‹2
å¸¯åºƒç•œç”£å¤§å­¦ã€€æƒ…å ±å‡¦ç†ã‚»ãƒ³ã‚¿ãƒ¼1éšã€€ç¬¬1å®Ÿç¿’å®¤ ï¼ˆåŒ—æµ·é“å¸¯åºƒå¸‚ç¨²ç”°ç”ºè¥¿2ç·š11ç•ªåœ°ï¼‰
15:30-17:00ã€€ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ„ãƒ¼ãƒ« â€¦â€¦æ£® å®™å²ï¼ˆå›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ï¼‰
http://doi.org/10.7875/ajacs.2019.033
ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ„ãƒ¼ãƒ«ï½œçµ±åˆTV(togotv)
1:28:59

2019å¹´8æœˆ7æ—¥
https://doi.org/10.7875/togotv.2019.121
ã€How to use1ã€‘MicrobeDB.jpã®ä½¿ã„æ–¹ @ AJACSç•ªç”º3 | TogoTV
æœ¬æ—¥ã®çµ±åˆTVã¯ã€2019å¹´8æœˆ7æ—¥ã«é–‹å‚¬ã•ã‚ŒãŸçµ±åˆãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹è¬›ç¿’ä¼š:AJACSç•ªç”º3ã‹ã‚‰ã€å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ æ£® å®™å² åŠ©æ•™ã«ã‚ˆã‚‹ã€Œã€How to use1ã€‘MicrobeDB.jpã®ä½¿ã„æ–¹ã€ã‚’ãŠé€ã‚Šã—ã¾ã™ã€‚ç´„25åˆ†ã§ã™ã€‚
https://youtu.be/tdKrSdmmSz4?t=882
14:45 / 25:09
https://microbedb.jp/
Rickettsia
https://github.com/MicrobeDBjp/document/blob/master/Lecture/MicrobeDBjpver2/20180329JSB/lecture-keyword.md
building associatedï¼ˆå»ºç¯‰ç‰©é–¢é€£ï¼‰

http://doi.org/10.7875/ajacs.2017.016
é–‹å‚¬æ—¥æ™‚ï¼š2017-08-31
è¬›ç¿’ä¼šåï¼š 2017NGSãƒãƒ³ã‚ºã‚ªãƒ³
è¬›å¸«ï¼š æ£® å®™å²ã€€ï¼ˆ å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ï¼‰
ä¼šå ´ï¼š æ±äº¬å¤§å­¦ è¾²å­¦éƒ¨2å·é¤¨ 2éšç¬¬1è¬›ç¾©å®¤

https://doi.org/10.7875/togotv.2017.119
2017-12-06 ã€NGSãƒãƒ³ã‚ºã‚ªãƒ³2017ã€‘NGSè§£æ(åˆã€œä¸­ç´š) ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ
æœ¬æ—¥ã®çµ±åˆTVã¯ã€2017å¹´8æœˆ28æ—¥(æœˆ) ï½ 9æœˆ2æ—¥(é‡‘)ã«é–‹å‚¬ã•ã‚ŒãŸã€ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒãƒ†ã‚£ã‚¯ã‚¹äººæè‚²æˆã‚«ãƒªã‚­ãƒ¥ãƒ©ãƒ  å¹³æˆ29å¹´åº¦ NGSãƒãƒ³ã‚ºã‚ªãƒ³è¬›ç¿’ä¼š ã‹ã‚‰ã€å›½ç«‹éºä¼å­¦ç ”ç©¶æ‰€ æ£® å®™å² æ° ã«ã‚ˆã‚‹ã€ŒNGSè§£æ(åˆã€œä¸­ç´š) ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã€ã‚’ãŠé€ã‚Šã—ã¾ã™ã€‚ç´„5æ™‚é–“22åˆ†ã§ã™ã€‚

https://biosciencedbc.jp/gadget/human/170831_mori_170830.pdf
2017å¹´ NGSãƒãƒ³ã‚¹ã‚™ã‚ªãƒ³è¬›ç¿’ä¼š 8æœˆ31æ—¥
ãƒ¡ã‚¿ã‚±ã‚™ãƒãƒ è§£æ
æ£® å®™å²(HiroshiMori),Ph.D.

ä»£è¡¨çš„ãªãƒ¡ã‚¿ã‚±ã‚™ãƒãƒ ãƒ†ã‚™ãƒ¼ã‚¿ãƒ˜ã‚™ãƒ¼ã‚¹
MG-RAST
EBI-Metagenomics


https://www.slideshare.net/DDBJslide/ddbjing30-43012395
[DDBJing30] ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æã¨å¾®ç”Ÿç‰©çµ±åˆãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹
è¬›å¸«ï¼šæ£® å®™å²
2014å¹´12æœˆ18æ—¥


### Rocha

https://www.ncbi.nlm.nih.gov/pubmed/20090831
PLoS Genet. 2010 Jan 15;6(1):e1000808. doi: 10.1371/journal.pgen.1000808.
The systemic imprint of growth and its uses in ecological (meta)genomics.
Vieira-Silva S1, Rocha EP.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2797632/

----------
## Knight R

https://knightlab.ucsd.edu/wordpress/?page_id=25

### 2024-05
https://pubmed.ncbi.nlm.nih.gov/37500913/
Nat Biotechnol
. 2024 May;42(5):715-718. doi: 10.1038/s41587-023-01845-1. Epub 2023 Jul 27.
Greengenes2 unifies microbial data in a single reference tree
Daniel McDonald 1, Yueyu Jiang 2, Metin Balaban 3, Kalen Cantrell 4, Qiyun Zhu 5 6, Antonio Gonzalez 1, James T Morton 7, Giorgia Nicolaou 8, Donovan H Parks 9, SÃ¸ren M Karst 10, Mads Albertsen 11, Philip Hugenholtz 9, Todd DeSantis 12, Se Jin Song 13, Andrew Bartko 13, Aki S Havulinna 14 15, Pekka Jousilahti 14, Susan Cheng 16 17, Michael Inouye 18 19, Teemu Niiranen 14 20, Mohit Jain 21, Veikko Salomaa 14, Leo Lahti 22, Siavash Mirarab 2, Rob Knight 23 24 25 26
https://www.nature.com/articles/s41587-023-01845-1
https://pmc.ncbi.nlm.nih.gov/articles/PMC10818020/

### 2018-07
https://pubmed.ncbi.nlm.nih.gov/29795328/
Review Nat Rev Microbiol
. 2018 Jul;16(7):410-422. doi: 10.1038/s41579-018-0029-9.
Best practices for analysing microbiomes
Rob Knight 1 2 3, Alison Vrbanac 4, Bryn C Taylor 4, Alexander Aksenov 5, Chris Callewaert 6 7, Justine Debelius 6, Antonio Gonzalez 6, Tomasz Kosciolek 6, Laura-Isobel McCall 5, Daniel McDonald 6, Alexey V Melnik 5, James T Morton 6 8, Jose Navas 8, Robert A Quinn 5, Jon G Sanders 6, Austin D Swafford 9, Luke R Thompson 10 11, Anupriya Tripathi 12, Zhenjiang Z Xu 6, Jesse R Zaneveld 13, Qiyun Zhu 6, J Gregory Caporaso 14, Pieter C Dorrestein 9 5 6
https://www.nature.com/articles/s41579-018-0029-9
As rodents are coprophagic, cage mate faecal microbiomes become more homogeneous over time, so experiments must be replicated across multiple cages to control for cage effects19.

7:25 AM Â· May 24, 2018
https://twitter.com/merenbey/status/999416078446071809
A. Murat Eren (Meren) on Twitter: "This is unfair to DADA2 ğŸ˜ When someone as big as Rob Knight says "this is the best practice", it creates a nightmare for younger scientists to promote their tool. I think, from "Deblur" to "Deblur or DADA2" would've added much more than just 9 more characters to this chart. https://t.co/5q71yxh4Yt" / Twitter

### 2018-06
https://pubmed.ncbi.nlm.nih.gov/29795540/
Review Nat Microbiol
. 2018 Jun;3(6):652-661. doi: 10.1038/s41564-018-0156-0. Epub 2018 May 24.
Methods for phylogenetic analysis of microbiome data
Alex D Washburne # 1, James T Morton # 2 3, Jon Sanders 3, Daniel McDonald 3, Qiyun Zhu 3, Angela M Oliverio 4 5, Rob Knight 2 3
https://www.nature.com/articles/s41564-018-0156-0
In this Review Article, we organize the field of phylogenetically structured data analysis by discussing the major classes of methods. We first emphasize a fundamental issue in the field: the imperfection of estimated phylogenies. We then define four classes: (1) comparative methods; (2) ancestral state reconstruction and descendant trait imputation; (3) phylogenetic variables; and (4) phylogeny-aware distances (Table 1); and provide Supplementary Online Tutorials with examples (https://knightlab-analyses.github.io/phylogenetic-tutorials/). 

Fig. 3: Phylogeny-aware distances.
UniFrac

----------
## Mason CE

https://www.ncbi.nlm.nih.gov/pubmed/29930288
Nat Rev Microbiol. 2018 Jun 21. doi: 10.1038/s41579-018-0047-7. [Epub ahead of print]
Surveying what's flushed away.
Iraola G1,2,3, Kumar N3.
https://www.nature.com/articles/s41579-018-0047-7

https://www.ncbi.nlm.nih.gov/pubmed/29890228
Drug Discov Today. 2018 Jun 8. pii: S1359-6446(17)30357-4. doi: 10.1016/j.drudis.2018.06.005. [Epub ahead of print]
Interrogating the microbiome: experimental and computational considerations in support of study reproducibility.
Poussin C1, Sierro N1, BouÃ© S1, Battey J1, Scotti E1, Belcastro V1, Peitsch MC1, Ivanov NV1, Hoeng J2.
Review
Informatics

https://link.springer.com/protocol/10.1007/978-1-4939-7463-4_8
Comparative Metagenomics | SpringerLink
Andrew Maltez ThomasFelipe Prata LimaLivia Maria Silva MouraAline Maria da SilvaEmmanuel Dias-NetoJoÃ£o C. SetubalEmail author
Protocol
First Online: 26 December 2017

https://www.ncbi.nlm.nih.gov/pubmed/28978331
Microbiome. 2017 Oct 4;5(1):132. doi: 10.1186/s40168-017-0349-4.
Communicating the promise, risks, and ethics of large-scale, open space microbiome and metagenome research.
Shamarina D1, Stoyantcheva I1, Mason CE2,3,4, Bibby K5, Elhaik E6.



https://www.ncbi.nlm.nih.gov/pubmed/28349976
Nat Microbiol. 2017 Mar 28;2:16275. doi: 10.1038/nmicrobiol.2016.275.
Geomicrobiology of the built environment.
Gadd GM1.
http://www.nature.com/articles/nmicrobiol2016275

https://www.ncbi.nlm.nih.gov/pubmed/27941038
Br Med Bull. 2016 Dec;120(1):27-33. Epub 2016 Nov 23.
Globalizing and crowdsourcing biomedical research.
Afshinnekoo E1,2,3, Ahsanuddin S1,2,4, Mason CE5,2,6.




----------


https://jp.illumina.com/content/dam/illumina-marketing/apac/japan/documents/pdf/publicaton_metagenome_for-human-health-j.pdf
ãƒ’ãƒˆã®å¥åº·ã«ãŠã‘ã‚‹ç´°èŒãŠã‚ˆã³ãƒ¡ã‚¿ã‚²ãƒãƒ ã‚¤ãƒ«ãƒŸãƒŠãƒ†ã‚¯ãƒãƒ­ã‚·ã‚™ãƒ¼ã‚’ä½¿ç”¨ã—ãŸæœ€æ–°è«–æ–‡ã®æ¦‚è¦
https://jp.illumina.com/science/publication-reviews.html
è«–æ–‡é›† | æœ€è¿‘ç™ºè¡¨ã•ã‚ŒãŸç ”ç©¶ã®è¦ç‚¹

å±±ç”°æ‹“å¸ è‘—

https://www.jstage.jst.go.jp/article/jim/29/1/29_19/_article/-char/ja/
https://www.jstage.jst.go.jp/article/jim/29/1/29_19/_pdf
è…¸å†…ç´°èŒç¾¤é›†æ§‹é€ ã®ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ - J-Stage
è…¸å†…ç´°èŒå­¦é›‘èªŒ 29 : 19-22,2015

https://jp.illumina.com/content/dam/illumina-marketing/apac/japan/documents/pdf/2014_webinar_mktg30_microbiology.pdf
[PDF]ãƒ’ãƒˆè…¸å†…ç´°èŒå¢ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ

https://www.jstage.jst.go.jp/article/kagakutoseibutsu/51/12/51_802/_pdf
ãƒ’ãƒˆè…¸å†…ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãŒ åºƒã’ã‚‹åŒ»ç™‚å±•é–‹ - J-Stage
åŒ–å­¦ã¨ç”Ÿç‰© Vol. 51, No. 12, 2013

----------
## metabolic inference
ãƒ¡ã‚¿16Sã‹ã‚‰ãƒ¡ã‚¿ã‚²ãƒãƒ æ©Ÿèƒ½æ¨å®š
functional prediction

æ—¥æœ¬èªã®ãƒ„ã‚¤ãƒ¼ãƒˆ
https://twitter.com/search?q=lang%3Aja%20(PICRUSt%20OR%20Tax4Fun%20OR%20Piphillin)&src=typed_query&f=live

https://doi.org/10.7875/togotv.2019.203
2019-12-16 ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æãƒ„ãƒ¼ãƒ« @ AJACSåå‹2
1:10:50 / 1:28:59
ã‚¢ãƒ³ãƒ—ãƒªã‚³ãƒ³è§£æãƒ‡ãƒ¼ã‚¿ã‹ã‚‰ã®éºä¼å­æ©Ÿèƒ½çµ„æˆæ¨å®š

### PICRUSt

01 June 2020
https://www.nature.com/articles/s41587-020-0548-6
PICRUSt2 for prediction of metagenome functions | Nature Biotechnology

11:32 AM Â· Jun 5, 2020
https://twitter.com/maskot1977/status/1268732318077693952
å¾®ç”Ÿç‰©ç¾¤é›†ã®ãƒãƒ¼ã‚«ãƒ¼éºä¼å­ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚·ãƒ³ã‚°ã®é™ç•Œã®ä¸€ã¤ã¯ã€ã‚µãƒ³ãƒ—ãƒªãƒ³ã‚°ã•ã‚ŒãŸç¾¤é›†ã®æ©Ÿèƒ½çš„æ§‹æˆã«é–¢ã™ã‚‹æƒ…å ±ãŒå¾—ã‚‰ã‚Œãªã„ã“ã¨ã§ã‚ã‚‹ã€‚
PICRUSt1ã¯2013å¹´ã«é–‹ç™ºã•ã‚Œã€ãƒãƒ¼ã‚«ãƒ¼éºä¼å­ã‚·ãƒ¼ã‚¯ã‚¨ãƒ³ã‚·ãƒ³ã‚°ãƒ—ãƒ­ãƒ•ã‚¡ã‚¤ãƒ«ã«åŸºã¥ã„ã¦ç´°èŒç¾¤é›†ã®æ©Ÿèƒ½çš„å¯èƒ½æ€§ã‚’äºˆæ¸¬ã™ã‚‹ãŸã‚ã«é–‹ç™ºã•ã‚Œã¾ã—ãŸãŒã€ã“ã®åº¦ã€ã‚ªãƒªã‚¸ãƒŠãƒ«ã®æ‰‹æ³•ã‚’æ”¹è‰¯ã—ãŸPICRUSt2ï¼ˆhttps://github.com/picrust/picrust2ï¼‰ã‚’ç™ºè¡¨ã—ã¾ã™ã€‚
å…·ä½“çš„ã«ã¯ã€PICRUSt2ã¯ã€éºä¼å­ãƒ•ã‚¡ãƒŸãƒªãƒ¼ã¨å‚ç…§ã‚²ãƒãƒ ã®æ›´æ–°ã•ã‚ŒãŸå¤§è¦æ¨¡ãªãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã‚’å«ã¿ã€ä»»æ„ã®ã‚ªãƒšãƒ¬ãƒ¼ã‚·ãƒ§ãƒŠãƒ«ãƒ»ã‚¿ã‚¯ã‚½ãƒãƒŸãƒƒã‚¯ãƒ»ãƒ¦ãƒ‹ãƒƒãƒˆï¼ˆOTUï¼‰ãƒ”ãƒƒã‚­ãƒ³ã‚°ã¾ãŸã¯ãƒ‡ãƒã‚¢ã‚¤ã‚¸ãƒ³ã‚°ãƒ»ã‚¢ãƒ«ã‚´ãƒªã‚ºãƒ ã¨ã®ç›¸äº’é‹ç”¨æ€§ã‚’æä¾›ã—ã€è¡¨ç¾å‹äºˆæ¸¬ã‚’å¯èƒ½ã«ã—ã¾ã™ã€‚
ãƒ™ãƒ³ãƒãƒãƒ¼ã‚¯ã§ã¯ã€PICRUSt2ã¯PICRUStã‚„ä»–ã®ç«¶åˆã™ã‚‹æ‰‹æ³•ã‚ˆã‚Šã‚‚å…¨ä½“çš„ã«ç²¾åº¦ãŒé«˜ã„ã“ã¨ãŒç¤ºã•ã‚Œã¦ã„ã¾ã™ã€‚PICRUSt2ã§ã¯ã€ã‚«ã‚¹ã‚¿ãƒ ã®å‚ç…§ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã‚’è¿½åŠ ã™ã‚‹ã“ã¨ã‚‚å¯èƒ½ã§ã™ã€‚ã“ã‚Œã‚‰ã®æ”¹å–„ç‚¹ã¨ã€äºˆæ¸¬ã•ã‚ŒãŸãƒ¡ã‚¿ã‚²ãƒãƒ ã®ä½¿ç”¨ã«é–¢ã™ã‚‹é‡è¦ãªæ³¨æ„ç‚¹ã‚’å¼·èª¿ã—ã¦ã„ã¾ã™ã€‚

https://twitter.com/NoahFierer/status/1179101529044766720
Noah Fierer on Twitter: "PiCrust2: when you want to predict metagenomes slightly better than random (sometimes) https://t.co/4FGgo4oAHN" / Twitter
3:31 AM Â· Oct 2, 2019

Posted June 15, 2019.
https://www.biorxiv.org/content/10.1101/672295v1
PICRUSt2: An improved and extensible approach for metagenome inference | bioRxiv
https://github.com/picrust/picrust2
Please note that PICRUSt2 is currently in beta.
https://github.com/picrust/picrust2/wiki
PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.

https://en.wikipedia.org/wiki/PICRUSt

http://picrust.github.io/picrust/
Please note that PICRUSt2 is now available. PICRUSt2 is a re-written version of PICRUSt and is available here. We are no longer developing PICRUSt1 and we recommend users shift to PICRUSt2.

http://anonym.hatenablog.jp/entry/2018/10/25/172452
QIIME2ã§PICRUSt2ãŒåˆ©ç”¨å¯èƒ½ã«ãªã£ãŸã‚‰ã—ã„
http://anonym.hatenablog.jp/entry/2018/09/10/183941
ç´°èŒå¢è§£æã«ãŠã‘ã‚‹Functional Inference
PICRUSt
PICRUSt2
Tax4Fun
Piphillin
SINAPS
http://anonym.hatenablog.jp/entry/2018/07/05/100753
PICRUSt2ã‚’ä½¿ç”¨ã—ã¦ã¿ã‚‹
http://anonym.hatenablog.jp/entry/2018/03/12/123658
DADA2ã‹ã‚‰PICRUStã¸

### Tax4Fun
http://tax4fun.gobics.de/
Tax4Fun: predicting functional profiles from metagenomic 16S rRNA data

09 Dec 2018
http://europepmc.org/article/PPR/PPR64283
Tax4Fun2: a R-based tool for the rapid prediction of habitat-specific functional profiles and functional redundancy based on 16S rRNA gene marker gene sequences - Abstract - Europe PMC
https://www.biorxiv.org/content/10.1101/490037v1
https://www.biorxiv.org/content/10.1101/490037v1.full.pdf
In recent years, several freely available tools such as PICRUSt10, Tax4Fun11, Piphillin12, Faprotax13 and paprica14 have been developed. 

https://www.ncbi.nlm.nih.gov/pubmed/25957349
Bioinformatics. 2015 Sep 1;31(17):2882-4. doi: 10.1093/bioinformatics/btv287. Epub 2015 May 7.
Tax4Fun: predicting functional profiles from metagenomic 16S rRNA data.
AÃŸhauer KP1, Wemheuer B2, Daniel R2, Meinicke P1.

### themetagenomics
https://github.com/EESI/themetagenomics
themetagenomics provides functions to explore topics generated from 16S rRNA sequencing information on both the abundance and functional levels. It also provides an R implementation of PICRUSt and wraps Tax4fun, giving users a choice for their functional prediction strategy.

Posted June 21, 2019.
https://www.biorxiv.org/content/10.1101/678110v1
Themetagenomics: Exploring Thematic Structure and Predicted Functionality of 16s rRNA Amplicon Data | bioRxiv
https://www.biorxiv.org/content/10.1101/678110v1.full.pdf
Standard methods that predict function from 16S rRNA survey data include PICRUSt, Tax4fun, Piphillin, and SINAPS [14â€“17]. 

https://github.com/EESI/themetagenomics/issues/10
Package 'themetagenomics' removed from CRAN Â· Issue #10 Â· EESI/themetagenomics
The R package on CRAN was removed apparently because of "check errors":
https://cran.r-project.org/web/packages/themetagenomics/index.html
```
Package â€˜themetagenomicsâ€™ was removed from the CRAN repository.

Formerly available versions can be obtained from the archive.

Archived on 2019-12-21 as check problems were not corrected in time.
```

### paprica

Nov 10, 2019
https://github.com/bowmanjeffs/paprica
PAthway PRediction by phylogenetIC plAcement

August 18, 2015 by Jeff
https://www.polarmicrobes.org/introducing-paprica/
Introducing PAPRICA | The Bowman Lab

### Piphillin
http://piphillin.secondgenome.com/

https://www.ncbi.nlm.nih.gov/pubmed/31952477
BMC Genomics. 2020 Jan 17;21(1):56. doi: 10.1186/s12864-019-6427-1.
Piphillin predicts metagenomic composition and dynamics from DADA2-corrected 16S rDNA sequences.
Narayan NR1, Weinmaier T1, Laserna-Mendieta EJ2,3, Claesson MJ2,3, Shanahan F2,4, Dabbagh K1, Iwai S1, DeSantis TZ5.

https://www.ncbi.nlm.nih.gov/pubmed/27820856
PLoS One. 2016 Nov 7;11(11):e0166104. doi: 10.1371/journal.pone.0166104. eCollection 2016.
Piphillin: Improved Prediction of Metagenomic Content by Direct Inference from Human Microbiomes.
Iwai S1, Weinmaier T1, Schmidt BL2, Albertson DG2,3, Poloso NJ4, Dabbagh K1, DeSantis TZ1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5098786/
Table 2
Piphillin has less pre-requisites than PICRUSt and Tax4Fun.
https://journals.plos.org/plosone/article/figure?id=10.1371/journal.pone.0166104.t002

2017-03-13
https://pediatricsurgery.hatenadiary.jp/entry/2017/03/13/152234
16S rRNAéºä¼å­è§£æã®é™ç•Œã¨äºˆæ¸¬ãƒ¡ã‚¿ã‚²ãƒãƒ è§£æ - Piphillinã®å°å…¥ - Note of Pediatric Surgery 
ä¸‹è¨˜ã®å›³ã«PICRUStã¨Tax4Funã«å¿…è¦ãªæ¡ä»¶ãŒã¾ã¨ã‚ã‚‰ã‚Œã¦ã„ã¾ã™ã€‚
![](http://journals.plos.org/plosone/article/figure/image?size=large&download=&id=10.1371/journal.pone.0166104.t002)

2016-08-13
http://blog.goo.ne.jp/gooquruquru/e/72dd0c0494c54e0f2be827b54af33ba7
äºˆæ¸¬ã®ãŸã‚ã®ãƒ„ãƒ¼ãƒ«ãŒï¼“ã¤ï¼ˆPICRUSt, Tax4Fun, papricaï¼‰ã€

### 


https://twitter.com/sai_kai_cs/status/1115428029235322880
è…¸å†…ç´°èŒã®16Sã¨ãƒ¡ã‚¿ã‚²ãƒãƒ æ¯”è¼ƒè«–æ–‡ã€‚ã¡ã‚ƒã‚“ã¨èª­ã‚“ã§ãªã„ã‘ã©ã€PICRUStã¯Pathwayãƒ¬ãƒ™ãƒ«ã§ç›¸é–¢ä¿‚æ•°0.80ãã‚‰ã„ã®ã‚ˆã†ã  ä¼¼ã¦ã‚‹çµŒè·¯ã‚‚ã‚ã‚Œã°ä¼¼ã¦ãªã„çµŒè·¯ã‚‚ã‚ã‚‹æ§˜å­
https://www.ncbi.nlm.nih.gov/pubmed/27148170
Front Microbiol. 2016 Apr 20;7:459. doi: 10.3389/fmicb.2016.00459. eCollection 2016.
Characterization of the Gut Microbiome Using 16S or Shotgun Metagenomics.
Jovel J1, Patterson J1, Wang W1, Hotte N1, O'Keefe S1, Mitchel T1, Perry T1, Kao D1, Mason AL1, Madsen KL1, Wong GK2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4837688/
The software PICRUSt (Langille et al., 2013) can be used to infer metabolic capacity of the microbiome contained in 16S libraries.

https://www.nature.com/articles/s41579-018-0029-9
Best practices for analysing microbiomes

Predictive functional profiling38,39,40,41 is a technique for linking marker gene data with available microbial genomes to make predictions about metagenomic content and thus the putative biological functions of a microbial community. This analysis generally requires a reference-based OTU table. Methods based on evolutionary models (for example, PICRUSt39) provide confidence intervals on these predictions of gene content, which will tend to be wider in regions of the tree distant from reference genome sequences and narrower where many reference genomes are available. 

https://www.ncbi.nlm.nih.gov/pubmed/26409790
BMC Res Notes. 2015 Sep 26;8:479. doi: 10.1186/s13104-015-1462-8.
PanFP: pangenome-based functional profiles for microbial communities.
Jun SR1,2, Robeson MS3,4, Hauser LJ5, Schadt CW6,7, Gorin AA8.

https://www.ncbi.nlm.nih.gov/pubmed/23149747
Nat Commun. 2012;3:1203. doi: 10.1038/ncomms2203.
Virtual metagenome reconstruction from 16S rRNA gene sequences.
Okuda S1, Tsuchiya Y, Kiriyama C, Itoh M, Morisaki H.


----------








